{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "#import the qiime2 python module to allow for inline visualizations\n",
    "\n",
    "import qiime2 as q2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSystem versions\u001b[0m\n",
      "Python version: 3.6.10\n",
      "QIIME 2 release: 2020.6\n",
      "QIIME 2 version: 2020.6.0\n",
      "q2cli version: 2020.6.0\n",
      "\u001b[32m\n",
      "Installed plugins\u001b[0m\n",
      "alignment: 2020.6.0\n",
      "composition: 2020.6.0\n",
      "cutadapt: 2020.6.0\n",
      "dada2: 2020.6.0\n",
      "deblur: 2020.6.0\n",
      "demux: 2020.6.0\n",
      "diversity: 2020.6.0\n",
      "diversity-lib: 2020.6.0\n",
      "emperor: 2020.6.0\n",
      "feature-classifier: 2020.6.0\n",
      "feature-table: 2020.6.0\n",
      "fragment-insertion: 2020.6.0\n",
      "gneiss: 2020.6.0\n",
      "longitudinal: 2020.6.0\n",
      "metadata: 2020.6.0\n",
      "phylogeny: 2020.6.0\n",
      "quality-control: 2020.6.0\n",
      "quality-filter: 2020.6.0\n",
      "sample-classifier: 2020.6.0\n",
      "taxa: 2020.6.0\n",
      "types: 2020.6.0\n",
      "vsearch: 2020.6.0\n",
      "\u001b[32m\n",
      "Application config directory\u001b[0m\n",
      "/Users/stephaniematthews/miniconda3/envs/qiime2-2020.6/var/q2cli\u001b[0m\n",
      "\u001b[32m\n",
      "Getting help\u001b[0m\n",
      "To get help with QIIME 2, visit https://qiime2.org\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#which version of qiime2 and python are we using?\n",
    "\n",
    "!qiime info"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "First, import the paired forward and reverse sequences with a manifest file. Manifest prepared by listing all files in bash and formatting in Excel. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported leray_manifest_run1_grid.csv as PairedEndFastqManifestPhred33 to COI_1-paired-end-demux.qza\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#Import the fastq files into a QIIME2 artifact \n",
    "\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path leray_manifest_run1_grid.csv \\\n",
    "  --output-path COI_1-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Trim reverse primers\n",
    "See: https://docs.qiime2.org/2020.6/plugins/available/cutadapt/trim-paired/\n",
    "default is to match IUPAC wildcards ( --p-match-adapter-wildcards T )\n",
    "- these should be the F/R marker primers\n",
    "- cannot handle inosine bases; using N instead\n",
    "- f primer/ r primer\n",
    "- or could just trim off length of primer+barcode? this provides a test of primer accuracy/error \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S001_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S001_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S001_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S001_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S001_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S001_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S001_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S001_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,393 reads  @     89.6 µs/read;   0.67 M reads/minute\n",
      "Finished in 2.91 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,393\n",
      "  Read 1 with adapter:                  32,365 (99.9%)\n",
      "  Read 2 with adapter:                  32,122 (99.2%)\n",
      "Pairs written (passing filters):        32,393 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,107,367 bp\n",
      "  Read 1:     9,555,543 bp\n",
      "  Read 2:     9,551,824 bp\n",
      "Total written (filtered):     17,431,725 bp (91.2%)\n",
      "  Read 1:     8,713,815 bp\n",
      "  Read 2:     8,717,910 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32365 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t288\t0.0\t2\t0 277 11\n",
      "26\t31637\t0.0\t2\t30730 876 31\n",
      "27\t290\t0.0\t2\t0 275 15\n",
      "28\t134\t0.0\t2\t0 0 134\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32122 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t1379\t0.0\t2\t0 1347 32\n",
      "26\t30527\t0.0\t2\t29836 673 18\n",
      "27\t163\t0.0\t2\t0 159 4\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S002_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S002_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S002_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S002_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S002_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S002_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S002_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S002_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,557 reads  @     82.8 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.77 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,557\n",
      "  Read 1 with adapter:                  57,517 (99.9%)\n",
      "  Read 2 with adapter:                  57,309 (99.6%)\n",
      "Pairs written (passing filters):        57,557 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,955,633 bp\n",
      "  Read 1:    16,978,797 bp\n",
      "  Read 2:    16,976,836 bp\n",
      "Total written (filtered):     30,970,955 bp (91.2%)\n",
      "  Read 1:    15,483,296 bp\n",
      "  Read 2:    15,487,659 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57517 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t526\t0.0\t2\t0 512 14\n",
      "26\t56440\t0.0\t2\t55015 1406 19\n",
      "27\t341\t0.0\t2\t0 330 11\n",
      "28\t166\t0.0\t2\t0 0 166\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t1041\t0.0\t2\t0 1006 35\n",
      "26\t55949\t0.0\t2\t54468 1423 58\n",
      "27\t254\t0.0\t2\t0 235 19\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S003_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S003_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S003_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S003_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S003_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S003_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S003_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S003_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        69,039 reads  @     87.0 µs/read;   0.69 M reads/minute\n",
      "Finished in 6.01 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,039\n",
      "  Read 1 with adapter:                  68,994 (99.9%)\n",
      "  Read 2 with adapter:                  68,479 (99.2%)\n",
      "Pairs written (passing filters):        69,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,731,067 bp\n",
      "  Read 1:    20,365,377 bp\n",
      "  Read 2:    20,365,690 bp\n",
      "Total written (filtered):     37,158,126 bp (91.2%)\n",
      "  Read 1:    18,571,565 bp\n",
      "  Read 2:    18,586,561 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68994 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t729\t0.0\t2\t0 708 21\n",
      "26\t67632\t0.0\t2\t66043 1557 32\n",
      "27\t385\t0.0\t2\t0 371 14\n",
      "28\t202\t0.0\t2\t0 0 202\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68479 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t1542\t0.0\t2\t0 1504 38\n",
      "26\t66589\t0.0\t2\t65050 1475 64\n",
      "27\t287\t0.0\t2\t0 278 9\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S004_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S004_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S004_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S004_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S004_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S004_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S004_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S004_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        34,547 reads  @     97.2 µs/read;   0.62 M reads/minute\n",
      "Finished in 3.36 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,547\n",
      "  Read 1 with adapter:                  34,522 (99.9%)\n",
      "  Read 2 with adapter:                  33,737 (97.7%)\n",
      "Pairs written (passing filters):        34,546 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,267,932 bp\n",
      "  Read 1:    10,191,101 bp\n",
      "  Read 2:    10,076,831 bp\n",
      "Total written (filtered):     18,493,406 bp (91.2%)\n",
      "  Read 1:     9,293,443 bp\n",
      "  Read 2:     9,199,963 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34522 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t374\t0.0\t2\t0 362 12\n",
      "26\t33903\t0.0\t2\t33049 836 18\n",
      "27\t183\t0.0\t2\t0 171 12\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33737 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t416\t0.0\t2\t0 397 19\n",
      "26\t33128\t0.0\t2\t32112 962 54\n",
      "27\t148\t0.0\t2\t0 139 9\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S005_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S005_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S005_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S005_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S005_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S005_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S005_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S005_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:07        73,292 reads  @    103.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 7.56 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,292\n",
      "  Read 1 with adapter:                  73,230 (99.9%)\n",
      "  Read 2 with adapter:                  68,191 (93.0%)\n",
      "Pairs written (passing filters):        73,290 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,370,803 bp\n",
      "  Read 1:    21,619,064 bp\n",
      "  Read 2:    20,751,739 bp\n",
      "Total written (filtered):     38,694,944 bp (91.3%)\n",
      "  Read 1:    19,715,089 bp\n",
      "  Read 2:    18,979,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t898\t0.0\t2\t0 867 31\n",
      "26\t71889\t0.0\t2\t70098 1754 37\n",
      "27\t271\t0.0\t2\t0 267 4\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68191 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t68\t0.0\t2\t0 0 68\n",
      "25\t1314\t0.0\t2\t0 1251 63\n",
      "26\t66501\t0.0\t2\t64477 1916 108\n",
      "27\t248\t0.0\t2\t0 226 22\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S006_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S006_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S006_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S006_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S006_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S006_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S006_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S006_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        73,240 reads  @     98.9 µs/read;   0.61 M reads/minute\n",
      "Finished in 7.24 s (99 us/read; 0.61 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             73,240\n",
      "  Read 1 with adapter:                  73,159 (99.9%)\n",
      "  Read 2 with adapter:                  72,895 (99.5%)\n",
      "Pairs written (passing filters):        73,239 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,170,391 bp\n",
      "  Read 1:    21,603,403 bp\n",
      "  Read 2:    21,566,988 bp\n",
      "Total written (filtered):     39,374,532 bp (91.2%)\n",
      "  Read 1:    19,701,553 bp\n",
      "  Read 2:    19,672,979 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 73159 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t923\t0.0\t2\t0 898 25\n",
      "26\t71748\t0.0\t2\t69884 1832 32\n",
      "27\t302\t0.0\t2\t0 291 11\n",
      "28\t110\t0.0\t2\t0 0 110\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72895 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t82\t0.0\t2\t0 0 82\n",
      "25\t1447\t0.0\t2\t0 1408 39\n",
      "26\t71068\t0.0\t2\t69116 1869 83\n",
      "27\t246\t0.0\t2\t0 237 9\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S007_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S007_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S007_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S007_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S007_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S007_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S007_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S007_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        72,146 reads  @    103.2 µs/read;   0.58 M reads/minute\n",
      "Finished in 7.45 s (103 us/read; 0.58 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,146\n",
      "  Read 1 with adapter:                  72,068 (99.9%)\n",
      "  Read 2 with adapter:                  72,005 (99.8%)\n",
      "Pairs written (passing filters):        72,146 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,557,841 bp\n",
      "  Read 1:    21,278,823 bp\n",
      "  Read 2:    21,279,018 bp\n",
      "Total written (filtered):     38,812,603 bp (91.2%)\n",
      "  Read 1:    19,405,303 bp\n",
      "  Read 2:    19,407,300 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 72068 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t88\t0.0\t2\t0 0 88\n",
      "25\t872\t0.0\t2\t0 843 29\n",
      "26\t70524\t0.0\t2\t68636 1860 28\n",
      "27\t368\t0.0\t2\t0 348 20\n",
      "28\t216\t0.0\t2\t0 0 216\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72005 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t792\t0.0\t2\t0 758 34\n",
      "26\t70797\t0.0\t2\t68826 1898 73\n",
      "27\t312\t0.0\t2\t0 292 20\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S008_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S008_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S008_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S008_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S008_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S008_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S008_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S008_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        63,056 reads  @     93.2 µs/read;   0.64 M reads/minute\n",
      "Finished in 5.88 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,056\n",
      "  Read 1 with adapter:                  62,981 (99.9%)\n",
      "  Read 2 with adapter:                  62,934 (99.8%)\n",
      "Pairs written (passing filters):        63,056 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,196,828 bp\n",
      "  Read 1:    18,596,411 bp\n",
      "  Read 2:    18,600,417 bp\n",
      "Total written (filtered):     33,923,493 bp (91.2%)\n",
      "  Read 1:    16,958,911 bp\n",
      "  Read 2:    16,964,582 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62981 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t636\t0.0\t2\t0 617 19\n",
      "26\t61698\t0.0\t2\t60090 1566 42\n",
      "27\t412\t0.0\t2\t0 396 16\n",
      "28\t172\t0.0\t2\t0 0 172\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62934 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t677\t0.0\t2\t0 651 26\n",
      "26\t61915\t0.0\t2\t60408 1457 50\n",
      "27\t252\t0.0\t2\t0 241 11\n",
      "28\t39\t0.0\t2\t0 0 39\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S009_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S009_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S009_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S009_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S009_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S009_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S009_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S009_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,359 reads  @     86.0 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.64 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,359\n",
      "  Read 1 with adapter:                  42,324 (99.9%)\n",
      "  Read 2 with adapter:                  42,000 (99.2%)\n",
      "Pairs written (passing filters):        42,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,986,505 bp\n",
      "  Read 1:    12,495,588 bp\n",
      "  Read 2:    12,490,917 bp\n",
      "Total written (filtered):     22,795,704 bp (91.2%)\n",
      "  Read 1:    11,395,041 bp\n",
      "  Read 2:    11,400,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42324 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t304\t0.0\t2\t0 292 12\n",
      "26\t41663\t0.0\t2\t40625 1014 24\n",
      "27\t175\t0.0\t2\t0 168 7\n",
      "28\t154\t0.0\t2\t0 0 154\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t1868\t0.0\t2\t0 1814 54\n",
      "26\t39879\t0.0\t2\t38963 894 22\n",
      "27\t144\t0.0\t2\t0 137 7\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S010_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S010_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S010_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S010_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S010_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S010_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S010_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S010_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,219 reads  @     68.9 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.08 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,219\n",
      "  Read 1 with adapter:                  59,180 (99.9%)\n",
      "  Read 2 with adapter:                  59,039 (99.7%)\n",
      "Pairs written (passing filters):        59,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,937,638 bp\n",
      "  Read 1:    17,469,140 bp\n",
      "  Read 2:    17,468,498 bp\n",
      "Total written (filtered):     31,864,726 bp (91.2%)\n",
      "  Read 1:    15,930,406 bp\n",
      "  Read 2:    15,934,320 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59180 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t441\t0.0\t2\t0 425 16\n",
      "26\t58261\t0.0\t2\t56930 1313 18\n",
      "27\t261\t0.0\t2\t0 254 7\n",
      "28\t167\t0.0\t2\t0 0 167\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59039 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t996\t0.0\t2\t0 971 25\n",
      "26\t57754\t0.0\t2\t56435 1276 43\n",
      "27\t230\t0.0\t2\t0 209 21\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S011_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S011_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S011_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S011_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S011_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S011_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S011_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S011_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,187 reads  @     72.8 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.51 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,187\n",
      "  Read 1 with adapter:                  48,150 (99.9%)\n",
      "  Read 2 with adapter:                  48,039 (99.7%)\n",
      "Pairs written (passing filters):        48,186 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,428,516 bp\n",
      "  Read 1:    14,214,601 bp\n",
      "  Read 2:    14,213,915 bp\n",
      "Total written (filtered):     25,928,360 bp (91.2%)\n",
      "  Read 1:    12,962,713 bp\n",
      "  Read 2:    12,965,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48150 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t420\t0.0\t2\t0 404 16\n",
      "26\t47351\t0.0\t2\t46243 1088 20\n",
      "27\t218\t0.0\t2\t0 217 1\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48039 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t1139\t0.0\t2\t0 1107 32\n",
      "26\t46671\t0.0\t2\t45559 1081 31\n",
      "27\t182\t0.0\t2\t0 174 8\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S012_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S012_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S012_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S012_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S012_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S012_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S012_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S012_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        41,308 reads  @     97.1 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.01 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,308\n",
      "  Read 1 with adapter:                  40,966 (99.2%)\n",
      "  Read 2 with adapter:                  40,406 (97.8%)\n",
      "Pairs written (passing filters):        41,307 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,244,733 bp\n",
      "  Read 1:    12,185,401 bp\n",
      "  Read 2:    12,059,332 bp\n",
      "Total written (filtered):     22,129,198 bp (91.3%)\n",
      "  Read 1:    11,120,068 bp\n",
      "  Read 2:    11,009,130 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40966 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t353\t0.0\t2\t0 342 11\n",
      "26\t40333\t0.0\t2\t39294 1024 15\n",
      "27\t163\t0.0\t2\t0 152 11\n",
      "28\t93\t0.0\t2\t0 0 93\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t509\t0.0\t2\t0 485 24\n",
      "26\t39665\t0.0\t2\t38406 1178 81\n",
      "27\t193\t0.0\t2\t0 182 11\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S013_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S013_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S013_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S013_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S013_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S013_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S013_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S013_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        81,229 reads  @     99.7 µs/read;   0.60 M reads/minute\n",
      "Finished in 8.10 s (100 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             81,229\n",
      "  Read 1 with adapter:                  81,120 (99.9%)\n",
      "  Read 2 with adapter:                  76,307 (93.9%)\n",
      "Pairs written (passing filters):        81,228 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,095,166 bp\n",
      "  Read 1:    23,961,272 bp\n",
      "  Read 2:    23,133,894 bp\n",
      "Total written (filtered):     43,003,473 bp (91.3%)\n",
      "  Read 1:    21,852,595 bp\n",
      "  Read 2:    21,150,878 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 81120 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t825\t0.0\t2\t0 791 34\n",
      "26\t79823\t0.0\t2\t77683 2110 30\n",
      "27\t314\t0.0\t2\t0 301 13\n",
      "28\t96\t0.0\t2\t0 0 96\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 76307 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t1472\t0.0\t2\t0 1425 47\n",
      "26\t74421\t0.0\t2\t72022 2262 137\n",
      "27\t287\t0.0\t2\t0 268 19\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S014_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S014_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S014_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S014_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S014_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S014_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S014_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S014_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        50,951 reads  @     95.2 µs/read;   0.63 M reads/minute\n",
      "Finished in 4.85 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,951\n",
      "  Read 1 with adapter:                  50,915 (99.9%)\n",
      "  Read 2 with adapter:                  50,763 (99.6%)\n",
      "Pairs written (passing filters):        50,948 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,034,833 bp\n",
      "  Read 1:    15,029,291 bp\n",
      "  Read 2:    15,005,542 bp\n",
      "Total written (filtered):     27,391,620 bp (91.2%)\n",
      "  Read 1:    13,705,003 bp\n",
      "  Read 2:    13,686,617 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50915 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t499\t0.0\t2\t0 484 15\n",
      "26\t50120\t0.0\t2\t48760 1343 17\n",
      "27\t186\t0.0\t2\t0 181 5\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50763 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t993\t0.0\t2\t0 947 46\n",
      "26\t49518\t0.0\t2\t48218 1247 53\n",
      "27\t164\t0.0\t2\t0 161 3\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S015_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S015_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S015_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S015_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S015_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S015_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S015_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S015_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,420 reads  @     82.3 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.73 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,420\n",
      "  Read 1 with adapter:                  57,365 (99.9%)\n",
      "  Read 2 with adapter:                  57,296 (99.8%)\n",
      "Pairs written (passing filters):        57,419 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,874,877 bp\n",
      "  Read 1:    16,938,524 bp\n",
      "  Read 2:    16,936,353 bp\n",
      "Total written (filtered):     30,893,918 bp (91.2%)\n",
      "  Read 1:    15,447,241 bp\n",
      "  Read 2:    15,446,677 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57365 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t567\t0.0\t2\t0 545 22\n",
      "26\t56400\t0.0\t2\t54844 1529 27\n",
      "27\t244\t0.0\t2\t0 231 13\n",
      "28\t106\t0.0\t2\t0 0 106\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57296 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t622\t0.0\t2\t0 588 34\n",
      "26\t56274\t0.0\t2\t54574 1623 77\n",
      "27\t287\t0.0\t2\t0 271 16\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S016_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S016_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S016_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S016_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S016_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S016_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S016_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S016_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,126 reads  @     87.2 µs/read;   0.69 M reads/minute\n",
      "Finished in 4.81 s (87 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,126\n",
      "  Read 1 with adapter:                  55,081 (99.9%)\n",
      "  Read 2 with adapter:                  55,058 (99.9%)\n",
      "Pairs written (passing filters):        55,126 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,523,930 bp\n",
      "  Read 1:    16,262,074 bp\n",
      "  Read 2:    16,261,856 bp\n",
      "Total written (filtered):     29,660,878 bp (91.2%)\n",
      "  Read 1:    14,830,183 bp\n",
      "  Read 2:    14,830,695 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55081 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t527\t0.0\t2\t0 516 11\n",
      "26\t54197\t0.0\t2\t52715 1449 33\n",
      "27\t246\t0.0\t2\t0 239 7\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55058 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t558\t0.0\t2\t0 544 14\n",
      "26\t54199\t0.0\t2\t52825 1333 41\n",
      "27\t223\t0.0\t2\t0 214 9\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S017_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S017_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S017_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S017_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S017_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S017_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S017_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S017_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,055 reads  @     86.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.06 s (86 us/read; 0.69 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,055\n",
      "  Read 1 with adapter:                  46,927 (99.7%)\n",
      "  Read 2 with adapter:                  46,618 (99.1%)\n",
      "Pairs written (passing filters):        47,055 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,750,894 bp\n",
      "  Read 1:    13,876,162 bp\n",
      "  Read 2:    13,874,732 bp\n",
      "Total written (filtered):     25,320,683 bp (91.2%)\n",
      "  Read 1:    12,656,038 bp\n",
      "  Read 2:    12,664,645 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46927 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t402\t0.0\t2\t0 386 16\n",
      "26\t46132\t0.0\t2\t44861 1249 22\n",
      "27\t210\t0.0\t2\t0 200 10\n",
      "28\t145\t0.0\t2\t0 0 145\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46618 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t2120\t0.0\t2\t0 2059 61\n",
      "26\t44186\t0.0\t2\t42972 1169 45\n",
      "27\t237\t0.0\t2\t0 233 4\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S018_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S018_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S018_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S018_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S018_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S018_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S018_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S018_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,486 reads  @     75.3 µs/read;   0.80 M reads/minute\n",
      "Finished in 3.58 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,486\n",
      "  Read 1 with adapter:                  47,253 (99.5%)\n",
      "  Read 2 with adapter:                  47,271 (99.5%)\n",
      "Pairs written (passing filters):        47,486 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,004,961 bp\n",
      "  Read 1:    13,998,814 bp\n",
      "  Read 2:    14,006,147 bp\n",
      "Total written (filtered):     25,548,007 bp (91.2%)\n",
      "  Read 1:    12,770,192 bp\n",
      "  Read 2:    12,777,815 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47253 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t335\t0.0\t2\t0 331 4\n",
      "26\t46545\t0.0\t2\t45482 1051 12\n",
      "27\t223\t0.0\t2\t0 214 9\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47271 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t882\t0.0\t2\t0 852 30\n",
      "26\t46105\t0.0\t2\t44818 1222 65\n",
      "27\t232\t0.0\t2\t0 218 14\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S019_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S019_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S019_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S019_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S019_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S019_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S019_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S019_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        31,412 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 2.26 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             31,412\n",
      "  Read 1 with adapter:                  31,287 (99.6%)\n",
      "  Read 2 with adapter:                  31,245 (99.5%)\n",
      "Pairs written (passing filters):        31,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    18,523,040 bp\n",
      "  Read 1:     9,261,342 bp\n",
      "  Read 2:     9,261,698 bp\n",
      "Total written (filtered):     16,897,757 bp (91.2%)\n",
      "  Read 1:     8,447,838 bp\n",
      "  Read 2:     8,449,919 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 31287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t225\t0.0\t2\t0 218 7\n",
      "26\t30841\t0.0\t2\t30136 696 9\n",
      "27\t115\t0.0\t2\t0 112 3\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 31245 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t688\t0.0\t2\t0 665 23\n",
      "26\t30388\t0.0\t2\t29651 700 37\n",
      "27\t137\t0.0\t2\t0 129 8\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S020_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S020_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S020_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S020_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S020_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S020_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S020_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S020_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,191 reads  @     76.7 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.08 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,191\n",
      "  Read 1 with adapter:                  40,065 (99.7%)\n",
      "  Read 2 with adapter:                  39,907 (99.3%)\n",
      "Pairs written (passing filters):        40,191 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,693,493 bp\n",
      "  Read 1:    11,851,145 bp\n",
      "  Read 2:    11,842,348 bp\n",
      "Total written (filtered):     21,614,563 bp (91.2%)\n",
      "  Read 1:    10,809,377 bp\n",
      "  Read 2:    10,805,186 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t326\t0.0\t2\t0 313 13\n",
      "26\t39393\t0.0\t2\t38378 999 16\n",
      "27\t172\t0.0\t2\t0 165 7\n",
      "28\t145\t0.0\t2\t0 0 145\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39907 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t541\t0.0\t2\t0 518 23\n",
      "26\t39120\t0.0\t2\t37686 1335 99\n",
      "27\t195\t0.0\t2\t0 175 20\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S021_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S021_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S021_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S021_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S021_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S021_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S021_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S021_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,366 reads  @     80.7 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.75 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,366\n",
      "  Read 1 with adapter:                  46,232 (99.7%)\n",
      "  Read 2 with adapter:                  45,671 (98.5%)\n",
      "Pairs written (passing filters):        46,366 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,248,937 bp\n",
      "  Read 1:    13,673,276 bp\n",
      "  Read 2:    13,575,661 bp\n",
      "Total written (filtered):     24,860,178 bp (91.2%)\n",
      "  Read 1:    12,471,232 bp\n",
      "  Read 2:    12,388,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46232 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t376\t0.0\t2\t0 361 15\n",
      "26\t45501\t0.0\t2\t44308 1168 25\n",
      "27\t202\t0.0\t2\t0 199 3\n",
      "28\t123\t0.0\t2\t0 0 123\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45671 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t882\t0.0\t2\t0 839 43\n",
      "26\t44499\t0.0\t2\t42804 1559 136\n",
      "27\t213\t0.0\t2\t0 188 25\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S022_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S022_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S022_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S022_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S022_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S022_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S022_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S022_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,214 reads  @     73.6 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.62 s (74 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,214\n",
      "  Read 1 with adapter:                  49,149 (99.9%)\n",
      "  Read 2 with adapter:                  48,738 (99.0%)\n",
      "Pairs written (passing filters):        49,214 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,950,792 bp\n",
      "  Read 1:    14,515,921 bp\n",
      "  Read 2:    14,434,871 bp\n",
      "Total written (filtered):     26,406,817 bp (91.2%)\n",
      "  Read 1:    13,238,328 bp\n",
      "  Read 2:    13,168,489 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49149 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t469\t0.0\t2\t0 455 14\n",
      "26\t48413\t0.0\t2\t47178 1217 18\n",
      "27\t162\t0.0\t2\t0 155 7\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t965\t0.0\t2\t0 925 40\n",
      "26\t47475\t0.0\t2\t46058 1340 77\n",
      "27\t177\t0.0\t2\t0 171 6\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S023_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S023_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S023_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S023_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S023_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S023_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S023_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S023_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,939 reads  @     76.7 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.52 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,939\n",
      "  Read 1 with adapter:                  58,822 (99.8%)\n",
      "  Read 2 with adapter:                  58,802 (99.8%)\n",
      "Pairs written (passing filters):        58,939 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,763,809 bp\n",
      "  Read 1:    17,378,988 bp\n",
      "  Read 2:    17,384,821 bp\n",
      "Total written (filtered):     31,706,360 bp (91.2%)\n",
      "  Read 1:    15,850,027 bp\n",
      "  Read 2:    15,856,333 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58822 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t620\t0.0\t2\t0 603 17\n",
      "26\t57936\t0.0\t2\t56365 1548 23\n",
      "27\t187\t0.0\t2\t0 183 4\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58802 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t661\t0.0\t2\t0 637 24\n",
      "26\t57755\t0.0\t2\t55917 1757 81\n",
      "27\t267\t0.0\t2\t0 257 10\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S024_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S024_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S024_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S024_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S024_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S024_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S024_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S024_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        50,524 reads  @     84.6 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.28 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,524\n",
      "  Read 1 with adapter:                  50,397 (99.7%)\n",
      "  Read 2 with adapter:                  50,443 (99.8%)\n",
      "Pairs written (passing filters):        50,524 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,798,261 bp\n",
      "  Read 1:    14,894,438 bp\n",
      "  Read 2:    14,903,823 bp\n",
      "Total written (filtered):     27,177,043 bp (91.2%)\n",
      "  Read 1:    13,584,360 bp\n",
      "  Read 2:    13,592,683 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t490\t0.0\t2\t0 468 22\n",
      "26\t49615\t0.0\t2\t48194 1395 26\n",
      "27\t170\t0.0\t2\t0 165 5\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50443 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t593\t0.0\t2\t0 570 23\n",
      "26\t49542\t0.0\t2\t48138 1335 69\n",
      "27\t201\t0.0\t2\t0 194 7\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S025_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S025_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S025_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S025_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S025_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S025_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S025_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S025_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        20,219 reads  @     79.1 µs/read;   0.76 M reads/minute\n",
      "Finished in 1.60 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,219\n",
      "  Read 1 with adapter:                  20,198 (99.9%)\n",
      "  Read 2 with adapter:                  20,016 (99.0%)\n",
      "Pairs written (passing filters):        20,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    11,925,969 bp\n",
      "  Read 1:     5,964,065 bp\n",
      "  Read 2:     5,961,904 bp\n",
      "Total written (filtered):     10,881,311 bp (91.2%)\n",
      "  Read 1:     5,438,995 bp\n",
      "  Read 2:     5,442,316 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 20198 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t192\t0.0\t2\t0 179 13\n",
      "26\t19884\t0.0\t2\t19115 740 29\n",
      "27\t70\t0.0\t2\t0 68 2\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 20016 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t918\t0.0\t2\t0 897 21\n",
      "26\t18941\t0.0\t2\t18430 499 12\n",
      "27\t80\t0.0\t2\t0 75 5\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S026_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S026_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S026_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S026_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S026_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S026_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S026_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S026_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        68,110 reads  @     67.3 µs/read;   0.89 M reads/minute\n",
      "Finished in 4.59 s (67 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,110\n",
      "  Read 1 with adapter:                  68,057 (99.9%)\n",
      "  Read 2 with adapter:                  67,974 (99.8%)\n",
      "Pairs written (passing filters):        68,110 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,178,468 bp\n",
      "  Read 1:    20,091,620 bp\n",
      "  Read 2:    20,086,848 bp\n",
      "Total written (filtered):     36,643,070 bp (91.2%)\n",
      "  Read 1:    18,322,517 bp\n",
      "  Read 2:    18,320,553 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68057 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t616\t0.0\t2\t0 597 19\n",
      "26\t67101\t0.0\t2\t65112 1951 38\n",
      "27\t231\t0.0\t2\t0 225 6\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67974 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t84\t0.0\t2\t0 0 84\n",
      "25\t1328\t0.0\t2\t0 1284 44\n",
      "26\t66193\t0.0\t2\t64302 1826 65\n",
      "27\t271\t0.0\t2\t0 259 12\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S027_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S027_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S027_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S027_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S027_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S027_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S027_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S027_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,021 reads  @     75.9 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.56 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,021\n",
      "  Read 1 with adapter:                  59,987 (99.9%)\n",
      "  Read 2 with adapter:                  59,850 (99.7%)\n",
      "Pairs written (passing filters):        60,021 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,410,488 bp\n",
      "  Read 1:    17,705,793 bp\n",
      "  Read 2:    17,704,695 bp\n",
      "Total written (filtered):     32,296,070 bp (91.2%)\n",
      "  Read 1:    16,146,229 bp\n",
      "  Read 2:    16,149,841 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59987 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t438\t0.0\t2\t0 427 11\n",
      "26\t59201\t0.0\t2\t57734 1445 22\n",
      "27\t212\t0.0\t2\t0 205 7\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59850 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t1415\t0.0\t2\t0 1380 35\n",
      "26\t58123\t0.0\t2\t56692 1387 44\n",
      "27\t231\t0.0\t2\t0 224 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S028_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S028_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S028_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S028_55_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S028_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S028_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S028_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S028_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,305 reads  @     89.7 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.78 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,305\n",
      "  Read 1 with adapter:                  53,273 (99.9%)\n",
      "  Read 2 with adapter:                  52,961 (99.4%)\n",
      "Pairs written (passing filters):        53,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,419,020 bp\n",
      "  Read 1:    15,724,783 bp\n",
      "  Read 2:    15,694,237 bp\n",
      "Total written (filtered):     28,657,477 bp (91.2%)\n",
      "  Read 1:    14,339,729 bp\n",
      "  Read 2:    14,317,748 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t423\t0.0\t2\t0 405 18\n",
      "26\t52492\t0.0\t2\t51141 1323 28\n",
      "27\t217\t0.0\t2\t0 209 8\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52961 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t720\t0.0\t2\t0 700 20\n",
      "26\t51921\t0.0\t2\t50381 1456 84\n",
      "27\t265\t0.0\t2\t0 255 10\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S029_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S029_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S029_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S029_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S029_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S029_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S029_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S029_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,305 reads  @     85.1 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.54 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,305\n",
      "  Read 1 with adapter:                  53,273 (99.9%)\n",
      "  Read 2 with adapter:                  52,961 (99.4%)\n",
      "Pairs written (passing filters):        53,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,419,020 bp\n",
      "  Read 1:    15,724,783 bp\n",
      "  Read 2:    15,694,237 bp\n",
      "Total written (filtered):     28,657,477 bp (91.2%)\n",
      "  Read 1:    14,339,729 bp\n",
      "  Read 2:    14,317,748 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t423\t0.0\t2\t0 405 18\n",
      "26\t52492\t0.0\t2\t51141 1323 28\n",
      "27\t217\t0.0\t2\t0 209 8\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52961 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t720\t0.0\t2\t0 700 20\n",
      "26\t51921\t0.0\t2\t50381 1456 84\n",
      "27\t265\t0.0\t2\t0 255 10\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S030_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S030_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S030_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S030_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S030_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S030_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S030_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S030_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        67,443 reads  @     80.2 µs/read;   0.75 M reads/minute\n",
      "Finished in 5.41 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,443\n",
      "  Read 1 with adapter:                  67,398 (99.9%)\n",
      "  Read 2 with adapter:                  67,227 (99.7%)\n",
      "Pairs written (passing filters):        67,443 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,770,902 bp\n",
      "  Read 1:    19,895,185 bp\n",
      "  Read 2:    19,875,717 bp\n",
      "Total written (filtered):     36,272,147 bp (91.2%)\n",
      "  Read 1:    18,142,988 bp\n",
      "  Read 2:    18,129,159 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67398 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t549\t0.0\t2\t0 528 21\n",
      "26\t66411\t0.0\t2\t64756 1624 31\n",
      "27\t262\t0.0\t2\t0 249 13\n",
      "28\t122\t0.0\t2\t0 0 122\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67227 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t86\t0.0\t2\t0 0 86\n",
      "25\t1457\t0.0\t2\t0 1414 43\n",
      "26\t65411\t0.0\t2\t63748 1603 60\n",
      "27\t261\t0.0\t2\t0 251 10\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S031_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S031_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S031_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S031_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S031_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S031_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S031_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S031_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        28,148 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 2.59 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,148\n",
      "  Read 1 with adapter:                  28,129 (99.9%)\n",
      "  Read 2 with adapter:                  28,028 (99.6%)\n",
      "Pairs written (passing filters):        28,148 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,604,448 bp\n",
      "  Read 1:     8,303,378 bp\n",
      "  Read 2:     8,301,070 bp\n",
      "Total written (filtered):     15,144,623 bp (91.2%)\n",
      "  Read 1:     7,572,079 bp\n",
      "  Read 2:     7,572,544 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 28129 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t264\t0.0\t2\t0 256 8\n",
      "26\t27678\t0.0\t2\t26770 892 16\n",
      "27\t101\t0.0\t2\t0 99 2\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28028 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t327\t0.0\t2\t0 311 16\n",
      "26\t27518\t0.0\t2\t26326 1112 80\n",
      "27\t145\t0.0\t2\t0 133 12\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S032_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S032_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S032_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S032_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S032_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S032_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S032_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S032_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        66,776 reads  @     76.9 µs/read;   0.78 M reads/minute\n",
      "Finished in 5.13 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,776\n",
      "  Read 1 with adapter:                  66,730 (99.9%)\n",
      "  Read 2 with adapter:                  66,683 (99.9%)\n",
      "Pairs written (passing filters):        66,776 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,397,377 bp\n",
      "  Read 1:    19,698,812 bp\n",
      "  Read 2:    19,698,565 bp\n",
      "Total written (filtered):     35,929,380 bp (91.2%)\n",
      "  Read 1:    17,964,115 bp\n",
      "  Read 2:    17,965,265 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66730 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t594\t0.0\t2\t0 569 25\n",
      "26\t65786\t0.0\t2\t64002 1755 29\n",
      "27\t225\t0.0\t2\t0 215 10\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66683 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t708\t0.0\t2\t0 683 25\n",
      "26\t65597\t0.0\t2\t63982 1578 37\n",
      "27\t254\t0.0\t2\t0 247 7\n",
      "28\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S033_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S033_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S033_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S033_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S033_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S033_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S033_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S033_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,390 reads  @     80.4 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.73 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,390\n",
      "  Read 1 with adapter:                  46,355 (99.9%)\n",
      "  Read 2 with adapter:                  46,026 (99.2%)\n",
      "Pairs written (passing filters):        46,390 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,364,714 bp\n",
      "  Read 1:    13,684,915 bp\n",
      "  Read 2:    13,679,799 bp\n",
      "Total written (filtered):     24,965,142 bp (91.2%)\n",
      "  Read 1:    12,479,971 bp\n",
      "  Read 2:    12,485,171 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46355 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t488\t0.0\t2\t0 471 17\n",
      "26\t45602\t0.0\t2\t44117 1449 36\n",
      "27\t180\t0.0\t2\t0 173 7\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46026 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t2182\t0.0\t2\t0 2127 55\n",
      "26\t43563\t0.0\t2\t42519 1017 27\n",
      "27\t192\t0.0\t2\t0 181 11\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S034_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S034_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S034_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S034_67_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S034_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S034_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S034_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S034_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        63,163 reads  @     77.1 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.87 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,163\n",
      "  Read 1 with adapter:                  63,085 (99.9%)\n",
      "  Read 2 with adapter:                  63,057 (99.8%)\n",
      "Pairs written (passing filters):        63,163 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,262,409 bp\n",
      "  Read 1:    18,631,273 bp\n",
      "  Read 2:    18,631,136 bp\n",
      "Total written (filtered):     33,983,975 bp (91.2%)\n",
      "  Read 1:    16,991,384 bp\n",
      "  Read 2:    16,992,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 63085 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t578\t0.0\t2\t0 558 20\n",
      "26\t62199\t0.0\t2\t60614 1552 33\n",
      "27\t215\t0.0\t2\t0 207 8\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63057 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t1174\t0.0\t2\t0 1145 29\n",
      "26\t61530\t0.0\t2\t59934 1531 65\n",
      "27\t281\t0.0\t2\t0 269 12\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S035_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S035_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S035_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S035_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S035_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S035_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S035_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S035_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,974 reads  @     70.2 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.28 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,974\n",
      "  Read 1 with adapter:                  60,899 (99.9%)\n",
      "  Read 2 with adapter:                  60,868 (99.8%)\n",
      "Pairs written (passing filters):        60,974 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,970,814 bp\n",
      "  Read 1:    17,985,154 bp\n",
      "  Read 2:    17,985,660 bp\n",
      "Total written (filtered):     32,806,358 bp (91.2%)\n",
      "  Read 1:    16,402,011 bp\n",
      "  Read 2:    16,404,347 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t570\t0.0\t2\t0 547 23\n",
      "26\t59957\t0.0\t2\t58298 1624 35\n",
      "27\t253\t0.0\t2\t0 244 9\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t73\t0.0\t2\t0 0 73\n",
      "25\t1408\t0.0\t2\t0 1381 27\n",
      "26\t59108\t0.0\t2\t57655 1423 30\n",
      "27\t259\t0.0\t2\t0 250 9\n",
      "28\t20\t0.0\t2\t0 0 20\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S036_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S036_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S036_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S036_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S036_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S036_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S036_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S036_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,938 reads  @     78.1 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.44 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,938\n",
      "  Read 1 with adapter:                  43,854 (99.8%)\n",
      "  Read 2 with adapter:                  43,656 (99.4%)\n",
      "Pairs written (passing filters):        43,938 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,900,364 bp\n",
      "  Read 1:    12,959,857 bp\n",
      "  Read 2:    12,940,507 bp\n",
      "Total written (filtered):     23,625,665 bp (91.2%)\n",
      "  Read 1:    11,819,819 bp\n",
      "  Read 2:    11,805,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43854 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t407\t0.0\t2\t0 396 11\n",
      "26\t43186\t0.0\t2\t42037 1132 17\n",
      "27\t181\t0.0\t2\t0 173 8\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43656 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t579\t0.0\t2\t0 549 30\n",
      "26\t42833\t0.0\t2\t41439 1319 75\n",
      "27\t200\t0.0\t2\t0 185 15\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S037_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S037_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S037_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S037_73_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S037_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S037_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S037_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S037_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        35,320 reads  @     79.9 µs/read;   0.75 M reads/minute\n",
      "Finished in 2.83 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             35,320\n",
      "  Read 1 with adapter:                  35,241 (99.8%)\n",
      "  Read 2 with adapter:                  34,362 (97.3%)\n",
      "Pairs written (passing filters):        35,320 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,678,343 bp\n",
      "  Read 1:    10,417,144 bp\n",
      "  Read 2:    10,261,199 bp\n",
      "Total written (filtered):     18,869,417 bp (91.3%)\n",
      "  Read 1:     9,501,014 bp\n",
      "  Read 2:     9,368,403 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 35241 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t333\t0.0\t2\t0 319 14\n",
      "26\t34707\t0.0\t2\t33658 1029 20\n",
      "27\t145\t0.0\t2\t0 138 7\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34362 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t701\t0.0\t2\t0 669 32\n",
      "26\t33455\t0.0\t2\t32311 1087 57\n",
      "27\t147\t0.0\t2\t0 136 11\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S038_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S038_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S038_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S038_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S038_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S038_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S038_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S038_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        68,497 reads  @     73.3 µs/read;   0.82 M reads/minute\n",
      "Finished in 5.03 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,497\n",
      "  Read 1 with adapter:                  68,420 (99.9%)\n",
      "  Read 2 with adapter:                  68,228 (99.6%)\n",
      "Pairs written (passing filters):        68,495 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,382,365 bp\n",
      "  Read 1:    20,204,720 bp\n",
      "  Read 2:    20,177,645 bp\n",
      "Total written (filtered):     36,830,574 bp (91.2%)\n",
      "  Read 1:    18,425,545 bp\n",
      "  Read 2:    18,405,029 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68420 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t664\t0.0\t2\t0 641 23\n",
      "26\t67365\t0.0\t2\t65574 1760 31\n",
      "27\t273\t0.0\t2\t0 264 9\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 68228 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t91\t0.0\t2\t0 0 91\n",
      "25\t1497\t0.0\t2\t0 1455 42\n",
      "26\t66292\t0.0\t2\t64476 1754 62\n",
      "27\t329\t0.0\t2\t0 316 13\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S039_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S039_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S039_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S039_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S039_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S039_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S039_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S039_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        69,193 reads  @     78.2 µs/read;   0.77 M reads/minute\n",
      "Finished in 5.41 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,193\n",
      "  Read 1 with adapter:                  69,111 (99.9%)\n",
      "  Read 2 with adapter:                  69,031 (99.8%)\n",
      "Pairs written (passing filters):        69,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,811,644 bp\n",
      "  Read 1:    20,410,442 bp\n",
      "  Read 2:    20,401,202 bp\n",
      "Total written (filtered):     37,220,785 bp (91.2%)\n",
      "  Read 1:    18,613,849 bp\n",
      "  Read 2:    18,606,936 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 69111 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t618\t0.0\t2\t0 606 12\n",
      "26\t68089\t0.0\t2\t66296 1746 47\n",
      "27\t275\t0.0\t2\t0 265 10\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 69031 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t802\t0.0\t2\t0 776 26\n",
      "26\t67822\t0.0\t2\t65843 1874 105\n",
      "27\t336\t0.0\t2\t0 319 17\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S040_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S040_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S040_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S040_79_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S040_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S040_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S040_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S040_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         2,414 reads  @    140.2 µs/read;   0.43 M reads/minute\n",
      "Finished in 0.34 s (141 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              2,414\n",
      "  Read 1 with adapter:                   2,374 (98.3%)\n",
      "  Read 2 with adapter:                   2,373 (98.3%)\n",
      "Pairs written (passing filters):         2,414 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,423,006 bp\n",
      "  Read 1:       710,878 bp\n",
      "  Read 2:       712,128 bp\n",
      "Total written (filtered):      1,299,620 bp (91.3%)\n",
      "  Read 1:       649,166 bp\n",
      "  Read 2:       650,454 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 2374 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t25\t0.0\t2\t0 24 1\n",
      "26\t2336\t0.0\t2\t2253 82 1\n",
      "27\t5\t0.0\t2\t0 5\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 2373 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t3\t0.0\t2\t0 0 3\n",
      "25\t36\t0.0\t2\t0 34 2\n",
      "26\t2320\t0.0\t2\t2233 86 1\n",
      "27\t10\t0.0\t2\t0 10\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S041_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S041_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S041_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S041_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S041_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S041_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S041_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S041_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        29,089 reads  @     84.7 µs/read;   0.71 M reads/minute\n",
      "Finished in 2.47 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,089\n",
      "  Read 1 with adapter:                  29,068 (99.9%)\n",
      "  Read 2 with adapter:                  28,858 (99.2%)\n",
      "Pairs written (passing filters):        29,089 (100.0%)\n",
      "\n",
      "Total basepairs processed:    17,158,681 bp\n",
      "  Read 1:     8,580,915 bp\n",
      "  Read 2:     8,577,766 bp\n",
      "Total written (filtered):     15,653,916 bp (91.2%)\n",
      "  Read 1:     7,825,086 bp\n",
      "  Read 2:     7,828,830 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 29068 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t292\t0.0\t2\t0 283 9\n",
      "26\t28467\t0.0\t2\t27650 794 23\n",
      "27\t173\t0.0\t2\t0 160 13\n",
      "28\t113\t0.0\t2\t0 0 113\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 28858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t1422\t0.0\t2\t0 1389 33\n",
      "26\t27275\t0.0\t2\t26616 642 17\n",
      "27\t112\t0.0\t2\t0 110 2\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S042_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S042_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S042_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S042_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S042_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S042_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S042_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S042_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,202 reads  @     78.7 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.95 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,202\n",
      "  Read 1 with adapter:                  50,171 (99.9%)\n",
      "  Read 2 with adapter:                  50,076 (99.7%)\n",
      "Pairs written (passing filters):        50,202 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,618,012 bp\n",
      "  Read 1:    14,809,583 bp\n",
      "  Read 2:    14,808,429 bp\n",
      "Total written (filtered):     27,012,354 bp (91.2%)\n",
      "  Read 1:    13,505,198 bp\n",
      "  Read 2:    13,507,156 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50171 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t460\t0.0\t2\t0 448 12\n",
      "26\t49311\t0.0\t2\t48241 1054 16\n",
      "27\t241\t0.0\t2\t0 236 5\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50076 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t855\t0.0\t2\t0 835 20\n",
      "26\t48956\t0.0\t2\t47784 1128 44\n",
      "27\t222\t0.0\t2\t0 213 9\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S043_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S043_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S043_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S043_85_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S043_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S043_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S043_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S043_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07       100,648 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 7.63 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            100,648\n",
      "  Read 1 with adapter:                 100,577 (99.9%)\n",
      "  Read 2 with adapter:                 100,409 (99.8%)\n",
      "Pairs written (passing filters):       100,648 (100.0%)\n",
      "\n",
      "Total basepairs processed:    59,381,312 bp\n",
      "  Read 1:    29,690,783 bp\n",
      "  Read 2:    29,690,529 bp\n",
      "Total written (filtered):     54,158,050 bp (91.2%)\n",
      "  Read 1:    27,076,054 bp\n",
      "  Read 2:    27,081,996 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 100577 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t989\t0.0\t2\t0 955 34\n",
      "26\t98879\t0.0\t2\t96546 2287 46\n",
      "27\t450\t0.0\t2\t0 433 17\n",
      "28\t196\t0.0\t2\t0 0 196\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 100409 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t104\t0.0\t2\t0 0 104\n",
      "25\t2365\t0.0\t2\t0 2300 65\n",
      "26\t97483\t0.0\t2\t95250 2161 72\n",
      "27\t442\t0.0\t2\t0 430 12\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S044_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S044_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S044_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S044_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S044_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S044_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S044_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S044_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,321 reads  @     82.5 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.82 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,321\n",
      "  Read 1 with adapter:                  46,282 (99.9%)\n",
      "  Read 2 with adapter:                  45,744 (98.8%)\n",
      "Pairs written (passing filters):        46,321 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,247,994 bp\n",
      "  Read 1:    13,664,306 bp\n",
      "  Read 2:    13,583,688 bp\n",
      "Total written (filtered):     24,855,948 bp (91.2%)\n",
      "  Read 1:    12,461,191 bp\n",
      "  Read 2:    12,394,757 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46282 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t474\t0.0\t2\t0 461 13\n",
      "26\t45514\t0.0\t2\t44381 1116 17\n",
      "27\t223\t0.0\t2\t0 213 10\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45744 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t588\t0.0\t2\t0 561 27\n",
      "26\t44903\t0.0\t2\t43645 1198 60\n",
      "27\t211\t0.0\t2\t0 196 15\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S045_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S045_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S045_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S045_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S045_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S045_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S045_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S045_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,667 reads  @     88.7 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.67 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,667\n",
      "  Read 1 with adapter:                  52,641 (100.0%)\n",
      "  Read 2 with adapter:                  45,637 (86.7%)\n",
      "Pairs written (passing filters):        52,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,855,874 bp\n",
      "  Read 1:    15,536,450 bp\n",
      "  Read 2:    14,319,424 bp\n",
      "Total written (filtered):     27,301,502 bp (91.4%)\n",
      "  Read 1:    14,167,927 bp\n",
      "  Read 2:    13,133,575 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t578\t0.0\t2\t0 556 22\n",
      "26\t51816\t0.0\t2\t50574 1228 14\n",
      "27\t180\t0.0\t2\t0 176 4\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45637 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t871\t0.0\t2\t0 847 24\n",
      "26\t44507\t0.0\t2\t43214 1228 65\n",
      "27\t188\t0.0\t2\t0 179 9\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S046_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S046_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S046_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S046_91_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S046_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S046_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S046_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S046_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,360 reads  @     83.6 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.13 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,360\n",
      "  Read 1 with adapter:                  49,325 (99.9%)\n",
      "  Read 2 with adapter:                  49,160 (99.6%)\n",
      "Pairs written (passing filters):        49,359 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,099,179 bp\n",
      "  Read 1:    14,560,706 bp\n",
      "  Read 2:    14,538,473 bp\n",
      "Total written (filtered):     26,539,589 bp (91.2%)\n",
      "  Read 1:    13,278,352 bp\n",
      "  Read 2:    13,261,237 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49325 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t549\t0.0\t2\t0 532 17\n",
      "26\t48547\t0.0\t2\t47409 1122 16\n",
      "27\t158\t0.0\t2\t0 154 4\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49160 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t1079\t0.0\t2\t0 1051 28\n",
      "26\t47795\t0.0\t2\t46586 1158 51\n",
      "27\t197\t0.0\t2\t0 187 10\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S047_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S047_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S047_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S047_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S047_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S047_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S047_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S047_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        54,840 reads  @     86.3 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.73 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,840\n",
      "  Read 1 with adapter:                  54,793 (99.9%)\n",
      "  Read 2 with adapter:                  54,739 (99.8%)\n",
      "Pairs written (passing filters):        54,839 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,352,352 bp\n",
      "  Read 1:    16,176,947 bp\n",
      "  Read 2:    16,175,405 bp\n",
      "Total written (filtered):     29,504,761 bp (91.2%)\n",
      "  Read 1:    14,752,192 bp\n",
      "  Read 2:    14,752,569 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54793 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t533\t0.0\t2\t0 521 12\n",
      "26\t53852\t0.0\t2\t52542 1291 19\n",
      "27\t263\t0.0\t2\t0 258 5\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54739 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t606\t0.0\t2\t0 587 19\n",
      "26\t53799\t0.0\t2\t52463 1272 64\n",
      "27\t256\t0.0\t2\t0 248 8\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S048_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S048_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S048_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S048_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S048_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S048_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S048_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S048_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,720 reads  @     84.5 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.54 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,720\n",
      "  Read 1 with adapter:                  53,683 (99.9%)\n",
      "  Read 2 with adapter:                  53,644 (99.9%)\n",
      "Pairs written (passing filters):        53,720 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,693,849 bp\n",
      "  Read 1:    15,846,833 bp\n",
      "  Read 2:    15,847,016 bp\n",
      "Total written (filtered):     28,903,989 bp (91.2%)\n",
      "  Read 1:    14,451,327 bp\n",
      "  Read 2:    14,452,662 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53683 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t645\t0.0\t2\t0 622 23\n",
      "26\t52612\t0.0\t2\t51123 1453 36\n",
      "27\t263\t0.0\t2\t0 258 5\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53644 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t589\t0.0\t2\t0 561 28\n",
      "26\t52728\t0.0\t2\t51320 1362 46\n",
      "27\t231\t0.0\t2\t0 215 16\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S049_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S049_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S049_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S049_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S049_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S049_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S049_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S049_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,711 reads  @     82.4 µs/read;   0.73 M reads/minute\n",
      "Finished in 2.78 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,711\n",
      "  Read 1 with adapter:                  33,679 (99.9%)\n",
      "  Read 2 with adapter:                  33,439 (99.2%)\n",
      "Pairs written (passing filters):        33,711 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,881,163 bp\n",
      "  Read 1:     9,940,572 bp\n",
      "  Read 2:     9,940,591 bp\n",
      "Total written (filtered):     18,137,433 bp (91.2%)\n",
      "  Read 1:     9,064,770 bp\n",
      "  Read 2:     9,072,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33679 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t240\t0.0\t2\t0 226 14\n",
      "26\t33123\t0.0\t2\t32090 1011 22\n",
      "27\t148\t0.0\t2\t0 144 4\n",
      "28\t144\t0.0\t2\t0 0 144\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33439 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t1594\t0.0\t2\t0 1561 33\n",
      "26\t31629\t0.0\t2\t30918 692 19\n",
      "27\t132\t0.0\t2\t0 129 3\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S050_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S050_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S050_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S050_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S050_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S050_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S050_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S050_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,642 reads  @     65.6 µs/read;   0.91 M reads/minute\n",
      "Finished in 4.05 s (66 us/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,642\n",
      "  Read 1 with adapter:                  61,597 (99.9%)\n",
      "  Read 2 with adapter:                  61,532 (99.8%)\n",
      "Pairs written (passing filters):        61,642 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,363,311 bp\n",
      "  Read 1:    18,181,070 bp\n",
      "  Read 2:    18,182,241 bp\n",
      "Total written (filtered):     33,162,818 bp (91.2%)\n",
      "  Read 1:    16,579,462 bp\n",
      "  Read 2:    16,583,356 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t369\t0.0\t2\t0 356 13\n",
      "26\t60829\t0.0\t2\t59609 1207 13\n",
      "27\t219\t0.0\t2\t0 213 6\n",
      "28\t149\t0.0\t2\t0 0 149\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61532 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t1114\t0.0\t2\t0 1081 33\n",
      "26\t60084\t0.0\t2\t58802 1225 57\n",
      "27\t265\t0.0\t2\t0 254 11\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S051_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S051_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S051_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S051_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S051_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S051_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S051_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S051_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,977 reads  @     68.2 µs/read;   0.88 M reads/minute\n",
      "Finished in 4.02 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,977\n",
      "  Read 1 with adapter:                  58,931 (99.9%)\n",
      "  Read 2 with adapter:                  58,887 (99.8%)\n",
      "Pairs written (passing filters):        58,977 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,791,286 bp\n",
      "  Read 1:    17,395,411 bp\n",
      "  Read 2:    17,395,875 bp\n",
      "Total written (filtered):     31,729,238 bp (91.2%)\n",
      "  Read 1:    15,863,139 bp\n",
      "  Read 2:    15,866,099 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58931 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t417\t0.0\t2\t0 403 14\n",
      "26\t58111\t0.0\t2\t56807 1283 21\n",
      "27\t235\t0.0\t2\t0 229 6\n",
      "28\t146\t0.0\t2\t0 0 146\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t1439\t0.0\t2\t0 1393 46\n",
      "26\t57148\t0.0\t2\t55946 1158 44\n",
      "27\t243\t0.0\t2\t0 234 9\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S052_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S052_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S052_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S052_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S052_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S052_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S052_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S052_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,244 reads  @     82.0 µs/read;   0.73 M reads/minute\n",
      "Finished in 4.20 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,244\n",
      "  Read 1 with adapter:                  51,109 (99.7%)\n",
      "  Read 2 with adapter:                  50,512 (98.6%)\n",
      "Pairs written (passing filters):        51,244 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,106,857 bp\n",
      "  Read 1:    15,092,209 bp\n",
      "  Read 2:    15,014,648 bp\n",
      "Total written (filtered):     27,465,186 bp (91.2%)\n",
      "  Read 1:    13,763,401 bp\n",
      "  Read 2:    13,701,785 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51109 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t387\t0.0\t2\t0 380 7\n",
      "26\t50379\t0.0\t2\t49218 1151 10\n",
      "27\t197\t0.0\t2\t0 187 10\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50512 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t629\t0.0\t2\t0 598 31\n",
      "26\t49592\t0.0\t2\t48242 1273 77\n",
      "27\t234\t0.0\t2\t0 218 16\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S053_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S053_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S053_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S053_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S053_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S053_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S053_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S053_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        68,624 reads  @     93.5 µs/read;   0.64 M reads/minute\n",
      "Finished in 6.42 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,624\n",
      "  Read 1 with adapter:                  68,366 (99.6%)\n",
      "  Read 2 with adapter:                  63,077 (91.9%)\n",
      "Pairs written (passing filters):        68,624 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,483,854 bp\n",
      "  Read 1:    20,185,585 bp\n",
      "  Read 2:    19,298,269 bp\n",
      "Total written (filtered):     36,067,654 bp (91.3%)\n",
      "  Read 1:    18,408,288 bp\n",
      "  Read 2:    17,659,366 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 68366 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t536\t0.0\t2\t0 521 15\n",
      "26\t67446\t0.0\t2\t65815 1606 25\n",
      "27\t255\t0.0\t2\t0 245 10\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63077 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t1257\t0.0\t2\t0 1220 37\n",
      "26\t61484\t0.0\t2\t59832 1574 78\n",
      "27\t230\t0.0\t2\t0 213 17\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S054_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S054_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S054_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S054_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S054_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S054_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S054_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S054_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,038 reads  @     89.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 5.85 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,038\n",
      "  Read 1 with adapter:                  64,791 (99.6%)\n",
      "  Read 2 with adapter:                  64,780 (99.6%)\n",
      "Pairs written (passing filters):        65,035 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,286,601 bp\n",
      "  Read 1:    19,133,842 bp\n",
      "  Read 2:    19,152,759 bp\n",
      "Total written (filtered):     34,918,394 bp (91.2%)\n",
      "  Read 1:    17,448,772 bp\n",
      "  Read 2:    17,469,622 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64791 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t573\t0.0\t2\t0 560 13\n",
      "26\t63876\t0.0\t2\t62308 1537 31\n",
      "27\t242\t0.0\t2\t0 233 9\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64780 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t78\t0.0\t2\t0 0 78\n",
      "25\t1305\t0.0\t2\t0 1271 34\n",
      "26\t63117\t0.0\t2\t61647 1422 48\n",
      "27\t242\t0.0\t2\t0 233 9\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S055_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S055_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S055_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S055_109_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S055_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S055_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S055_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S055_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,827 reads  @     80.2 µs/read;   0.75 M reads/minute\n",
      "Finished in 5.28 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,827\n",
      "  Read 1 with adapter:                  65,589 (99.6%)\n",
      "  Read 2 with adapter:                  65,718 (99.8%)\n",
      "Pairs written (passing filters):        65,827 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,784,842 bp\n",
      "  Read 1:    19,367,591 bp\n",
      "  Read 2:    19,417,251 bp\n",
      "Total written (filtered):     35,371,519 bp (91.2%)\n",
      "  Read 1:    17,662,467 bp\n",
      "  Read 2:    17,709,052 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 65589 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t529\t0.0\t2\t0 518 11\n",
      "26\t64646\t0.0\t2\t63017 1605 24\n",
      "27\t273\t0.0\t2\t0 259 14\n",
      "28\t87\t0.0\t2\t0 0 87\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 65718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t797\t0.0\t2\t0 763 34\n",
      "26\t64468\t0.0\t2\t62662 1754 52\n",
      "27\t358\t0.0\t2\t0 344 14\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S056_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S056_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S056_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S056_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S056_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S056_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S056_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S056_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:06        75,333 reads  @     85.2 µs/read;   0.70 M reads/minute\n",
      "Finished in 6.42 s (85 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,333\n",
      "  Read 1 with adapter:                  74,991 (99.5%)\n",
      "  Read 2 with adapter:                  75,216 (99.8%)\n",
      "Pairs written (passing filters):        75,333 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,364,372 bp\n",
      "  Read 1:    22,142,751 bp\n",
      "  Read 2:    22,221,621 bp\n",
      "Total written (filtered):     40,459,820 bp (91.2%)\n",
      "  Read 1:    20,193,212 bp\n",
      "  Read 2:    20,266,608 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t654\t0.0\t2\t0 631 23\n",
      "26\t73877\t0.0\t2\t71919 1924 34\n",
      "27\t301\t0.0\t2\t0 292 9\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 75216 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t78\t0.0\t2\t0 0 78\n",
      "25\t841\t0.0\t2\t0 811 30\n",
      "26\t73951\t0.0\t2\t72187 1706 58\n",
      "27\t298\t0.0\t2\t0 292 6\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S057_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S057_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S057_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S057_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S057_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S057_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S057_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S057_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,702 reads  @     81.0 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.52 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,702\n",
      "  Read 1 with adapter:                  55,657 (99.9%)\n",
      "  Read 2 with adapter:                  55,189 (99.1%)\n",
      "Pairs written (passing filters):        55,702 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,856,311 bp\n",
      "  Read 1:    16,431,388 bp\n",
      "  Read 2:    16,424,923 bp\n",
      "Total written (filtered):     29,976,676 bp (91.2%)\n",
      "  Read 1:    14,984,313 bp\n",
      "  Read 2:    14,992,363 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t421\t0.0\t2\t0 406 15\n",
      "26\t54839\t0.0\t2\t53482 1333 24\n",
      "27\t212\t0.0\t2\t0 206 6\n",
      "28\t143\t0.0\t2\t0 0 143\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55189 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t94\t0.0\t2\t0 0 94\n",
      "25\t2482\t0.0\t2\t0 2424 58\n",
      "26\t52306\t0.0\t2\t51002 1263 41\n",
      "27\t298\t0.0\t2\t0 280 18\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S058_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S058_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S058_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S058_115_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S058_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S058_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S058_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S058_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,712 reads  @     78.1 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.04 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,712\n",
      "  Read 1 with adapter:                  51,679 (99.9%)\n",
      "  Read 2 with adapter:                  51,572 (99.7%)\n",
      "Pairs written (passing filters):        51,712 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,507,595 bp\n",
      "  Read 1:    15,254,469 bp\n",
      "  Read 2:    15,253,126 bp\n",
      "Total written (filtered):     27,823,840 bp (91.2%)\n",
      "  Read 1:    13,910,851 bp\n",
      "  Read 2:    13,912,989 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51679 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t416\t0.0\t2\t0 402 14\n",
      "26\t50861\t0.0\t2\t49790 1058 13\n",
      "27\t264\t0.0\t2\t0 257 7\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51572 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t903\t0.0\t2\t0 882 21\n",
      "26\t50394\t0.0\t2\t49159 1184 51\n",
      "27\t226\t0.0\t2\t0 217 9\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S059_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S059_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S059_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S059_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S059_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S059_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S059_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S059_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,424 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.92 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,424\n",
      "  Read 1 with adapter:                  54,391 (99.9%)\n",
      "  Read 2 with adapter:                  54,321 (99.8%)\n",
      "Pairs written (passing filters):        54,424 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,108,571 bp\n",
      "  Read 1:    16,054,406 bp\n",
      "  Read 2:    16,054,165 bp\n",
      "Total written (filtered):     29,283,261 bp (91.2%)\n",
      "  Read 1:    14,640,182 bp\n",
      "  Read 2:    14,643,079 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54391 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t392\t0.0\t2\t0 378 14\n",
      "26\t53601\t0.0\t2\t52570 1013 18\n",
      "27\t238\t0.0\t2\t0 231 7\n",
      "28\t133\t0.0\t2\t0 0 133\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54321 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t1371\t0.0\t2\t0 1332 39\n",
      "26\t52664\t0.0\t2\t51388 1226 50\n",
      "27\t221\t0.0\t2\t0 209 12\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S060_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S060_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S060_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S060_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S060_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S060_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S060_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S060_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,063 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.13 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,063\n",
      "  Read 1 with adapter:                  43,037 (99.9%)\n",
      "  Read 2 with adapter:                  42,788 (99.4%)\n",
      "Pairs written (passing filters):        43,063 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,384,317 bp\n",
      "  Read 1:    12,703,200 bp\n",
      "  Read 2:    12,681,117 bp\n",
      "Total written (filtered):     23,153,169 bp (91.2%)\n",
      "  Read 1:    11,584,163 bp\n",
      "  Read 2:    11,569,006 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43037 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t286\t0.0\t2\t0 275 11\n",
      "26\t42438\t0.0\t2\t41562 862 14\n",
      "27\t169\t0.0\t2\t0 163 6\n",
      "28\t120\t0.0\t2\t0 0 120\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42788 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t525\t0.0\t2\t0 504 21\n",
      "26\t42033\t0.0\t2\t40867 1096 70\n",
      "27\t188\t0.0\t2\t0 174 14\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S061_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S061_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S061_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S061_121_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S061_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S061_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S061_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S061_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,251 reads  @     78.8 µs/read;   0.76 M reads/minute\n",
      "Finished in 4.51 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,251\n",
      "  Read 1 with adapter:                  57,193 (99.9%)\n",
      "  Read 2 with adapter:                  56,626 (98.9%)\n",
      "Pairs written (passing filters):        57,251 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,682,939 bp\n",
      "  Read 1:    16,883,510 bp\n",
      "  Read 2:    16,799,429 bp\n",
      "Total written (filtered):     30,724,554 bp (91.2%)\n",
      "  Read 1:    15,396,477 bp\n",
      "  Read 2:    15,328,077 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57193 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t431\t0.0\t2\t0 416 15\n",
      "26\t56356\t0.0\t2\t55044 1291 21\n",
      "27\t246\t0.0\t2\t0 242 4\n",
      "28\t130\t0.0\t2\t0 0 130\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t1066\t0.0\t2\t0 1022 44\n",
      "26\t55221\t0.0\t2\t53441 1664 116\n",
      "27\t252\t0.0\t2\t0 236 16\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S062_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S062_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S062_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S062_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S062_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S062_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S062_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S062_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        64,513 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.50 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,513\n",
      "  Read 1 with adapter:                  64,480 (99.9%)\n",
      "  Read 2 with adapter:                  63,941 (99.1%)\n",
      "Pairs written (passing filters):        64,510 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,963,887 bp\n",
      "  Read 1:    19,029,827 bp\n",
      "  Read 2:    18,934,060 bp\n",
      "Total written (filtered):     34,625,503 bp (91.2%)\n",
      "  Read 1:    17,352,794 bp\n",
      "  Read 2:    17,272,709 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64480 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t486\t0.0\t2\t0 469 17\n",
      "26\t63682\t0.0\t2\t62297 1361 24\n",
      "27\t188\t0.0\t2\t0 185 3\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63941 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t94\t0.0\t2\t0 0 94\n",
      "25\t1392\t0.0\t2\t0 1348 44\n",
      "26\t62106\t0.0\t2\t60469 1573 64\n",
      "27\t233\t0.0\t2\t0 219 14\n",
      "28\t116\t0.0\t2\t0 0 116\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S063_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S063_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S063_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S063_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S063_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S063_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S063_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S063_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        67,375 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.83 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,375\n",
      "  Read 1 with adapter:                  67,314 (99.9%)\n",
      "  Read 2 with adapter:                  67,239 (99.8%)\n",
      "Pairs written (passing filters):        67,375 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,746,356 bp\n",
      "  Read 1:    19,873,639 bp\n",
      "  Read 2:    19,872,717 bp\n",
      "Total written (filtered):     36,248,787 bp (91.2%)\n",
      "  Read 1:    18,123,807 bp\n",
      "  Read 2:    18,124,980 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67314 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t550\t0.0\t2\t0 529 21\n",
      "26\t66449\t0.0\t2\t64878 1543 28\n",
      "27\t204\t0.0\t2\t0 201 3\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t784\t0.0\t2\t0 759 25\n",
      "26\t66038\t0.0\t2\t64229 1729 80\n",
      "27\t287\t0.0\t2\t0 278 9\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S064_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S064_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S064_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S064_127_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S064_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S064_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S064_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S064_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        72,830 reads  @     68.7 µs/read;   0.87 M reads/minute\n",
      "Finished in 5.01 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,830\n",
      "  Read 1 with adapter:                  72,779 (99.9%)\n",
      "  Read 2 with adapter:                  72,724 (99.9%)\n",
      "Pairs written (passing filters):        72,830 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,967,981 bp\n",
      "  Read 1:    21,483,754 bp\n",
      "  Read 2:    21,484,227 bp\n",
      "Total written (filtered):     39,185,741 bp (91.2%)\n",
      "  Read 1:    19,591,837 bp\n",
      "  Read 2:    19,593,904 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 72779 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t657\t0.0\t2\t0 632 25\n",
      "26\t71747\t0.0\t2\t69830 1886 31\n",
      "27\t226\t0.0\t2\t0 220 6\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72724 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t835\t0.0\t2\t0 804 31\n",
      "26\t71428\t0.0\t2\t69557 1799 72\n",
      "27\t284\t0.0\t2\t0 273 11\n",
      "28\t101\t0.0\t2\t0 0 101\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S065_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S065_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S065_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S065_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S065_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S065_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S065_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S065_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,841 reads  @     69.6 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.82 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,841\n",
      "  Read 1 with adapter:                  54,406 (99.2%)\n",
      "  Read 2 with adapter:                  54,293 (99.0%)\n",
      "Pairs written (passing filters):        54,841 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,285,776 bp\n",
      "  Read 1:    16,115,741 bp\n",
      "  Read 2:    16,170,035 bp\n",
      "Total written (filtered):     29,462,108 bp (91.3%)\n",
      "  Read 1:    14,701,361 bp\n",
      "  Read 2:    14,760,747 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54406 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t468\t0.0\t2\t0 445 23\n",
      "26\t53590\t0.0\t2\t52034 1525 31\n",
      "27\t216\t0.0\t2\t0 211 5\n",
      "28\t85\t0.0\t2\t0 0 85\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54293 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t98\t0.0\t2\t0 0 98\n",
      "25\t2548\t0.0\t2\t0 2463 85\n",
      "26\t51328\t0.0\t2\t49909 1369 50\n",
      "27\t224\t0.0\t2\t0 213 11\n",
      "28\t95\t0.0\t2\t0 0 95\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S066_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S066_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S066_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S066_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S066_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S066_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S066_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S066_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S067_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S067_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S067_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S067_133_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S067_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S067_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S067_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S067_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,890 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 4.42 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,890\n",
      "  Read 1 with adapter:                  59,783 (99.8%)\n",
      "  Read 2 with adapter:                  59,772 (99.8%)\n",
      "Pairs written (passing filters):        59,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,323,806 bp\n",
      "  Read 1:    17,657,191 bp\n",
      "  Read 2:    17,666,615 bp\n",
      "Total written (filtered):     32,216,738 bp (91.2%)\n",
      "  Read 1:    16,102,883 bp\n",
      "  Read 2:    16,113,855 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59783 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t443\t0.0\t2\t0 431 12\n",
      "26\t58928\t0.0\t2\t57542 1363 23\n",
      "27\t247\t0.0\t2\t0 237 10\n",
      "28\t119\t0.0\t2\t0 0 119\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59772 times\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t68\t0.0\t2\t0 0 68\n",
      "25\t1507\t0.0\t2\t0 1469 38\n",
      "26\t57887\t0.0\t2\t56631 1212 44\n",
      "27\t289\t0.0\t2\t0 280 9\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S068_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S068_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S068_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S068_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S068_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S068_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S068_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S068_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,854 reads  @     77.4 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.17 s (77 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,854\n",
      "  Read 1 with adapter:                  40,774 (99.8%)\n",
      "  Read 2 with adapter:                  40,511 (99.2%)\n",
      "Pairs written (passing filters):        40,854 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,054,421 bp\n",
      "  Read 1:    12,044,084 bp\n",
      "  Read 2:    12,010,337 bp\n",
      "Total written (filtered):     21,941,403 bp (91.2%)\n",
      "  Read 1:    10,983,950 bp\n",
      "  Read 2:    10,957,453 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40774 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t293\t0.0\t2\t0 287 6\n",
      "26\t40203\t0.0\t2\t39257 921 25\n",
      "27\t165\t0.0\t2\t0 162 3\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40511 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t532\t0.0\t2\t0 525 7\n",
      "26\t39766\t0.0\t2\t38708 987 71\n",
      "27\t168\t0.0\t2\t0 160 8\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S069_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S069_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S069_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S069_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S069_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S069_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S069_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S069_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,083 reads  @     77.4 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.26 s (77 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,083\n",
      "  Read 1 with adapter:                  54,990 (99.8%)\n",
      "  Read 2 with adapter:                  54,033 (98.1%)\n",
      "Pairs written (passing filters):        55,083 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,324,159 bp\n",
      "  Read 1:    16,239,479 bp\n",
      "  Read 2:    16,084,680 bp\n",
      "Total written (filtered):     29,490,481 bp (91.2%)\n",
      "  Read 1:    14,809,757 bp\n",
      "  Read 2:    14,680,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54990 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t402\t0.0\t2\t0 393 9\n",
      "26\t54207\t0.0\t2\t52938 1246 23\n",
      "27\t230\t0.0\t2\t0 226 4\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t1036\t0.0\t2\t0 1006 30\n",
      "26\t52680\t0.0\t2\t51171 1427 82\n",
      "27\t216\t0.0\t2\t0 201 15\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S070_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S070_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S070_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S070_139_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S070_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S070_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S070_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S070_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        68,046 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 5.02 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             68,046\n",
      "  Read 1 with adapter:                  67,942 (99.8%)\n",
      "  Read 2 with adapter:                  67,838 (99.7%)\n",
      "Pairs written (passing filters):        68,046 (100.0%)\n",
      "\n",
      "Total basepairs processed:    40,120,745 bp\n",
      "  Read 1:    20,063,966 bp\n",
      "  Read 2:    20,056,779 bp\n",
      "Total written (filtered):     36,591,736 bp (91.2%)\n",
      "  Read 1:    18,297,531 bp\n",
      "  Read 2:    18,294,205 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67942 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t536\t0.0\t2\t0 525 11\n",
      "26\t66922\t0.0\t2\t65412 1482 28\n",
      "27\t281\t0.0\t2\t0 272 9\n",
      "28\t151\t0.0\t2\t0 0 151\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 67838 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t100\t0.0\t2\t0 0 100\n",
      "25\t1391\t0.0\t2\t0 1351 40\n",
      "26\t66000\t0.0\t2\t64347 1595 58\n",
      "27\t317\t0.0\t2\t0 304 13\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S071_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S071_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S071_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S071_141_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S071_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S071_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S071_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S071_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        82,493 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 6.01 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             82,493\n",
      "  Read 1 with adapter:                  82,252 (99.7%)\n",
      "  Read 2 with adapter:                  82,327 (99.8%)\n",
      "Pairs written (passing filters):        82,493 (100.0%)\n",
      "\n",
      "Total basepairs processed:    48,639,002 bp\n",
      "  Read 1:    24,308,066 bp\n",
      "  Read 2:    24,330,936 bp\n",
      "Total written (filtered):     44,360,743 bp (91.2%)\n",
      "  Read 1:    22,169,711 bp\n",
      "  Read 2:    22,191,032 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 82252 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t688\t0.0\t2\t0 666 22\n",
      "26\t81051\t0.0\t2\t79065 1952 34\n",
      "27\t323\t0.0\t2\t0 315 8\n",
      "28\t137\t0.0\t2\t0 0 137\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 82327 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t69\t0.0\t2\t0 0 69\n",
      "25\t923\t0.0\t2\t0 893 30\n",
      "26\t80915\t0.0\t2\t78805 2026 84\n",
      "27\t377\t0.0\t2\t0 361 16\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S072_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S072_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S072_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S072_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S072_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S072_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S072_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S072_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8<-----] 00:00:14       204,052 reads  @     71.4 µs/read;   0.84 M reads/minute\n",
      "Finished in 14.58 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            204,052\n",
      "  Read 1 with adapter:                 202,937 (99.5%)\n",
      "  Read 2 with adapter:                 203,762 (99.9%)\n",
      "Pairs written (passing filters):       204,052 (100.0%)\n",
      "\n",
      "Total basepairs processed:   120,238,092 bp\n",
      "  Read 1:    60,044,742 bp\n",
      "  Read 2:    60,193,350 bp\n",
      "Total written (filtered):    109,666,113 bp (91.2%)\n",
      "  Read 1:    54,769,253 bp\n",
      "  Read 2:    54,896,860 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 202937 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t117\t0.0\t2\t0 0 117\n",
      "25\t1740\t0.0\t2\t0 1683 57\n",
      "26\t200188\t0.0\t2\t194922 5179 87\n",
      "27\t683\t0.0\t2\t0 664 19\n",
      "28\t209\t0.0\t2\t0 0 209\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 203762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t177\t0.0\t2\t0 0 177\n",
      "25\t2208\t0.0\t2\t0 2144 64\n",
      "26\t200322\t0.0\t2\t195518 4663 141\n",
      "27\t870\t0.0\t2\t0 838 32\n",
      "28\t185\t0.0\t2\t0 0 185\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S073_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S073_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S073_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S073_145_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S073_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S073_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S073_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S073_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,666 reads  @     81.4 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.05 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,666\n",
      "  Read 1 with adapter:                  49,607 (99.9%)\n",
      "  Read 2 with adapter:                  49,266 (99.2%)\n",
      "Pairs written (passing filters):        49,666 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,292,850 bp\n",
      "  Read 1:    14,647,431 bp\n",
      "  Read 2:    14,645,419 bp\n",
      "Total written (filtered):     26,724,677 bp (91.2%)\n",
      "  Read 1:    13,358,106 bp\n",
      "  Read 2:    13,366,571 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t565\t0.0\t2\t0 544 21\n",
      "26\t48858\t0.0\t2\t47400 1434 24\n",
      "27\t120\t0.0\t2\t0 114 6\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49266 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t87\t0.0\t2\t0 0 87\n",
      "25\t2214\t0.0\t2\t0 2159 55\n",
      "26\t46692\t0.0\t2\t45603 1053 36\n",
      "27\t226\t0.0\t2\t0 217 9\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S074_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S074_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S074_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S074_147_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S074_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S074_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S074_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S074_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        60,305 reads  @     72.0 µs/read;   0.83 M reads/minute\n",
      "Finished in 4.35 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             60,305\n",
      "  Read 1 with adapter:                  60,230 (99.9%)\n",
      "  Read 2 with adapter:                  60,215 (99.9%)\n",
      "Pairs written (passing filters):        60,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,575,654 bp\n",
      "  Read 1:    17,787,359 bp\n",
      "  Read 2:    17,788,295 bp\n",
      "Total written (filtered):     32,445,583 bp (91.2%)\n",
      "  Read 1:    16,221,955 bp\n",
      "  Read 2:    16,223,628 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60230 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t731\t0.0\t2\t0 703 28\n",
      "26\t59249\t0.0\t2\t57679 1543 27\n",
      "27\t153\t0.0\t2\t0 148 5\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t1157\t0.0\t2\t0 1139 18\n",
      "26\t58711\t0.0\t2\t57327 1337 47\n",
      "27\t284\t0.0\t2\t0 274 10\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S075_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S075_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S075_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S075_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S075_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S075_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S075_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S075_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        61,695 reads  @     70.1 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.33 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,695\n",
      "  Read 1 with adapter:                  61,632 (99.9%)\n",
      "  Read 2 with adapter:                  61,597 (99.8%)\n",
      "Pairs written (passing filters):        61,695 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,395,627 bp\n",
      "  Read 1:    18,198,071 bp\n",
      "  Read 2:    18,197,556 bp\n",
      "Total written (filtered):     33,193,542 bp (91.2%)\n",
      "  Read 1:    16,596,129 bp\n",
      "  Read 2:    16,597,413 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61632 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t713\t0.0\t2\t0 696 17\n",
      "26\t60657\t0.0\t2\t59017 1608 32\n",
      "27\t145\t0.0\t2\t0 138 7\n",
      "28\t78\t0.0\t2\t0 0 78\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61597 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t1591\t0.0\t2\t0 1560 31\n",
      "26\t59635\t0.0\t2\t58200 1370 65\n",
      "27\t294\t0.0\t2\t0 279 15\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S076_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S076_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S076_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S076_151_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S076_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S076_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S076_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S076_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,215 reads  @     75.3 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.16 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,215\n",
      "  Read 1 with adapter:                  55,149 (99.9%)\n",
      "  Read 2 with adapter:                  54,783 (99.2%)\n",
      "Pairs written (passing filters):        55,215 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,535,638 bp\n",
      "  Read 1:    16,287,003 bp\n",
      "  Read 2:    16,248,635 bp\n",
      "Total written (filtered):     29,678,406 bp (91.2%)\n",
      "  Read 1:    14,853,581 bp\n",
      "  Read 2:    14,824,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55149 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t647\t0.0\t2\t0 625 22\n",
      "26\t54270\t0.0\t2\t52907 1329 34\n",
      "27\t145\t0.0\t2\t0 141 4\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54783 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t731\t0.0\t2\t0 699 32\n",
      "26\t53771\t0.0\t2\t52181 1467 123\n",
      "27\t227\t0.0\t2\t0 214 13\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S077_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S077_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S077_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S077_153_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S077_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S077_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S077_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S077_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        61,305 reads  @     82.9 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.08 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,305\n",
      "  Read 1 with adapter:                  61,243 (99.9%)\n",
      "  Read 2 with adapter:                  60,243 (98.3%)\n",
      "Pairs written (passing filters):        61,305 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,002,358 bp\n",
      "  Read 1:    18,082,846 bp\n",
      "  Read 2:    17,919,512 bp\n",
      "Total written (filtered):     32,845,229 bp (91.2%)\n",
      "  Read 1:    16,491,000 bp\n",
      "  Read 2:    16,354,229 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61243 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t716\t0.0\t2\t0 699 17\n",
      "26\t60217\t0.0\t2\t58615 1577 25\n",
      "27\t188\t0.0\t2\t0 173 15\n",
      "28\t75\t0.0\t2\t0 0 75\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60243 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t1198\t0.0\t2\t0 1150 48\n",
      "26\t58725\t0.0\t2\t56772 1814 139\n",
      "27\t245\t0.0\t2\t0 227 18\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S078_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S078_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S078_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S078_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S078_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S078_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S078_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S078_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        33,289 reads  @     79.1 µs/read;   0.76 M reads/minute\n",
      "Finished in 2.63 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             33,289\n",
      "  Read 1 with adapter:                  33,255 (99.9%)\n",
      "  Read 2 with adapter:                  33,157 (99.6%)\n",
      "Pairs written (passing filters):        33,289 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,625,793 bp\n",
      "  Read 1:     9,819,894 bp\n",
      "  Read 2:     9,805,899 bp\n",
      "Total written (filtered):     17,899,997 bp (91.2%)\n",
      "  Read 1:     8,955,524 bp\n",
      "  Read 2:     8,944,473 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 33255 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t394\t0.0\t2\t0 384 10\n",
      "26\t32700\t0.0\t2\t31767 922 11\n",
      "27\t104\t0.0\t2\t0 101 3\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 33157 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t760\t0.0\t2\t0 730 30\n",
      "26\t32181\t0.0\t2\t31277 870 34\n",
      "27\t160\t0.0\t2\t0 151 9\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S079_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S079_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S079_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S079_157_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S079_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S079_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S079_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S079_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:06        84,683 reads  @     77.1 µs/read;   0.78 M reads/minute\n",
      "Finished in 6.53 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,683\n",
      "  Read 1 with adapter:                  84,601 (99.9%)\n",
      "  Read 2 with adapter:                  84,502 (99.8%)\n",
      "Pairs written (passing filters):        84,683 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,950,367 bp\n",
      "  Read 1:    24,977,387 bp\n",
      "  Read 2:    24,972,980 bp\n",
      "Total written (filtered):     45,555,092 bp (91.2%)\n",
      "  Read 1:    22,778,528 bp\n",
      "  Read 2:    22,776,564 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 84601 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t73\t0.0\t2\t0 0 73\n",
      "25\t1032\t0.0\t2\t0 999 33\n",
      "26\t83165\t0.0\t2\t80809 2309 47\n",
      "27\t251\t0.0\t2\t0 241 10\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 84502 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t981\t0.0\t2\t0 948 33\n",
      "26\t82991\t0.0\t2\t80713 2187 91\n",
      "27\t435\t0.0\t2\t0 417 18\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S081_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S081_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S081_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S081_159_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S081_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S081_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S081_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S081_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00           465 reads  @    182.7 µs/read;   0.33 M reads/minute\n",
      "Finished in 0.09 s (188 us/read; 0.32 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                465\n",
      "  Read 1 with adapter:                     453 (97.4%)\n",
      "  Read 2 with adapter:                     450 (96.8%)\n",
      "Pairs written (passing filters):           465 (100.0%)\n",
      "\n",
      "Total basepairs processed:       273,930 bp\n",
      "  Read 1:       136,997 bp\n",
      "  Read 2:       136,933 bp\n",
      "Total written (filtered):        250,487 bp (91.4%)\n",
      "  Read 1:       125,230 bp\n",
      "  Read 2:       125,257 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 453 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t11\t0.0\t2\t0 11\n",
      "26\t440\t0.0\t2\t422 17 1\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 450 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t28\t0.0\t2\t0 28\n",
      "26\t419\t0.0\t2\t407 12\n",
      "27\t2\t0.0\t2\t0 2\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S084_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S084_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S084_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S084_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S084_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S084_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S084_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S084_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,080 reads  @     84.8 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.33 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,080\n",
      "  Read 1 with adapter:                  51,038 (99.9%)\n",
      "  Read 2 with adapter:                  51,003 (99.8%)\n",
      "Pairs written (passing filters):        51,080 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,135,840 bp\n",
      "  Read 1:    15,068,330 bp\n",
      "  Read 2:    15,067,510 bp\n",
      "Total written (filtered):     27,483,005 bp (91.2%)\n",
      "  Read 1:    13,741,243 bp\n",
      "  Read 2:    13,741,762 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51038 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t472\t0.0\t2\t0 456 16\n",
      "26\t50032\t0.0\t2\t48921 1095 16\n",
      "27\t333\t0.0\t2\t0 324 9\n",
      "28\t160\t0.0\t2\t0 0 160\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51003 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t494\t0.0\t2\t0 478 16\n",
      "26\t50251\t0.0\t2\t49052 1140 59\n",
      "27\t184\t0.0\t2\t0 180 4\n",
      "28\t32\t0.0\t2\t0 0 32\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S085_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S085_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S085_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S085_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S085_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S085_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S085_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S085_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        64,589 reads  @     89.7 µs/read;   0.67 M reads/minute\n",
      "Finished in 5.80 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,589\n",
      "  Read 1 with adapter:                  64,542 (99.9%)\n",
      "  Read 2 with adapter:                  64,430 (99.8%)\n",
      "Pairs written (passing filters):        64,589 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,101,632 bp\n",
      "  Read 1:    19,053,250 bp\n",
      "  Read 2:    19,048,382 bp\n",
      "Total written (filtered):     34,748,734 bp (91.2%)\n",
      "  Read 1:    17,375,085 bp\n",
      "  Read 2:    17,373,649 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64542 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t606\t0.0\t2\t0 595 11\n",
      "26\t63284\t0.0\t2\t61908 1347 29\n",
      "27\t389\t0.0\t2\t0 379 10\n",
      "28\t204\t0.0\t2\t0 0 204\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64430 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t637\t0.0\t2\t0 615 22\n",
      "26\t63495\t0.0\t2\t61955 1479 61\n",
      "27\t250\t0.0\t2\t0 235 15\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S086_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S086_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S086_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S086_165_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S086_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S086_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S086_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S086_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:11        61,149 reads  @    196.0 µs/read;   0.31 M reads/minute\n",
      "Finished in 11.99 s (196 us/read; 0.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,149\n",
      "  Read 1 with adapter:                  61,090 (99.9%)\n",
      "  Read 2 with adapter:                  61,000 (99.8%)\n",
      "Pairs written (passing filters):        61,149 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,068,555 bp\n",
      "  Read 1:    18,037,969 bp\n",
      "  Read 2:    18,030,586 bp\n",
      "Total written (filtered):     32,894,600 bp (91.2%)\n",
      "  Read 1:    16,449,654 bp\n",
      "  Read 2:    16,444,946 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61090 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t643\t0.0\t2\t0 629 14\n",
      "26\t59866\t0.0\t2\t58608 1226 32\n",
      "27\t324\t0.0\t2\t0 311 13\n",
      "28\t202\t0.0\t2\t0 0 202\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61000 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t587\t0.0\t2\t0 564 23\n",
      "26\t60069\t0.0\t2\t58421 1567 81\n",
      "27\t277\t0.0\t2\t0 261 16\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S087_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S087_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S087_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S087_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S087_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S087_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S087_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S087_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:11        67,304 reads  @    173.6 µs/read;   0.35 M reads/minute\n",
      "Finished in 11.69 s (174 us/read; 0.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             67,304\n",
      "  Read 1 with adapter:                  67,239 (99.9%)\n",
      "  Read 2 with adapter:                  66,757 (99.2%)\n",
      "Pairs written (passing filters):        67,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,643,732 bp\n",
      "  Read 1:    19,853,331 bp\n",
      "  Read 2:    19,790,401 bp\n",
      "Total written (filtered):     36,160,674 bp (91.2%)\n",
      "  Read 1:    18,105,532 bp\n",
      "  Read 2:    18,055,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 67239 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t830\t0.0\t2\t0 814 16\n",
      "26\t65919\t0.0\t2\t64382 1516 21\n",
      "27\t345\t0.0\t2\t0 337 8\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66757 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t635\t0.0\t2\t0 612 23\n",
      "26\t65802\t0.0\t2\t64210 1547 45\n",
      "27\t264\t0.0\t2\t0 257 7\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S088_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S088_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S088_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S088_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S088_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S088_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S088_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S088_169_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:10        58,816 reads  @    181.6 µs/read;   0.33 M reads/minute\n",
      "Finished in 10.68 s (182 us/read; 0.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,816\n",
      "  Read 1 with adapter:                  58,761 (99.9%)\n",
      "  Read 2 with adapter:                  58,727 (99.8%)\n",
      "Pairs written (passing filters):        58,816 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,694,597 bp\n",
      "  Read 1:    17,350,231 bp\n",
      "  Read 2:    17,344,366 bp\n",
      "Total written (filtered):     31,640,677 bp (91.2%)\n",
      "  Read 1:    15,822,803 bp\n",
      "  Read 2:    15,817,874 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58761 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t572\t0.0\t2\t0 554 18\n",
      "26\t57910\t0.0\t2\t56405 1474 31\n",
      "27\t180\t0.0\t2\t0 170 10\n",
      "28\t58\t0.0\t2\t0 0 58\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58727 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t623\t0.0\t2\t0 605 18\n",
      "26\t57753\t0.0\t2\t56413 1299 41\n",
      "27\t245\t0.0\t2\t0 236 9\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S089_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S089_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S089_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S089_171_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S089_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S089_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S089_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S089_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10        62,219 reads  @    168.3 µs/read;   0.36 M reads/minute\n",
      "Finished in 10.48 s (168 us/read; 0.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,219\n",
      "  Read 1 with adapter:                  62,174 (99.9%)\n",
      "  Read 2 with adapter:                  62,126 (99.9%)\n",
      "Pairs written (passing filters):        62,219 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,705,203 bp\n",
      "  Read 1:    18,354,004 bp\n",
      "  Read 2:    18,351,199 bp\n",
      "Total written (filtered):     33,474,233 bp (91.2%)\n",
      "  Read 1:    16,737,911 bp\n",
      "  Read 2:    16,736,322 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62174 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t605\t0.0\t2\t0 585 20\n",
      "26\t61283\t0.0\t2\t59779 1482 22\n",
      "27\t194\t0.0\t2\t0 188 6\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62126 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t654\t0.0\t2\t0 632 22\n",
      "26\t61124\t0.0\t2\t59605 1457 62\n",
      "27\t273\t0.0\t2\t0 259 14\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S090_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S090_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S090_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S090_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S090_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S090_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S090_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S090_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:13        74,513 reads  @    184.2 µs/read;   0.33 M reads/minute\n",
      "Finished in 13.74 s (184 us/read; 0.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,513\n",
      "  Read 1 with adapter:                  74,426 (99.9%)\n",
      "  Read 2 with adapter:                  74,416 (99.9%)\n",
      "Pairs written (passing filters):        74,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,958,439 bp\n",
      "  Read 1:    21,980,649 bp\n",
      "  Read 2:    21,977,790 bp\n",
      "Total written (filtered):     40,089,255 bp (91.2%)\n",
      "  Read 1:    20,045,873 bp\n",
      "  Read 2:    20,043,382 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74426 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t734\t0.0\t2\t0 709 25\n",
      "26\t73233\t0.0\t2\t71421 1788 24\n",
      "27\t280\t0.0\t2\t0 273 7\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 74416 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t708\t0.0\t2\t0 680 28\n",
      "26\t73266\t0.0\t2\t71406 1807 53\n",
      "27\t332\t0.0\t2\t0 320 12\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S091_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S091_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S091_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S091_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S091_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S091_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S091_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S091_175_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:11        66,840 reads  @    177.6 µs/read;   0.34 M reads/minute\n",
      "Finished in 11.87 s (178 us/read; 0.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,840\n",
      "  Read 1 with adapter:                  66,785 (99.9%)\n",
      "  Read 2 with adapter:                  66,667 (99.7%)\n",
      "Pairs written (passing filters):        66,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,433,803 bp\n",
      "  Read 1:    19,717,639 bp\n",
      "  Read 2:    19,716,164 bp\n",
      "Total written (filtered):     35,964,584 bp (91.2%)\n",
      "  Read 1:    17,981,379 bp\n",
      "  Read 2:    17,983,205 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66785 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t625\t0.0\t2\t0 608 17\n",
      "26\t65660\t0.0\t2\t64046 1587 27\n",
      "27\t313\t0.0\t2\t0 300 13\n",
      "28\t134\t0.0\t2\t0 0 134\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66667 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t646\t0.0\t2\t0 626 20\n",
      "26\t65656\t0.0\t2\t63921 1665 70\n",
      "27\t263\t0.0\t2\t0 248 15\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S092_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S092_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S092_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S092_177_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S092_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S092_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S092_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S092_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        52,811 reads  @    142.3 µs/read;   0.42 M reads/minute\n",
      "Finished in 7.52 s (142 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,811\n",
      "  Read 1 with adapter:                  52,723 (99.8%)\n",
      "  Read 2 with adapter:                  52,729 (99.8%)\n",
      "Pairs written (passing filters):        52,811 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,153,521 bp\n",
      "  Read 1:    15,575,586 bp\n",
      "  Read 2:    15,577,935 bp\n",
      "Total written (filtered):     28,412,072 bp (91.2%)\n",
      "  Read 1:    14,204,749 bp\n",
      "  Read 2:    14,207,323 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52723 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t424\t0.0\t2\t0 406 18\n",
      "26\t51860\t0.0\t2\t50632 1208 20\n",
      "27\t259\t0.0\t2\t0 252 7\n",
      "28\t141\t0.0\t2\t0 0 141\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52729 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t544\t0.0\t2\t0 509 35\n",
      "26\t51885\t0.0\t2\t50588 1226 71\n",
      "27\t174\t0.0\t2\t0 165 9\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S093_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S093_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S093_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S093_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S093_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S093_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S093_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S093_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:05        45,877 reads  @    112.6 µs/read;   0.53 M reads/minute\n",
      "Finished in 5.17 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,877\n",
      "  Read 1 with adapter:                  45,768 (99.8%)\n",
      "  Read 2 with adapter:                  45,794 (99.8%)\n",
      "Pairs written (passing filters):        45,877 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,059,715 bp\n",
      "  Read 1:    13,528,658 bp\n",
      "  Read 2:    13,531,057 bp\n",
      "Total written (filtered):     24,679,426 bp (91.2%)\n",
      "  Read 1:    12,338,657 bp\n",
      "  Read 2:    12,340,769 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45768 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t356\t0.0\t2\t0 348 8\n",
      "26\t45054\t0.0\t2\t44061 973 20\n",
      "27\t183\t0.0\t2\t0 179 4\n",
      "28\t139\t0.0\t2\t0 0 139\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45794 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t483\t0.0\t2\t0 462 21\n",
      "26\t45095\t0.0\t2\t44008 1034 53\n",
      "27\t177\t0.0\t2\t0 166 11\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S094_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S094_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S094_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S094_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S094_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S094_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S094_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S094_181_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,222 reads  @     89.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.61 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,222\n",
      "  Read 1 with adapter:                  51,017 (99.6%)\n",
      "  Read 2 with adapter:                  51,139 (99.8%)\n",
      "Pairs written (passing filters):        51,222 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,208,156 bp\n",
      "  Read 1:    15,101,548 bp\n",
      "  Read 2:    15,106,608 bp\n",
      "Total written (filtered):     27,552,302 bp (91.2%)\n",
      "  Read 1:    13,775,010 bp\n",
      "  Read 2:    13,777,292 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t408\t0.0\t2\t0 391 17\n",
      "26\t50134\t0.0\t2\t48995 1115 24\n",
      "27\t266\t0.0\t2\t0 262 4\n",
      "28\t164\t0.0\t2\t0 0 164\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t452\t0.0\t2\t0 437 15\n",
      "26\t50403\t0.0\t2\t49248 1118 37\n",
      "27\t230\t0.0\t2\t0 219 11\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S095_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S095_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S095_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S095_183_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S095_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S095_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S095_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S095_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        52,396 reads  @     99.3 µs/read;   0.60 M reads/minute\n",
      "Finished in 5.21 s (99 us/read; 0.60 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,396\n",
      "  Read 1 with adapter:                  52,306 (99.8%)\n",
      "  Read 2 with adapter:                  52,291 (99.8%)\n",
      "Pairs written (passing filters):        52,396 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,908,476 bp\n",
      "  Read 1:    15,453,445 bp\n",
      "  Read 2:    15,455,031 bp\n",
      "Total written (filtered):     28,189,284 bp (91.2%)\n",
      "  Read 1:    14,093,529 bp\n",
      "  Read 2:    14,095,755 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52306 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t418\t0.0\t2\t0 400 18\n",
      "26\t51523\t0.0\t2\t50258 1243 22\n",
      "27\t200\t0.0\t2\t0 195 5\n",
      "28\t127\t0.0\t2\t0 0 127\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52291 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t489\t0.0\t2\t0 472 17\n",
      "26\t51557\t0.0\t2\t50296 1229 32\n",
      "27\t211\t0.0\t2\t0 201 10\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S096_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S096_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S096_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S096_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S096_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S096_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S096_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S096_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10        69,663 reads  @    155.0 µs/read;   0.39 M reads/minute\n",
      "Finished in 10.80 s (155 us/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,663\n",
      "  Read 1 with adapter:                  69,610 (99.9%)\n",
      "  Read 2 with adapter:                  69,558 (99.8%)\n",
      "Pairs written (passing filters):        69,663 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,095,502 bp\n",
      "  Read 1:    20,549,151 bp\n",
      "  Read 2:    20,546,351 bp\n",
      "Total written (filtered):     37,477,860 bp (91.2%)\n",
      "  Read 1:    18,739,629 bp\n",
      "  Read 2:    18,738,231 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 69610 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t623\t0.0\t2\t0 605 18\n",
      "26\t68605\t0.0\t2\t66823 1760 22\n",
      "27\t251\t0.0\t2\t0 242 9\n",
      "28\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 69558 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t656\t0.0\t2\t0 634 22\n",
      "26\t68515\t0.0\t2\t66821 1637 57\n",
      "27\t306\t0.0\t2\t0 295 11\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S097_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S097_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S097_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S097_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S097_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S097_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S097_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S097_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10        61,246 reads  @    174.9 µs/read;   0.34 M reads/minute\n",
      "Finished in 10.72 s (175 us/read; 0.34 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,246\n",
      "  Read 1 with adapter:                  61,196 (99.9%)\n",
      "  Read 2 with adapter:                  61,154 (99.8%)\n",
      "Pairs written (passing filters):        61,246 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,128,125 bp\n",
      "  Read 1:    18,064,697 bp\n",
      "  Read 2:    18,063,428 bp\n",
      "Total written (filtered):     32,947,726 bp (91.2%)\n",
      "  Read 1:    16,473,918 bp\n",
      "  Read 2:    16,473,808 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61196 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t48\t0.0\t2\t0 0 48\n",
      "25\t526\t0.0\t2\t0 512 14\n",
      "26\t60368\t0.0\t2\t58954 1390 24\n",
      "27\t203\t0.0\t2\t0 199 4\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61154 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t618\t0.0\t2\t0 595 23\n",
      "26\t60218\t0.0\t2\t58682 1479 57\n",
      "27\t262\t0.0\t2\t0 253 9\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S098_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S098_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S098_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S098_189_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S098_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S098_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S098_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S098_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8=------] 00:00:12        92,211 reads  @    138.7 µs/read;   0.43 M reads/minute\n",
      "Finished in 12.79 s (139 us/read; 0.43 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             92,211\n",
      "  Read 1 with adapter:                  92,138 (99.9%)\n",
      "  Read 2 with adapter:                  92,047 (99.8%)\n",
      "Pairs written (passing filters):        92,211 (100.0%)\n",
      "\n",
      "Total basepairs processed:    54,397,490 bp\n",
      "  Read 1:    27,200,427 bp\n",
      "  Read 2:    27,197,063 bp\n",
      "Total written (filtered):     49,609,379 bp (91.2%)\n",
      "  Read 1:    24,805,155 bp\n",
      "  Read 2:    24,804,224 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 92138 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t784\t0.0\t2\t0 754 30\n",
      "26\t90775\t0.0\t2\t88018 2693 64\n",
      "27\t386\t0.0\t2\t0 373 13\n",
      "28\t117\t0.0\t2\t0 0 117\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 92047 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t939\t0.0\t2\t0 896 43\n",
      "26\t90461\t0.0\t2\t87678 2678 105\n",
      "27\t470\t0.0\t2\t0 441 29\n",
      "28\t110\t0.0\t2\t0 0 110\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S099_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S099_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S099_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S099_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S099_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S099_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S099_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S099_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11        75,259 reads  @    149.1 µs/read;   0.40 M reads/minute\n",
      "Finished in 11.22 s (149 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             75,259\n",
      "  Read 1 with adapter:                  75,209 (99.9%)\n",
      "  Read 2 with adapter:                  75,102 (99.8%)\n",
      "Pairs written (passing filters):        75,259 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,400,580 bp\n",
      "  Read 1:    22,200,841 bp\n",
      "  Read 2:    22,199,739 bp\n",
      "Total written (filtered):     40,493,093 bp (91.2%)\n",
      "  Read 1:    20,245,710 bp\n",
      "  Read 2:    20,247,383 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 75209 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t622\t0.0\t2\t0 599 23\n",
      "26\t74192\t0.0\t2\t72398 1759 35\n",
      "27\t275\t0.0\t2\t0 267 8\n",
      "28\t71\t0.0\t2\t0 0 71\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 75102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t657\t0.0\t2\t0 635 22\n",
      "26\t74002\t0.0\t2\t72114 1807 81\n",
      "27\t337\t0.0\t2\t0 314 23\n",
      "28\t59\t0.0\t2\t0 0 59\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S100_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S100_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S100_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S100_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S100_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S100_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S100_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S100_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        54,315 reads  @    143.5 µs/read;   0.42 M reads/minute\n",
      "Finished in 7.80 s (144 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,315\n",
      "  Read 1 with adapter:                  54,263 (99.9%)\n",
      "  Read 2 with adapter:                  54,210 (99.8%)\n",
      "Pairs written (passing filters):        54,315 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,044,538 bp\n",
      "  Read 1:    16,022,503 bp\n",
      "  Read 2:    16,022,035 bp\n",
      "Total written (filtered):     29,224,644 bp (91.2%)\n",
      "  Read 1:    14,611,711 bp\n",
      "  Read 2:    14,612,933 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54263 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t504\t0.0\t2\t0 490 14\n",
      "26\t53342\t0.0\t2\t51904 1413 25\n",
      "27\t260\t0.0\t2\t0 252 8\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54210 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t544\t0.0\t2\t0 514 30\n",
      "26\t53321\t0.0\t2\t51859 1393 69\n",
      "27\t272\t0.0\t2\t0 261 11\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S101_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S101_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S101_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S101_195_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S101_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S101_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S101_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S101_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:05        50,317 reads  @    118.1 µs/read;   0.51 M reads/minute\n",
      "Finished in 5.95 s (118 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,317\n",
      "  Read 1 with adapter:                  50,278 (99.9%)\n",
      "  Read 2 with adapter:                  50,203 (99.8%)\n",
      "Pairs written (passing filters):        50,317 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,683,750 bp\n",
      "  Read 1:    14,843,223 bp\n",
      "  Read 2:    14,840,527 bp\n",
      "Total written (filtered):     27,071,560 bp (91.2%)\n",
      "  Read 1:    13,535,985 bp\n",
      "  Read 2:    13,535,575 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t394\t0.0\t2\t0 381 13\n",
      "26\t49481\t0.0\t2\t48392 1068 21\n",
      "27\t270\t0.0\t2\t0 266 4\n",
      "28\t100\t0.0\t2\t0 0 100\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50203 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t511\t0.0\t2\t0 495 16\n",
      "26\t49425\t0.0\t2\t48103 1264 58\n",
      "27\t233\t0.0\t2\t0 224 9\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S102_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S102_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S102_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S102_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S102_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S102_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S102_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S102_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:08        69,938 reads  @    123.6 µs/read;   0.49 M reads/minute\n",
      "Finished in 8.65 s (124 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             69,938\n",
      "  Read 1 with adapter:                  69,884 (99.9%)\n",
      "  Read 2 with adapter:                  69,777 (99.8%)\n",
      "Pairs written (passing filters):        69,938 (100.0%)\n",
      "\n",
      "Total basepairs processed:    41,255,627 bp\n",
      "  Read 1:    20,630,569 bp\n",
      "  Read 2:    20,625,058 bp\n",
      "Total written (filtered):     37,624,713 bp (91.2%)\n",
      "  Read 1:    18,813,472 bp\n",
      "  Read 2:    18,811,241 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 69884 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t532\t0.0\t2\t0 522 10\n",
      "26\t68760\t0.0\t2\t67238 1494 28\n",
      "27\t359\t0.0\t2\t0 347 12\n",
      "28\t188\t0.0\t2\t0 0 188\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 69777 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t598\t0.0\t2\t0 577 21\n",
      "26\t68859\t0.0\t2\t67030 1728 101\n",
      "27\t275\t0.0\t2\t0 262 13\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S103_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S103_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S103_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S103_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S103_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S103_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S103_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S103_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        64,204 reads  @    121.3 µs/read;   0.49 M reads/minute\n",
      "Finished in 7.79 s (121 us/read; 0.49 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             64,204\n",
      "  Read 1 with adapter:                  64,158 (99.9%)\n",
      "  Read 2 with adapter:                  64,033 (99.7%)\n",
      "Pairs written (passing filters):        64,204 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,878,540 bp\n",
      "  Read 1:    18,939,272 bp\n",
      "  Read 2:    18,939,268 bp\n",
      "Total written (filtered):     34,545,955 bp (91.2%)\n",
      "  Read 1:    17,271,138 bp\n",
      "  Read 2:    17,274,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 64158 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t533\t0.0\t2\t0 515 18\n",
      "26\t63139\t0.0\t2\t61714 1413 12\n",
      "27\t285\t0.0\t2\t0 280 5\n",
      "28\t169\t0.0\t2\t0 0 169\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 64033 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t592\t0.0\t2\t0 564 28\n",
      "26\t63145\t0.0\t2\t61513 1575 57\n",
      "27\t251\t0.0\t2\t0 241 10\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S104_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S104_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S104_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S104_201_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S104_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S104_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S104_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S104_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:10        78,979 reads  @    135.0 µs/read;   0.44 M reads/minute\n",
      "Finished in 10.67 s (135 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             78,979\n",
      "  Read 1 with adapter:                  78,917 (99.9%)\n",
      "  Read 2 with adapter:                  78,836 (99.8%)\n",
      "Pairs written (passing filters):        78,979 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,595,053 bp\n",
      "  Read 1:    23,298,381 bp\n",
      "  Read 2:    23,296,672 bp\n",
      "Total written (filtered):     42,494,219 bp (91.2%)\n",
      "  Read 1:    21,246,789 bp\n",
      "  Read 2:    21,247,430 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 78917 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t793\t0.0\t2\t0 769 24\n",
      "26\t77573\t0.0\t2\t75861 1693 19\n",
      "27\t331\t0.0\t2\t0 325 6\n",
      "28\t163\t0.0\t2\t0 0 163\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 78836 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t789\t0.0\t2\t0 751 38\n",
      "26\t77595\t0.0\t2\t75621 1882 92\n",
      "27\t369\t0.0\t2\t0 342 27\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S105_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S105_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S105_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S105_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S105_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S105_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S105_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S105_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:07        63,455 reads  @    124.2 µs/read;   0.48 M reads/minute\n",
      "Finished in 7.89 s (124 us/read; 0.48 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             63,455\n",
      "  Read 1 with adapter:                  63,412 (99.9%)\n",
      "  Read 2 with adapter:                  63,341 (99.8%)\n",
      "Pairs written (passing filters):        63,455 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,434,407 bp\n",
      "  Read 1:    18,719,197 bp\n",
      "  Read 2:    18,715,210 bp\n",
      "Total written (filtered):     34,139,648 bp (91.2%)\n",
      "  Read 1:    17,070,931 bp\n",
      "  Read 2:    17,068,717 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 63412 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t686\t0.0\t2\t0 672 14\n",
      "26\t62413\t0.0\t2\t60944 1445 24\n",
      "27\t206\t0.0\t2\t0 200 6\n",
      "28\t62\t0.0\t2\t0 0 62\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 63341 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t681\t0.0\t2\t0 658 23\n",
      "26\t62320\t0.0\t2\t60633 1593 94\n",
      "27\t228\t0.0\t2\t0 224 4\n",
      "28\t76\t0.0\t2\t0 0 76\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S106_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S106_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S106_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S106_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S106_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S106_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S106_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S106_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09        66,956 reads  @    142.8 µs/read;   0.42 M reads/minute\n",
      "Finished in 9.56 s (143 us/read; 0.42 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,956\n",
      "  Read 1 with adapter:                  66,885 (99.9%)\n",
      "  Read 2 with adapter:                  66,839 (99.8%)\n",
      "Pairs written (passing filters):        66,956 (100.0%)\n",
      "\n",
      "Total basepairs processed:    39,500,853 bp\n",
      "  Read 1:    19,751,516 bp\n",
      "  Read 2:    19,749,337 bp\n",
      "Total written (filtered):     36,024,842 bp (91.2%)\n",
      "  Read 1:    18,013,025 bp\n",
      "  Read 2:    18,011,817 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 66885 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t732\t0.0\t2\t0 715 17\n",
      "26\t65826\t0.0\t2\t64154 1641 31\n",
      "27\t221\t0.0\t2\t0 209 12\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66839 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t669\t0.0\t2\t0 640 29\n",
      "26\t65746\t0.0\t2\t63908 1749 89\n",
      "27\t289\t0.0\t2\t0 274 15\n",
      "28\t89\t0.0\t2\t0 0 89\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S107_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S107_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S107_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S107_207_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S107_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S107_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S107_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S107_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        17,625 reads  @    144.9 µs/read;   0.41 M reads/minute\n",
      "Finished in 2.56 s (145 us/read; 0.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,625\n",
      "  Read 1 with adapter:                  17,611 (99.9%)\n",
      "  Read 2 with adapter:                  17,598 (99.8%)\n",
      "Pairs written (passing filters):        17,625 (100.0%)\n",
      "\n",
      "Total basepairs processed:    10,398,170 bp\n",
      "  Read 1:     5,199,367 bp\n",
      "  Read 2:     5,198,803 bp\n",
      "Total written (filtered):      9,482,930 bp (91.2%)\n",
      "  Read 1:     4,741,585 bp\n",
      "  Read 2:     4,741,345 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 17611 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t210\t0.0\t2\t0 204 6\n",
      "26\t17272\t0.0\t2\t16767 496 9\n",
      "27\t84\t0.0\t2\t0 80 4\n",
      "28\t28\t0.0\t2\t0 0 28\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 17598 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t197\t0.0\t2\t0 188 9\n",
      "26\t17279\t0.0\t2\t16759 498 22\n",
      "27\t81\t0.0\t2\t0 77 4\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S108_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S108_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S108_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S108_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S108_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S108_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S108_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S108_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        39,322 reads  @    116.5 µs/read;   0.52 M reads/minute\n",
      "Finished in 4.58 s (117 us/read; 0.51 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,322\n",
      "  Read 1 with adapter:                  39,017 (99.2%)\n",
      "  Read 2 with adapter:                  39,267 (99.9%)\n",
      "Pairs written (passing filters):        39,322 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,125,237 bp\n",
      "  Read 1:    11,526,151 bp\n",
      "  Read 2:    11,599,086 bp\n",
      "Total written (filtered):     21,090,243 bp (91.2%)\n",
      "  Read 1:    10,511,839 bp\n",
      "  Read 2:    10,578,404 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t347\t0.0\t2\t0 333 14\n",
      "26\t38443\t0.0\t2\t37311 1118 14\n",
      "27\t137\t0.0\t2\t0 136 1\n",
      "28\t65\t0.0\t2\t0 0 65\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39267 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t455\t0.0\t2\t0 435 20\n",
      "26\t38559\t0.0\t2\t37469 1050 40\n",
      "27\t143\t0.0\t2\t0 138 5\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S109_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S109_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S109_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S109_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S109_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S109_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S109_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S109_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        36,381 reads  @    126.9 µs/read;   0.47 M reads/minute\n",
      "Finished in 4.62 s (127 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             36,381\n",
      "  Read 1 with adapter:                  36,271 (99.7%)\n",
      "  Read 2 with adapter:                  36,337 (99.9%)\n",
      "Pairs written (passing filters):        36,381 (100.0%)\n",
      "\n",
      "Total basepairs processed:    21,441,947 bp\n",
      "  Read 1:    10,710,449 bp\n",
      "  Read 2:    10,731,498 bp\n",
      "Total written (filtered):     19,554,661 bp (91.2%)\n",
      "  Read 1:     9,767,638 bp\n",
      "  Read 2:     9,787,023 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 36271 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t349\t0.0\t2\t0 335 14\n",
      "26\t35759\t0.0\t2\t34849 886 24\n",
      "27\t112\t0.0\t2\t0 111 1\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 36337 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t408\t0.0\t2\t0 385 23\n",
      "26\t35739\t0.0\t2\t34925 789 25\n",
      "27\t127\t0.0\t2\t0 122 5\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S110_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S110_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S110_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S110_213_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S110_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S110_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S110_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S110_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[     8=-----] 00:00:16       129,515 reads  @    127.2 µs/read;   0.47 M reads/minute\n",
      "Finished in 16.48 s (127 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:            129,515\n",
      "  Read 1 with adapter:                 129,419 (99.9%)\n",
      "  Read 2 with adapter:                 129,298 (99.8%)\n",
      "Pairs written (passing filters):       129,515 (100.0%)\n",
      "\n",
      "Total basepairs processed:    76,397,180 bp\n",
      "  Read 1:    38,202,137 bp\n",
      "  Read 2:    38,195,043 bp\n",
      "Total written (filtered):     69,671,669 bp (91.2%)\n",
      "  Read 1:    34,837,502 bp\n",
      "  Read 2:    34,834,167 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 129419 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t1010\t0.0\t2\t0 984 26\n",
      "26\t127664\t0.0\t2\t124770 2846 48\n",
      "27\t451\t0.0\t2\t0 437 14\n",
      "28\t222\t0.0\t2\t0 0 222\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 129298 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t119\t0.0\t2\t0 0 119\n",
      "25\t1220\t0.0\t2\t0 1181 39\n",
      "26\t127403\t0.0\t2\t124471 2836 96\n",
      "27\t526\t0.0\t2\t0 503 23\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S111_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S111_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S111_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S111_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S111_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S111_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S111_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S111_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S112_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S112_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S112_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S112_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S112_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S112_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S112_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S112_217_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10        79,498 reads  @    135.4 µs/read;   0.44 M reads/minute\n",
      "Finished in 10.77 s (135 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,498\n",
      "  Read 1 with adapter:                  79,436 (99.9%)\n",
      "  Read 2 with adapter:                  79,383 (99.9%)\n",
      "Pairs written (passing filters):        79,498 (100.0%)\n",
      "\n",
      "Total basepairs processed:    46,899,908 bp\n",
      "  Read 1:    23,449,880 bp\n",
      "  Read 2:    23,450,028 bp\n",
      "Total written (filtered):     42,771,283 bp (91.2%)\n",
      "  Read 1:    21,384,620 bp\n",
      "  Read 2:    21,386,663 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 79436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t55\t0.0\t2\t0 0 55\n",
      "25\t593\t0.0\t2\t0 577 16\n",
      "26\t78334\t0.0\t2\t76506 1789 39\n",
      "27\t281\t0.0\t2\t0 274 7\n",
      "28\t173\t0.0\t2\t0 0 173\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 79383 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t81\t0.0\t2\t0 0 81\n",
      "25\t836\t0.0\t2\t0 809 27\n",
      "26\t78098\t0.0\t2\t76407 1661 30\n",
      "27\t331\t0.0\t2\t0 324 7\n",
      "28\t37\t0.0\t2\t0 0 37\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S113_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S113_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S113_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S113_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S113_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S113_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S113_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S113_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[    8<------] 00:00:11        84,583 reads  @    131.4 µs/read;   0.46 M reads/minute\n",
      "Finished in 11.11 s (131 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             84,583\n",
      "  Read 1 with adapter:                  84,542 (100.0%)\n",
      "  Read 2 with adapter:                  84,455 (99.8%)\n",
      "Pairs written (passing filters):        84,583 (100.0%)\n",
      "\n",
      "Total basepairs processed:    49,898,140 bp\n",
      "  Read 1:    24,951,439 bp\n",
      "  Read 2:    24,946,701 bp\n",
      "Total written (filtered):     45,504,937 bp (91.2%)\n",
      "  Read 1:    22,753,509 bp\n",
      "  Read 2:    22,751,428 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 84542 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t658\t0.0\t2\t0 639 19\n",
      "26\t83368\t0.0\t2\t81617 1723 28\n",
      "27\t308\t0.0\t2\t0 300 8\n",
      "28\t151\t0.0\t2\t0 0 151\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 84455 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t814\t0.0\t2\t0 797 17\n",
      "26\t83213\t0.0\t2\t81305 1844 64\n",
      "27\t347\t0.0\t2\t0 334 13\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S114_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S114_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S114_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S114_221_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S114_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S114_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S114_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S114_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10        74,360 reads  @    134.8 µs/read;   0.45 M reads/minute\n",
      "Finished in 10.03 s (135 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,360\n",
      "  Read 1 with adapter:                  74,302 (99.9%)\n",
      "  Read 2 with adapter:                  74,241 (99.8%)\n",
      "Pairs written (passing filters):        74,360 (100.0%)\n",
      "\n",
      "Total basepairs processed:    43,867,545 bp\n",
      "  Read 1:    21,935,192 bp\n",
      "  Read 2:    21,932,353 bp\n",
      "Total written (filtered):     40,006,104 bp (91.2%)\n",
      "  Read 1:    20,003,588 bp\n",
      "  Read 2:    20,002,516 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74302 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t587\t0.0\t2\t0 568 19\n",
      "26\t73329\t0.0\t2\t71688 1615 26\n",
      "27\t253\t0.0\t2\t0 240 13\n",
      "28\t88\t0.0\t2\t0 0 88\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 74241 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t702\t0.0\t2\t0 681 21\n",
      "26\t73137\t0.0\t2\t71428 1665 44\n",
      "27\t315\t0.0\t2\t0 300 15\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S115_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S115_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S115_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S115_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S115_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S115_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S115_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S115_223_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:08        72,433 reads  @    112.8 µs/read;   0.53 M reads/minute\n",
      "Finished in 8.18 s (113 us/read; 0.53 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             72,433\n",
      "  Read 1 with adapter:                  72,382 (99.9%)\n",
      "  Read 2 with adapter:                  72,309 (99.8%)\n",
      "Pairs written (passing filters):        72,433 (100.0%)\n",
      "\n",
      "Total basepairs processed:    42,734,495 bp\n",
      "  Read 1:    21,367,240 bp\n",
      "  Read 2:    21,367,255 bp\n",
      "Total written (filtered):     38,973,306 bp (91.2%)\n",
      "  Read 1:    19,485,644 bp\n",
      "  Read 2:    19,487,662 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 72382 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t584\t0.0\t2\t0 565 19\n",
      "26\t71482\t0.0\t2\t69804 1655 23\n",
      "27\t208\t0.0\t2\t0 202 6\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 72309 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t716\t0.0\t2\t0 703 13\n",
      "26\t71191\t0.0\t2\t69517 1612 62\n",
      "27\t293\t0.0\t2\t0 282 11\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S116_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S116_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S116_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S116_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S116_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S116_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S116_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S116_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:09        66,131 reads  @    149.8 µs/read;   0.40 M reads/minute\n",
      "Finished in 9.91 s (150 us/read; 0.40 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             66,131\n",
      "  Read 1 with adapter:                  65,678 (99.3%)\n",
      "  Read 2 with adapter:                  66,025 (99.8%)\n",
      "Pairs written (passing filters):        66,131 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,947,360 bp\n",
      "  Read 1:    19,443,748 bp\n",
      "  Read 2:    19,503,612 bp\n",
      "Total written (filtered):     35,523,896 bp (91.2%)\n",
      "  Read 1:    17,736,542 bp\n",
      "  Read 2:    17,787,354 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 65678 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t594\t0.0\t2\t0 584 10\n",
      "26\t64812\t0.0\t2\t63156 1637 19\n",
      "27\t200\t0.0\t2\t0 195 5\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 66025 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t47\t0.0\t2\t0 0 47\n",
      "25\t657\t0.0\t2\t0 638 19\n",
      "26\t65010\t0.0\t2\t63541 1427 42\n",
      "27\t263\t0.0\t2\t0 253 10\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S117_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S117_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S117_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S117_227_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S117_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S117_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S117_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S117_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:08        62,168 reads  @    129.3 µs/read;   0.46 M reads/minute\n",
      "Finished in 8.04 s (129 us/read; 0.46 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,168\n",
      "  Read 1 with adapter:                  62,110 (99.9%)\n",
      "  Read 2 with adapter:                  62,085 (99.9%)\n",
      "Pairs written (passing filters):        62,168 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,673,000 bp\n",
      "  Read 1:    18,336,271 bp\n",
      "  Read 2:    18,336,729 bp\n",
      "Total written (filtered):     33,444,604 bp (91.2%)\n",
      "  Read 1:    16,721,687 bp\n",
      "  Read 2:    16,722,917 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62110 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t480\t0.0\t2\t0 465 15\n",
      "26\t61328\t0.0\t2\t59889 1417 22\n",
      "27\t200\t0.0\t2\t0 195 5\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62085 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t591\t0.0\t2\t0 569 22\n",
      "26\t61193\t0.0\t2\t59684 1433 76\n",
      "27\t249\t0.0\t2\t0 236 13\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S118_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S118_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S118_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S118_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S118_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S118_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S118_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S118_229_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10        79,734 reads  @    137.6 µs/read;   0.44 M reads/minute\n",
      "Finished in 10.98 s (138 us/read; 0.44 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             79,734\n",
      "  Read 1 with adapter:                  79,657 (99.9%)\n",
      "  Read 2 with adapter:                  79,609 (99.8%)\n",
      "Pairs written (passing filters):        79,734 (100.0%)\n",
      "\n",
      "Total basepairs processed:    47,035,147 bp\n",
      "  Read 1:    23,517,450 bp\n",
      "  Read 2:    23,517,697 bp\n",
      "Total written (filtered):     42,895,079 bp (91.2%)\n",
      "  Read 1:    21,446,742 bp\n",
      "  Read 2:    21,448,337 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 79657 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t690\t0.0\t2\t0 673 17\n",
      "26\t78552\t0.0\t2\t76647 1878 27\n",
      "27\t274\t0.0\t2\t0 261 13\n",
      "28\t81\t0.0\t2\t0 0 81\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 79609 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t781\t0.0\t2\t0 759 22\n",
      "26\t78377\t0.0\t2\t76440 1874 63\n",
      "27\t355\t0.0\t2\t0 343 12\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S119_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S119_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S119_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S119_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S119_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S119_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S119_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S119_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8=-------] 00:00:11        87,641 reads  @    127.3 µs/read;   0.47 M reads/minute\n",
      "Finished in 11.16 s (127 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             87,641\n",
      "  Read 1 with adapter:                  87,568 (99.9%)\n",
      "  Read 2 with adapter:                  87,489 (99.8%)\n",
      "Pairs written (passing filters):        87,641 (100.0%)\n",
      "\n",
      "Total basepairs processed:    51,700,500 bp\n",
      "  Read 1:    25,848,792 bp\n",
      "  Read 2:    25,851,708 bp\n",
      "Total written (filtered):     47,149,880 bp (91.2%)\n",
      "  Read 1:    23,572,432 bp\n",
      "  Read 2:    23,577,448 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 87568 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t757\t0.0\t2\t0 735 22\n",
      "26\t86377\t0.0\t2\t84293 2044 40\n",
      "27\t315\t0.0\t2\t0 303 12\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 87489 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t769\t0.0\t2\t0 750 19\n",
      "26\t86235\t0.0\t2\t84138 2026 71\n",
      "27\t387\t0.0\t2\t0 374 13\n",
      "28\t31\t0.0\t2\t0 0 31\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S120_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S120_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S120_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S120_233_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S120_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S120_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S120_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S120_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8<--------] 00:00:07        61,940 reads  @    126.9 µs/read;   0.47 M reads/minute\n",
      "Finished in 7.87 s (127 us/read; 0.47 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,940\n",
      "  Read 1 with adapter:                  61,876 (99.9%)\n",
      "  Read 2 with adapter:                  61,803 (99.8%)\n",
      "Pairs written (passing filters):        61,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,530,798 bp\n",
      "  Read 1:    18,270,519 bp\n",
      "  Read 2:    18,260,279 bp\n",
      "Total written (filtered):     33,316,045 bp (91.2%)\n",
      "  Read 1:    16,662,221 bp\n",
      "  Read 2:    16,653,824 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61876 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t725\t0.0\t2\t0 699 26\n",
      "26\t60838\t0.0\t2\t59214 1596 28\n",
      "27\t163\t0.0\t2\t0 151 12\n",
      "28\t96\t0.0\t2\t0 0 96\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61803 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t636\t0.0\t2\t0 623 13\n",
      "26\t60820\t0.0\t2\t59220 1539 61\n",
      "27\t281\t0.0\t2\t0 266 15\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S121_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S121_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S121_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S121_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S121_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S121_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S121_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S121_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[  8=--------] 00:00:09        74,940 reads  @    120.9 µs/read;   0.50 M reads/minute\n",
      "Finished in 9.07 s (121 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             74,940\n",
      "  Read 1 with adapter:                  74,868 (99.9%)\n",
      "  Read 2 with adapter:                  74,837 (99.9%)\n",
      "Pairs written (passing filters):        74,940 (100.0%)\n",
      "\n",
      "Total basepairs processed:    44,209,135 bp\n",
      "  Read 1:    22,106,818 bp\n",
      "  Read 2:    22,102,317 bp\n",
      "Total written (filtered):     40,317,984 bp (91.2%)\n",
      "  Read 1:    20,160,879 bp\n",
      "  Read 2:    20,157,105 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 74868 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t833\t0.0\t2\t0 812 21\n",
      "26\t73733\t0.0\t2\t71927 1785 21\n",
      "27\t184\t0.0\t2\t0 176 8\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 74837 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t796\t0.0\t2\t0 763 33\n",
      "26\t73648\t0.0\t2\t71888 1695 65\n",
      "27\t324\t0.0\t2\t0 314 10\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S122_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S122_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S122_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S122_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S122_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-29_jua3j/S122_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S122_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1ptnr8kg/65315a26-20d2-4a2e-ba1b-c152e4f1bf59/data/S122_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[   8<-------] 00:00:10        85,904 reads  @    119.8 µs/read;   0.50 M reads/minute\n",
      "Finished in 10.30 s (120 us/read; 0.50 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             85,904\n",
      "  Read 1 with adapter:                  85,799 (99.9%)\n",
      "  Read 2 with adapter:                  85,738 (99.8%)\n",
      "Pairs written (passing filters):        85,904 (100.0%)\n",
      "\n",
      "Total basepairs processed:    50,675,309 bp\n",
      "  Read 1:    25,338,831 bp\n",
      "  Read 2:    25,336,478 bp\n",
      "Total written (filtered):     46,216,532 bp (91.2%)\n",
      "  Read 1:    23,108,716 bp\n",
      "  Read 2:    23,107,816 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 85799 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t958\t0.0\t2\t0 932 26\n",
      "26\t84422\t0.0\t2\t82233 2154 35\n",
      "27\t259\t0.0\t2\t0 245 14\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 85738 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t816\t0.0\t2\t0 791 25\n",
      "26\t84458\t0.0\t2\t82290 2078 90\n",
      "27\t358\t0.0\t2\t0 339 19\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: COI_1-demux-pairedend-trimmed1.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences COI_1-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences COI_1-demux-pairedend-trimmed1.qza \\\n",
    "    --verbose"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI_1-demux-pairedend-trimmed1.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "  --i-data COI_1-demux-pairedend-trimmed1.qza \\\n",
    "  --o-visualization COI_1-demux-pairedend-trimmed1.qzv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-1mpeha8e')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: fda6a695-fdc8-49a6-ae21-187eed0f40d3>"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('COI_1-demux-pairedend-trimmed1.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI_1-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI_1-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: COI_1-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#Generate ESVs (Exact Sequence Variants). QIIME2 has two packages available for this. We are using dada2 for our single end reads.\n",
    "# NOTE: if you are wanting to do paired end reads, you'll use qiime dada2 denoise-paired (https://docs.qiime2.org/2019.7/plugins/available/dada2/denoise-paired/)\n",
    "# NOTE: if you are wanting to join reads you'll have to use Deblur\n",
    "\n",
    "\n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs COI_1-demux-pairedend-trimmed1.qza \\\n",
    "  --p-trunc-len-f 260 \\\n",
    "  --p-trunc-len-r 220 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method consensus \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences COI_1-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table COI_1-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats COI_1-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI_1-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI_1-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: COI_1-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#Generate ESVs using pooled to test whether this increases diversity and leads to undersaturation of rarefaction curves\n",
    "\n",
    "\n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs COI_1-demux-pairedend-trimmed1.qza \\\n",
    "  --p-trunc-len-f 260 \\\n",
    "  --p-trunc-len-r 220 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method consensus \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences COI_1-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table COI_1-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats COI_1-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "now do it all again for the second run"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported leray_manifest_run2_v1.csv as PairedEndFastqManifestPhred33 to COI_2-paired-end-demux.qza\u001b[0m\n",
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S251_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S251_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S251_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S251_1_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S251_0_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S251_1_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S251_0_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S251_1_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,869 reads  @     82.5 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.87 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,869\n",
      "  Read 1 with adapter:                  46,812 (99.9%)\n",
      "  Read 2 with adapter:                  46,776 (99.8%)\n",
      "Pairs written (passing filters):        46,869 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,650,425 bp\n",
      "  Read 1:    13,825,108 bp\n",
      "  Read 2:    13,825,317 bp\n",
      "Total written (filtered):     25,217,847 bp (91.2%)\n",
      "  Read 1:    12,608,002 bp\n",
      "  Read 2:    12,609,845 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t303\t0.0\t2\t0 286 17\n",
      "26\t46212\t0.0\t2\t44741 1430 41\n",
      "27\t185\t0.0\t2\t0 175 10\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46776 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t75\t0.0\t2\t0 0 75\n",
      "25\t763\t0.0\t2\t0 742 21\n",
      "26\t45750\t0.0\t2\t44816 902 32\n",
      "27\t167\t0.0\t2\t0 161 6\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S252_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S252_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S252_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S252_3_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S252_2_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S252_3_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S252_2_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S252_3_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,127 reads  @     76.1 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.73 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,127\n",
      "  Read 1 with adapter:                  62,050 (99.9%)\n",
      "  Read 2 with adapter:                  61,983 (99.8%)\n",
      "Pairs written (passing filters):        62,127 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,651,992 bp\n",
      "  Read 1:    18,326,787 bp\n",
      "  Read 2:    18,325,205 bp\n",
      "Total written (filtered):     33,428,109 bp (91.2%)\n",
      "  Read 1:    16,713,544 bp\n",
      "  Read 2:    16,714,565 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62050 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t457\t0.0\t2\t0 442 15\n",
      "26\t61180\t0.0\t2\t59245 1887 48\n",
      "27\t262\t0.0\t2\t0 253 9\n",
      "28\t110\t0.0\t2\t0 0 110\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61983 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t89\t0.0\t2\t0 0 89\n",
      "25\t1032\t0.0\t2\t0 999 33\n",
      "26\t60577\t0.0\t2\t59179 1351 47\n",
      "27\t278\t0.0\t2\t0 265 13\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S253_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S253_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S253_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S253_5_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S253_4_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S253_5_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S253_4_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S253_5_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,830 reads  @     77.8 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.19 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,830\n",
      "  Read 1 with adapter:                  53,755 (99.9%)\n",
      "  Read 2 with adapter:                  53,702 (99.8%)\n",
      "Pairs written (passing filters):        53,830 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,756,516 bp\n",
      "  Read 1:    15,879,630 bp\n",
      "  Read 2:    15,876,886 bp\n",
      "Total written (filtered):     28,963,519 bp (91.2%)\n",
      "  Read 1:    14,482,038 bp\n",
      "  Read 2:    14,481,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53755 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t387\t0.0\t2\t0 377 10\n",
      "26\t52980\t0.0\t2\t51261 1672 47\n",
      "27\t243\t0.0\t2\t0 239 4\n",
      "28\t99\t0.0\t2\t0 0 99\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53702 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t89\t0.0\t2\t0 0 89\n",
      "25\t919\t0.0\t2\t0 899 20\n",
      "26\t52454\t0.0\t2\t51262 1161 31\n",
      "27\t230\t0.0\t2\t0 217 13\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S254_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S254_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S254_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S254_7_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S254_6_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S254_7_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S254_6_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S254_7_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        34,257 reads  @     89.3 µs/read;   0.67 M reads/minute\n",
      "Finished in 3.06 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,257\n",
      "  Read 1 with adapter:                  34,215 (99.9%)\n",
      "  Read 2 with adapter:                  34,087 (99.5%)\n",
      "Pairs written (passing filters):        34,257 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,198,540 bp\n",
      "  Read 1:    10,105,561 bp\n",
      "  Read 2:    10,092,979 bp\n",
      "Total written (filtered):     18,423,313 bp (91.2%)\n",
      "  Read 1:     9,216,124 bp\n",
      "  Read 2:     9,207,189 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34215 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t301\t0.0\t2\t0 288 13\n",
      "26\t33735\t0.0\t2\t32651 1050 34\n",
      "27\t130\t0.0\t2\t0 128 2\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34087 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t550\t0.0\t2\t0 531 19\n",
      "26\t33359\t0.0\t2\t32493 833 33\n",
      "27\t138\t0.0\t2\t0 129 9\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S255_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S255_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S255_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S255_9_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S255_8_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S255_9_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S255_8_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S255_9_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        39,440 reads  @     93.9 µs/read;   0.64 M reads/minute\n",
      "Finished in 3.71 s (94 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,440\n",
      "  Read 1 with adapter:                  39,385 (99.9%)\n",
      "  Read 2 with adapter:                  37,218 (94.4%)\n",
      "Pairs written (passing filters):        39,440 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,805,159 bp\n",
      "  Read 1:    11,633,658 bp\n",
      "  Read 2:    11,171,501 bp\n",
      "Total written (filtered):     20,814,368 bp (91.3%)\n",
      "  Read 1:    10,609,788 bp\n",
      "  Read 2:    10,204,580 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t309\t0.0\t2\t0 297 12\n",
      "26\t38865\t0.0\t2\t37669 1170 26\n",
      "27\t133\t0.0\t2\t0 129 4\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37218 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t789\t0.0\t2\t0 758 31\n",
      "26\t36209\t0.0\t2\t35329 854 26\n",
      "27\t150\t0.0\t2\t0 143 7\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S256_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S256_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S256_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S256_11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S256_10_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S256_11_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S256_10_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S256_11_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,723 reads  @     96.6 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.61 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,723\n",
      "  Read 1 with adapter:                  47,650 (99.8%)\n",
      "  Read 2 with adapter:                  47,529 (99.6%)\n",
      "Pairs written (passing filters):        47,721 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,143,456 bp\n",
      "  Read 1:    14,077,353 bp\n",
      "  Read 2:    14,066,103 bp\n",
      "Total written (filtered):     25,669,355 bp (91.2%)\n",
      "  Read 1:    12,838,167 bp\n",
      "  Read 2:    12,831,188 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47650 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t422\t0.0\t2\t0 413 9\n",
      "26\t46995\t0.0\t2\t45523 1432 40\n",
      "27\t164\t0.0\t2\t0 160 4\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47529 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t93\t0.0\t2\t0 0 93\n",
      "25\t845\t0.0\t2\t0 826 19\n",
      "26\t46404\t0.0\t2\t45316 1051 37\n",
      "27\t182\t0.0\t2\t0 175 7\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S257_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S257_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S257_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S257_13_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S257_12_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S257_13_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S257_12_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S257_13_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        56,870 reads  @     91.8 µs/read;   0.65 M reads/minute\n",
      "Finished in 5.22 s (92 us/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,870\n",
      "  Read 1 with adapter:                  56,790 (99.9%)\n",
      "  Read 2 with adapter:                  56,782 (99.8%)\n",
      "Pairs written (passing filters):        56,870 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,547,054 bp\n",
      "  Read 1:    16,774,963 bp\n",
      "  Read 2:    16,772,091 bp\n",
      "Total written (filtered):     30,595,322 bp (91.2%)\n",
      "  Read 1:    15,298,629 bp\n",
      "  Read 2:    15,296,693 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56790 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t397\t0.0\t2\t0 391 6\n",
      "26\t56127\t0.0\t2\t54507 1581 39\n",
      "27\t173\t0.0\t2\t0 165 8\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56782 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t78\t0.0\t2\t0 0 78\n",
      "25\t1015\t0.0\t2\t0 992 23\n",
      "26\t55460\t0.0\t2\t54269 1164 27\n",
      "27\t221\t0.0\t2\t0 215 6\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S258_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S258_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S258_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S258_15_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S258_14_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S258_15_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S258_14_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S258_15_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,189 reads  @     83.5 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.61 s (84 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,189\n",
      "  Read 1 with adapter:                  55,119 (99.9%)\n",
      "  Read 2 with adapter:                  55,078 (99.8%)\n",
      "Pairs written (passing filters):        55,189 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,560,239 bp\n",
      "  Read 1:    16,280,447 bp\n",
      "  Read 2:    16,279,792 bp\n",
      "Total written (filtered):     29,695,976 bp (91.2%)\n",
      "  Read 1:    14,847,484 bp\n",
      "  Read 2:    14,848,492 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55119 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t397\t0.0\t2\t0 390 7\n",
      "26\t54439\t0.0\t2\t52850 1548 41\n",
      "27\t188\t0.0\t2\t0 181 7\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55078 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t840\t0.0\t2\t0 821 19\n",
      "26\t53974\t0.0\t2\t52822 1118 34\n",
      "27\t180\t0.0\t2\t0 173 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S259_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S259_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S259_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S259_17_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S259_16_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S259_17_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S259_16_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S259_17_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,113 reads  @     87.8 µs/read;   0.68 M reads/minute\n",
      "Finished in 4.84 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,113\n",
      "  Read 1 with adapter:                  55,045 (99.9%)\n",
      "  Read 2 with adapter:                  54,966 (99.7%)\n",
      "Pairs written (passing filters):        55,113 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,515,219 bp\n",
      "  Read 1:    16,257,941 bp\n",
      "  Read 2:    16,257,278 bp\n",
      "Total written (filtered):     29,655,857 bp (91.2%)\n",
      "  Read 1:    14,826,879 bp\n",
      "  Read 2:    14,828,978 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55045 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t420\t0.0\t2\t0 409 11\n",
      "26\t54303\t0.0\t2\t52554 1688 61\n",
      "27\t228\t0.0\t2\t0 224 4\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54966 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t98\t0.0\t2\t0 0 98\n",
      "25\t927\t0.0\t2\t0 902 25\n",
      "26\t53675\t0.0\t2\t52575 1059 41\n",
      "27\t225\t0.0\t2\t0 213 12\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S260_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S260_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S260_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S260_19_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S260_18_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S260_19_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S260_18_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S260_19_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,246 reads  @     79.0 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.42 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,246\n",
      "  Read 1 with adapter:                  43,169 (99.8%)\n",
      "  Read 2 with adapter:                  43,137 (99.7%)\n",
      "Pairs written (passing filters):        43,246 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,512,748 bp\n",
      "  Read 1:    12,757,270 bp\n",
      "  Read 2:    12,755,478 bp\n",
      "Total written (filtered):     23,269,602 bp (91.2%)\n",
      "  Read 1:    11,635,024 bp\n",
      "  Read 2:    11,634,578 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43169 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t15\t0.0\t2\t0 0 15\n",
      "25\t370\t0.0\t2\t0 357 13\n",
      "26\t42583\t0.0\t2\t41141 1400 42\n",
      "27\t150\t0.0\t2\t0 146 4\n",
      "28\t51\t0.0\t2\t0 0 51\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43137 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t715\t0.0\t2\t0 703 12\n",
      "26\t42163\t0.0\t2\t41151 983 29\n",
      "27\t181\t0.0\t2\t0 175 6\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S261_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S261_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S261_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S261_21_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S261_20_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S261_21_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S261_20_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S261_21_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,797 reads  @     76.3 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.65 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,797\n",
      "  Read 1 with adapter:                  47,718 (99.8%)\n",
      "  Read 2 with adapter:                  47,719 (99.8%)\n",
      "Pairs written (passing filters):        47,796 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,196,437 bp\n",
      "  Read 1:    14,099,812 bp\n",
      "  Read 2:    14,096,625 bp\n",
      "Total written (filtered):     25,715,498 bp (91.2%)\n",
      "  Read 1:    12,859,296 bp\n",
      "  Read 2:    12,856,202 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t403\t0.0\t2\t0 390 13\n",
      "26\t47047\t0.0\t2\t45401 1613 33\n",
      "27\t169\t0.0\t2\t0 161 8\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47719 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t609\t0.0\t2\t0 589 20\n",
      "26\t46866\t0.0\t2\t45851 992 23\n",
      "27\t183\t0.0\t2\t0 174 9\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S262_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S262_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S262_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S262_23_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S262_22_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S262_23_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S262_22_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S262_23_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,787 reads  @     78.5 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.60 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,787\n",
      "  Read 1 with adapter:                  45,732 (99.9%)\n",
      "  Read 2 with adapter:                  45,518 (99.4%)\n",
      "Pairs written (passing filters):        45,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,988,157 bp\n",
      "  Read 1:    13,507,158 bp\n",
      "  Read 2:    13,480,999 bp\n",
      "Total written (filtered):     24,615,930 bp (91.2%)\n",
      "  Read 1:    12,317,925 bp\n",
      "  Read 2:    12,298,005 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45732 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t345\t0.0\t2\t0 336 9\n",
      "26\t45107\t0.0\t2\t43729 1334 44\n",
      "27\t191\t0.0\t2\t0 186 5\n",
      "28\t66\t0.0\t2\t0 0 66\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45518 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t35\t0.0\t2\t0 0 35\n",
      "25\t556\t0.0\t2\t0 532 24\n",
      "26\t44777\t0.0\t2\t43790 936 51\n",
      "27\t148\t0.0\t2\t0 145 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S263_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S263_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S263_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S263_25_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S263_24_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S263_25_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S263_24_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S263_25_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,933 reads  @     88.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.27 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,933\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                  47,538 (99.2%)\n",
      "Pairs written (passing filters):        47,932 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,154,146 bp\n",
      "  Read 1:    14,077,073 bp\n",
      "  Read 2:    14,077,073 bp\n",
      "Total written (filtered):     26,918,845 bp (95.6%)\n",
      "  Read 1:    14,077,073 bp\n",
      "  Read 2:    12,841,772 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47538 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t774\t0.0\t2\t0 751 23\n",
      "26\t46519\t0.0\t2\t45433 1035 51\n",
      "27\t175\t0.0\t2\t0 165 10\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S264_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S264_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S264_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S264_27_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S264_26_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S264_27_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S264_26_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S264_27_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        48,209 reads  @     93.4 µs/read;   0.64 M reads/minute\n",
      "Finished in 4.51 s (93 us/read; 0.64 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,209\n",
      "  Read 1 with adapter:                  48,145 (99.9%)\n",
      "  Read 2 with adapter:                  48,059 (99.7%)\n",
      "Pairs written (passing filters):        48,207 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,423,171 bp\n",
      "  Read 1:    14,221,650 bp\n",
      "  Read 2:    14,201,521 bp\n",
      "Total written (filtered):     25,922,109 bp (91.2%)\n",
      "  Read 1:    12,969,480 bp\n",
      "  Read 2:    12,952,629 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48145 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t395\t0.0\t2\t0 386 9\n",
      "26\t47477\t0.0\t2\t45748 1679 50\n",
      "27\t165\t0.0\t2\t0 157 8\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48059 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t717\t0.0\t2\t0 705 12\n",
      "26\t47068\t0.0\t2\t45952 1093 23\n",
      "27\t189\t0.0\t2\t0 180 9\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S265_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S265_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S265_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S265_29_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S265_28_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S265_29_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S265_28_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S265_29_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,096 reads  @     80.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.72 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,096\n",
      "  Read 1 with adapter:                  46,020 (99.8%)\n",
      "  Read 2 with adapter:                  45,996 (99.8%)\n",
      "Pairs written (passing filters):        46,096 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,192,507 bp\n",
      "  Read 1:    13,597,758 bp\n",
      "  Read 2:    13,594,749 bp\n",
      "Total written (filtered):     24,801,112 bp (91.2%)\n",
      "  Read 1:    12,401,521 bp\n",
      "  Read 2:    12,399,591 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46020 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t415\t0.0\t2\t0 397 18\n",
      "26\t45417\t0.0\t2\t43901 1469 47\n",
      "27\t116\t0.0\t2\t0 110 6\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45996 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t860\t0.0\t2\t0 834 26\n",
      "26\t44868\t0.0\t2\t43836 994 38\n",
      "27\t202\t0.0\t2\t0 194 8\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S266_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S266_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S266_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S266_31_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S266_30_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S266_31_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S266_30_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S266_31_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        52,616 reads  @     80.9 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.26 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,616\n",
      "  Read 1 with adapter:                  52,547 (99.9%)\n",
      "  Read 2 with adapter:                  52,493 (99.8%)\n",
      "Pairs written (passing filters):        52,616 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,042,444 bp\n",
      "  Read 1:    15,521,543 bp\n",
      "  Read 2:    15,520,901 bp\n",
      "Total written (filtered):     28,312,224 bp (91.2%)\n",
      "  Read 1:    14,155,525 bp\n",
      "  Read 2:    14,156,699 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t419\t0.0\t2\t0 409 10\n",
      "26\t51863\t0.0\t2\t50262 1550 51\n",
      "27\t195\t0.0\t2\t0 188 7\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52493 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t734\t0.0\t2\t0 717 17\n",
      "26\t51498\t0.0\t2\t50381 1079 38\n",
      "27\t160\t0.0\t2\t0 154 6\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S267_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S267_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S267_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S267_33_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S267_32_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S267_33_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S267_32_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S267_33_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:05        65,888 reads  @     83.0 µs/read;   0.72 M reads/minute\n",
      "Finished in 5.47 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             65,888\n",
      "  Read 1 with adapter:                  65,717 (99.7%)\n",
      "  Read 2 with adapter:                  65,744 (99.8%)\n",
      "Pairs written (passing filters):        65,888 (100.0%)\n",
      "\n",
      "Total basepairs processed:    38,866,280 bp\n",
      "  Read 1:    19,430,148 bp\n",
      "  Read 2:    19,436,132 bp\n",
      "Total written (filtered):     35,449,271 bp (91.2%)\n",
      "  Read 1:    17,721,547 bp\n",
      "  Read 2:    17,727,724 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 65717 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t439\t0.0\t2\t0 422 17\n",
      "26\t64868\t0.0\t2\t62768 2054 46\n",
      "27\t254\t0.0\t2\t0 246 8\n",
      "28\t114\t0.0\t2\t0 0 114\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 65744 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t95\t0.0\t2\t0 0 95\n",
      "25\t1098\t0.0\t2\t0 1065 33\n",
      "26\t64215\t0.0\t2\t62630 1533 52\n",
      "27\t320\t0.0\t2\t0 309 11\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S268_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S268_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S268_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S268_35_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S268_34_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S268_35_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S268_34_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S268_35_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,840 reads  @     78.4 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.38 s (78 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,840\n",
      "  Read 1 with adapter:                  55,598 (99.6%)\n",
      "  Read 2 with adapter:                  55,745 (99.8%)\n",
      "Pairs written (passing filters):        55,840 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,934,106 bp\n",
      "  Read 1:    16,463,440 bp\n",
      "  Read 2:    16,470,666 bp\n",
      "Total written (filtered):     30,039,868 bp (91.2%)\n",
      "  Read 1:    15,017,875 bp\n",
      "  Read 2:    15,021,993 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55598 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t397\t0.0\t2\t0 381 16\n",
      "26\t54803\t0.0\t2\t53084 1679 40\n",
      "27\t246\t0.0\t2\t0 237 9\n",
      "28\t118\t0.0\t2\t0 0 118\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55745 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t80\t0.0\t2\t0 0 80\n",
      "25\t776\t0.0\t2\t0 759 17\n",
      "26\t54656\t0.0\t2\t53351 1258 47\n",
      "27\t227\t0.0\t2\t0 214 13\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S269_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S269_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S269_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S269_37_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S269_36_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S269_37_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S269_36_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S269_37_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,171 reads  @     74.2 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.13 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,171\n",
      "  Read 1 with adapter:                  41,992 (99.6%)\n",
      "  Read 2 with adapter:                  42,102 (99.8%)\n",
      "Pairs written (passing filters):        42,171 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,871,331 bp\n",
      "  Read 1:    12,434,517 bp\n",
      "  Read 2:    12,436,814 bp\n",
      "Total written (filtered):     22,685,385 bp (91.2%)\n",
      "  Read 1:    11,342,732 bp\n",
      "  Read 2:    11,342,653 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41992 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 20\n",
      "25\t275\t0.0\t2\t0 269 6\n",
      "26\t41456\t0.0\t2\t40169 1244 43\n",
      "27\t174\t0.0\t2\t0 168 6\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42102 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t46\t0.0\t2\t0 0 46\n",
      "25\t551\t0.0\t2\t0 542 9\n",
      "26\t41354\t0.0\t2\t40525 802 27\n",
      "27\t150\t0.0\t2\t0 141 9\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S270_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S270_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S270_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S270_39_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S270_38_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S270_39_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S270_38_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S270_39_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,855 reads  @     79.3 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.24 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,855\n",
      "  Read 1 with adapter:                  40,750 (99.7%)\n",
      "  Read 2 with adapter:                  40,714 (99.7%)\n",
      "Pairs written (passing filters):        40,855 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,095,183 bp\n",
      "  Read 1:    12,050,009 bp\n",
      "  Read 2:    12,045,174 bp\n",
      "Total written (filtered):     21,977,700 bp (91.2%)\n",
      "  Read 1:    10,990,526 bp\n",
      "  Read 2:    10,987,174 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40750 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t306\t0.0\t2\t0 296 10\n",
      "26\t40144\t0.0\t2\t38865 1238 41\n",
      "27\t167\t0.0\t2\t0 163 4\n",
      "28\t97\t0.0\t2\t0 0 97\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40714 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t622\t0.0\t2\t0 609 13\n",
      "26\t39899\t0.0\t2\t38887 956 56\n",
      "27\t148\t0.0\t2\t0 142 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S271_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S271_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S271_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S271_41_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S271_40_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S271_41_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S271_40_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S271_41_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,916 reads  @     81.3 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.81 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,916\n",
      "  Read 1 with adapter:                  46,826 (99.8%)\n",
      "  Read 2 with adapter:                  46,304 (98.7%)\n",
      "Pairs written (passing filters):        46,915 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,570,307 bp\n",
      "  Read 1:    13,838,345 bp\n",
      "  Read 2:    13,731,962 bp\n",
      "Total written (filtered):     25,149,486 bp (91.2%)\n",
      "  Read 1:    12,620,563 bp\n",
      "  Read 2:    12,528,923 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46826 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t308\t0.0\t2\t0 299 9\n",
      "26\t46195\t0.0\t2\t44763 1396 36\n",
      "27\t185\t0.0\t2\t0 176 9\n",
      "28\t109\t0.0\t2\t0 0 109\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46304 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t926\t0.0\t2\t0 897 29\n",
      "26\t45158\t0.0\t2\t44108 999 51\n",
      "27\t163\t0.0\t2\t0 156 7\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S272_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S272_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S272_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S272_43_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S272_42_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S272_43_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S272_42_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S272_43_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        42,323 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.13 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             42,323\n",
      "  Read 1 with adapter:                  42,241 (99.8%)\n",
      "  Read 2 with adapter:                  42,211 (99.7%)\n",
      "Pairs written (passing filters):        42,321 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,961,469 bp\n",
      "  Read 1:    12,483,218 bp\n",
      "  Read 2:    12,478,251 bp\n",
      "Total written (filtered):     22,765,699 bp (91.2%)\n",
      "  Read 1:    11,384,348 bp\n",
      "  Read 2:    11,381,351 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42241 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t280\t0.0\t2\t0 273 7\n",
      "26\t41659\t0.0\t2\t40359 1267 33\n",
      "27\t174\t0.0\t2\t0 168 6\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42211 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t652\t0.0\t2\t0 637 15\n",
      "26\t41349\t0.0\t2\t40402 909 38\n",
      "27\t154\t0.0\t2\t0 147 7\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S273_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S273_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S273_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S273_45_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S273_44_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S273_45_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S273_44_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S273_45_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,309 reads  @     78.6 µs/read;   0.76 M reads/minute\n",
      "Finished in 3.88 s (79 us/read; 0.76 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,309\n",
      "  Read 1 with adapter:                  49,223 (99.8%)\n",
      "  Read 2 with adapter:                  49,181 (99.7%)\n",
      "Pairs written (passing filters):        49,309 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,086,434 bp\n",
      "  Read 1:    14,545,134 bp\n",
      "  Read 2:    14,541,300 bp\n",
      "Total written (filtered):     26,528,877 bp (91.2%)\n",
      "  Read 1:    13,265,406 bp\n",
      "  Read 2:    13,263,471 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t40\t0.0\t2\t0 0 40\n",
      "25\t377\t0.0\t2\t0 371 6\n",
      "26\t48499\t0.0\t2\t47003 1460 36\n",
      "27\t227\t0.0\t2\t0 215 12\n",
      "28\t80\t0.0\t2\t0 0 80\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49181 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t80\t0.0\t2\t0 0 80\n",
      "25\t946\t0.0\t2\t0 907 39\n",
      "26\t47945\t0.0\t2\t46889 1018 38\n",
      "27\t191\t0.0\t2\t0 181 10\n",
      "28\t19\t0.0\t2\t0 0 19\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S274_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S274_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S274_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S274_47_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S274_46_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S274_47_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S274_46_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S274_47_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,472 reads  @     79.8 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.43 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,472\n",
      "  Read 1 with adapter:                  55,061 (99.3%)\n",
      "  Read 2 with adapter:                  55,372 (99.8%)\n",
      "Pairs written (passing filters):        55,472 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,689,376 bp\n",
      "  Read 1:    16,326,076 bp\n",
      "  Read 2:    16,363,300 bp\n",
      "Total written (filtered):     29,819,151 bp (91.2%)\n",
      "  Read 1:    14,894,751 bp\n",
      "  Read 2:    14,924,400 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55061 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t469\t0.0\t2\t0 450 19\n",
      "26\t54333\t0.0\t2\t52540 1747 46\n",
      "27\t162\t0.0\t2\t0 158 4\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55372 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t68\t0.0\t2\t0 0 68\n",
      "25\t872\t0.0\t2\t0 831 41\n",
      "26\t54205\t0.0\t2\t52910 1232 63\n",
      "27\t218\t0.0\t2\t0 211 7\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S275_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S275_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S275_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S275_49_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S275_48_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S275_49_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S275_48_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S275_49_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        26,476 reads  @     81.3 µs/read;   0.74 M reads/minute\n",
      "Finished in 2.16 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,476\n",
      "  Read 1 with adapter:                  26,437 (99.9%)\n",
      "  Read 2 with adapter:                  26,424 (99.8%)\n",
      "Pairs written (passing filters):        26,476 (100.0%)\n",
      "\n",
      "Total basepairs processed:    15,619,359 bp\n",
      "  Read 1:     7,809,385 bp\n",
      "  Read 2:     7,809,974 bp\n",
      "Total written (filtered):     14,245,317 bp (91.2%)\n",
      "  Read 1:     7,122,012 bp\n",
      "  Read 2:     7,123,305 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 26437 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t149\t0.0\t2\t0 143 6\n",
      "26\t26115\t0.0\t2\t25078 1004 33\n",
      "27\t98\t0.0\t2\t0 95 3\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 26424 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t407\t0.0\t2\t0 400 7\n",
      "26\t25875\t0.0\t2\t25299 548 28\n",
      "27\t108\t0.0\t2\t0 105 3\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S276_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S276_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S276_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S276_51_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S276_50_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S276_51_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S276_50_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S276_51_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,056 reads  @     78.0 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.61 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,056\n",
      "  Read 1 with adapter:                  58,972 (99.9%)\n",
      "  Read 2 with adapter:                  58,925 (99.8%)\n",
      "Pairs written (passing filters):        59,056 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,840,117 bp\n",
      "  Read 1:    17,421,238 bp\n",
      "  Read 2:    17,418,879 bp\n",
      "Total written (filtered):     31,775,741 bp (91.2%)\n",
      "  Read 1:    15,888,143 bp\n",
      "  Read 2:    15,887,598 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58972 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t432\t0.0\t2\t0 424 8\n",
      "26\t58254\t0.0\t2\t56555 1658 41\n",
      "27\t165\t0.0\t2\t0 159 6\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58925 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t75\t0.0\t2\t0 0 75\n",
      "25\t862\t0.0\t2\t0 828 34\n",
      "26\t57748\t0.0\t2\t56501 1198 49\n",
      "27\t237\t0.0\t2\t0 231 6\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S277_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S277_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S277_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S277_53_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S277_52_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S277_53_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S277_52_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S277_53_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,078 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.43 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,078\n",
      "  Read 1 with adapter:                  47,017 (99.9%)\n",
      "  Read 2 with adapter:                  47,025 (99.9%)\n",
      "Pairs written (passing filters):        47,078 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,772,890 bp\n",
      "  Read 1:    13,887,603 bp\n",
      "  Read 2:    13,885,287 bp\n",
      "Total written (filtered):     25,328,447 bp (91.2%)\n",
      "  Read 1:    12,665,288 bp\n",
      "  Read 2:    12,663,159 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47017 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t316\t0.0\t2\t0 307 9\n",
      "26\t46500\t0.0\t2\t45044 1416 40\n",
      "27\t117\t0.0\t2\t0 114 3\n",
      "28\t60\t0.0\t2\t0 0 60\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47025 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t614\t0.0\t2\t0 599 15\n",
      "26\t46224\t0.0\t2\t45289 911 24\n",
      "27\t138\t0.0\t2\t0 135 3\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S278_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S278_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S278_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S278_55_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S278_54_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S278_55_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S278_54_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S278_55_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,786 reads  @     77.5 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.55 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,786\n",
      "  Read 1 with adapter:                  45,730 (99.9%)\n",
      "  Read 2 with adapter:                  45,668 (99.7%)\n",
      "Pairs written (passing filters):        45,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,007,989 bp\n",
      "  Read 1:    13,506,860 bp\n",
      "  Read 2:    13,501,129 bp\n",
      "Total written (filtered):     24,632,333 bp (91.2%)\n",
      "  Read 1:    12,317,991 bp\n",
      "  Read 2:    12,314,342 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45730 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t316\t0.0\t2\t0 308 8\n",
      "26\t45186\t0.0\t2\t43854 1300 32\n",
      "27\t139\t0.0\t2\t0 136 3\n",
      "28\t61\t0.0\t2\t0 0 61\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45668 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t682\t0.0\t2\t0 656 26\n",
      "26\t44742\t0.0\t2\t43669 1020 53\n",
      "27\t183\t0.0\t2\t0 173 10\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S279_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S279_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S279_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S279_57_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S279_56_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S279_57_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S279_56_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S279_57_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,468 reads  @     72.3 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.36 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,468\n",
      "  Read 1 with adapter:                  46,404 (99.9%)\n",
      "  Read 2 with adapter:                  46,001 (99.0%)\n",
      "Pairs written (passing filters):        46,468 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,334,355 bp\n",
      "  Read 1:    13,707,762 bp\n",
      "  Read 2:    13,626,593 bp\n",
      "Total written (filtered):     24,932,732 bp (91.2%)\n",
      "  Read 1:    12,501,369 bp\n",
      "  Read 2:    12,431,363 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46404 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t301\t0.0\t2\t0 289 12\n",
      "26\t45889\t0.0\t2\t44560 1297 32\n",
      "27\t130\t0.0\t2\t0 124 6\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46001 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t847\t0.0\t2\t0 820 27\n",
      "26\t44954\t0.0\t2\t44032 888 34\n",
      "27\t121\t0.0\t2\t0 119 2\n",
      "28\t22\t0.0\t2\t0 0 22\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S280_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S280_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S280_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S280_59_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S280_58_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S280_59_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S280_58_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S280_59_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,579 reads  @     68.8 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.41 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,579\n",
      "  Read 1 with adapter:                  49,507 (99.9%)\n",
      "  Read 2 with adapter:                  49,486 (99.8%)\n",
      "Pairs written (passing filters):        49,579 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,245,247 bp\n",
      "  Read 1:    14,625,143 bp\n",
      "  Read 2:    14,620,104 bp\n",
      "Total written (filtered):     26,672,046 bp (91.2%)\n",
      "  Read 1:    13,337,950 bp\n",
      "  Read 2:    13,334,096 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49507 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t307\t0.0\t2\t0 303 4\n",
      "26\t48926\t0.0\t2\t47369 1523 34\n",
      "27\t138\t0.0\t2\t0 132 6\n",
      "28\t113\t0.0\t2\t0 0 113\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49486 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t688\t0.0\t2\t0 671 17\n",
      "26\t48610\t0.0\t2\t47542 1031 37\n",
      "27\t144\t0.0\t2\t0 134 10\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S281_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S281_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S281_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S281_61_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S281_60_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S281_61_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S281_60_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S281_61_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        34,550 reads  @     88.0 µs/read;   0.68 M reads/minute\n",
      "Finished in 3.04 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,550\n",
      "  Read 1 with adapter:                  34,506 (99.9%)\n",
      "  Read 2 with adapter:                  34,433 (99.7%)\n",
      "Pairs written (passing filters):        34,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,375,879 bp\n",
      "  Read 1:    10,192,245 bp\n",
      "  Read 2:    10,183,634 bp\n",
      "Total written (filtered):     18,584,019 bp (91.2%)\n",
      "  Read 1:     9,295,044 bp\n",
      "  Read 2:     9,288,975 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t218\t0.0\t2\t0 212 6\n",
      "26\t34066\t0.0\t2\t32893 1142 31\n",
      "27\t105\t0.0\t2\t0 104 1\n",
      "28\t98\t0.0\t2\t0 0 98\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34433 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t612\t0.0\t2\t0 590 22\n",
      "26\t33659\t0.0\t2\t32614 991 54\n",
      "27\t111\t0.0\t2\t0 106 5\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S282_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S282_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S282_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S282_63_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S282_62_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S282_63_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S282_62_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S282_63_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,891 reads  @     69.0 µs/read;   0.87 M reads/minute\n",
      "Finished in 4.34 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,891\n",
      "  Read 1 with adapter:                  62,812 (99.9%)\n",
      "  Read 2 with adapter:                  62,772 (99.8%)\n",
      "Pairs written (passing filters):        62,891 (100.0%)\n",
      "\n",
      "Total basepairs processed:    37,104,906 bp\n",
      "  Read 1:    18,552,560 bp\n",
      "  Read 2:    18,552,346 bp\n",
      "Total written (filtered):     33,840,857 bp (91.2%)\n",
      "  Read 1:    16,919,779 bp\n",
      "  Read 2:    16,921,078 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62812 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t41\t0.0\t2\t0 0 41\n",
      "25\t431\t0.0\t2\t0 417 14\n",
      "26\t62184\t0.0\t2\t60127 2006 51\n",
      "27\t130\t0.0\t2\t0 126 4\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62772 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t930\t0.0\t2\t0 909 21\n",
      "26\t61536\t0.0\t2\t60134 1354 48\n",
      "27\t242\t0.0\t2\t0 231 11\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S283_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S283_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S283_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S283_65_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S283_64_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S283_65_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S283_64_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S283_65_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,750 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.34 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,750\n",
      "  Read 1 with adapter:                  45,706 (99.9%)\n",
      "  Read 2 with adapter:                  45,639 (99.8%)\n",
      "Pairs written (passing filters):        45,750 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,990,534 bp\n",
      "  Read 1:    13,494,815 bp\n",
      "  Read 2:    13,495,719 bp\n",
      "Total written (filtered):     24,616,636 bp (91.2%)\n",
      "  Read 1:    12,306,903 bp\n",
      "  Read 2:    12,309,733 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45706 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t567\t0.0\t2\t0 556 11\n",
      "26\t44947\t0.0\t2\t43419 1485 43\n",
      "27\t133\t0.0\t2\t0 125 8\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45639 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t64\t0.0\t2\t0 0 64\n",
      "25\t704\t0.0\t2\t0 689 15\n",
      "26\t44669\t0.0\t2\t43816 839 14\n",
      "27\t200\t0.0\t2\t0 197 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S284_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S284_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S284_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S284_67_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S284_66_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S284_67_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S284_66_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S284_67_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        55,158 reads  @     66.9 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.69 s (67 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,158\n",
      "  Read 1 with adapter:                  55,078 (99.9%)\n",
      "  Read 2 with adapter:                  55,067 (99.8%)\n",
      "Pairs written (passing filters):        55,158 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,541,629 bp\n",
      "  Read 1:    16,271,228 bp\n",
      "  Read 2:    16,270,401 bp\n",
      "Total written (filtered):     29,679,126 bp (91.2%)\n",
      "  Read 1:    14,839,692 bp\n",
      "  Read 2:    14,839,434 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55078 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t640\t0.0\t2\t0 626 14\n",
      "26\t54262\t0.0\t2\t52531 1686 45\n",
      "27\t120\t0.0\t2\t0 117 3\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55067 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t82\t0.0\t2\t0 0 82\n",
      "25\t842\t0.0\t2\t0 812 30\n",
      "26\t53915\t0.0\t2\t52819 1074 22\n",
      "27\t225\t0.0\t2\t0 220 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S285_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S285_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S285_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S285_69_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S285_68_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S285_69_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S285_68_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S285_69_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,576 reads  @     68.5 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.54 s (69 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,576\n",
      "  Read 1 with adapter:                  51,506 (99.9%)\n",
      "  Read 2 with adapter:                  51,481 (99.8%)\n",
      "Pairs written (passing filters):        51,576 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,426,233 bp\n",
      "  Read 1:    15,214,587 bp\n",
      "  Read 2:    15,211,646 bp\n",
      "Total written (filtered):     27,749,723 bp (91.2%)\n",
      "  Read 1:    13,875,928 bp\n",
      "  Read 2:    13,873,795 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51506 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t657\t0.0\t2\t0 640 17\n",
      "26\t50634\t0.0\t2\t48866 1725 43\n",
      "27\t142\t0.0\t2\t0 138 4\n",
      "28\t41\t0.0\t2\t0 0 41\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51481 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t790\t0.0\t2\t0 768 22\n",
      "26\t50391\t0.0\t2\t49315 1055 21\n",
      "27\t225\t0.0\t2\t0 218 7\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S286_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S286_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S286_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S286_71_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S286_70_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S286_71_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S286_70_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S286_71_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,574 reads  @    113.2 µs/read;   0.53 M reads/minute\n",
      "Finished in 0.18 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,574\n",
      "  Read 1 with adapter:                   1,569 (99.7%)\n",
      "  Read 2 with adapter:                   1,505 (95.6%)\n",
      "Pairs written (passing filters):         1,574 (100.0%)\n",
      "\n",
      "Total basepairs processed:       921,916 bp\n",
      "  Read 1:       464,119 bp\n",
      "  Read 2:       457,797 bp\n",
      "Total written (filtered):        842,024 bp (91.3%)\n",
      "  Read 1:       423,342 bp\n",
      "  Read 2:       418,682 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1569 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t26\t0.0\t2\t0 24 2\n",
      "26\t1532\t0.0\t2\t1450 79 3\n",
      "27\t9\t0.0\t2\t0 9\n",
      "28\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1505 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t31\t0.0\t2\t0 30 1\n",
      "26\t1455\t0.0\t2\t1385 64 6\n",
      "27\t6\t0.0\t2\t0 5 1\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S287_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S287_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S287_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S287_73_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S287_72_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S287_73_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S287_72_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S287_73_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        40,194 reads  @     76.7 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.08 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             40,194\n",
      "  Read 1 with adapter:                  40,157 (99.9%)\n",
      "  Read 2 with adapter:                  40,095 (99.8%)\n",
      "Pairs written (passing filters):        40,193 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,703,625 bp\n",
      "  Read 1:    11,856,763 bp\n",
      "  Read 2:    11,846,862 bp\n",
      "Total written (filtered):     21,617,744 bp (91.2%)\n",
      "  Read 1:    10,812,602 bp\n",
      "  Read 2:    10,805,142 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 40157 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t427\t0.0\t2\t0 412 15\n",
      "26\t39494\t0.0\t2\t38291 1181 22\n",
      "27\t139\t0.0\t2\t0 135 4\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40095 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t64\t0.0\t2\t0 0 64\n",
      "25\t789\t0.0\t2\t0 762 27\n",
      "26\t39077\t0.0\t2\t38244 805 28\n",
      "27\t163\t0.0\t2\t0 160 3\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S288_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S288_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S288_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S288_75_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S288_74_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S288_75_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S288_74_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S288_75_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,062 reads  @     69.9 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.64 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,062\n",
      "  Read 1 with adapter:                  51,991 (99.9%)\n",
      "  Read 2 with adapter:                  51,716 (99.3%)\n",
      "Pairs written (passing filters):        52,059 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,660,240 bp\n",
      "  Read 1:    15,357,717 bp\n",
      "  Read 2:    15,302,523 bp\n",
      "Total written (filtered):     27,964,355 bp (91.2%)\n",
      "  Read 1:    14,005,708 bp\n",
      "  Read 2:    13,958,647 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51991 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t702\t0.0\t2\t0 674 28\n",
      "26\t51067\t0.0\t2\t49238 1779 50\n",
      "27\t154\t0.0\t2\t0 150 4\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51716 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t859\t0.0\t2\t0 839 20\n",
      "26\t50545\t0.0\t2\t49262 1253 30\n",
      "27\t201\t0.0\t2\t0 196 5\n",
      "28\t35\t0.0\t2\t0 0 35\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S289_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S289_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S289_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S289_77_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S289_76_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S289_77_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S289_76_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S289_77_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        57,389 reads  @     88.8 µs/read;   0.68 M reads/minute\n",
      "Finished in 5.10 s (89 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,389\n",
      "  Read 1 with adapter:                  57,293 (99.8%)\n",
      "  Read 2 with adapter:                  57,287 (99.8%)\n",
      "Pairs written (passing filters):        57,389 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,853,108 bp\n",
      "  Read 1:    16,926,673 bp\n",
      "  Read 2:    16,926,435 bp\n",
      "Total written (filtered):     30,875,625 bp (91.2%)\n",
      "  Read 1:    15,437,617 bp\n",
      "  Read 2:    15,438,008 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57293 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t30\t0.0\t2\t0 0 30\n",
      "25\t716\t0.0\t2\t0 693 23\n",
      "26\t56369\t0.0\t2\t54501 1830 38\n",
      "27\t142\t0.0\t2\t0 135 7\n",
      "28\t36\t0.0\t2\t0 0 36\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57287 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t88\t0.0\t2\t0 0 88\n",
      "25\t1076\t0.0\t2\t0 1041 35\n",
      "26\t55919\t0.0\t2\t54640 1244 35\n",
      "27\t191\t0.0\t2\t0 184 7\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S290_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S290_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S290_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S290_79_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S290_78_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S290_79_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S290_78_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S290_79_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            71 reads  @    499.7 µs/read;   0.12 M reads/minute\n",
      "Finished in 0.04 s (537 us/read; 0.11 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 71\n",
      "  Read 1 with adapter:                      60 (84.5%)\n",
      "  Read 2 with adapter:                      60 (84.5%)\n",
      "Pairs written (passing filters):            71 (100.0%)\n",
      "\n",
      "Total basepairs processed:        41,890 bp\n",
      "  Read 1:        20,945 bp\n",
      "  Read 2:        20,945 bp\n",
      "Total written (filtered):         38,776 bp (92.6%)\n",
      "  Read 1:        19,388 bp\n",
      "  Read 2:        19,388 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 2\n",
      "26\t56\t0.0\t2\t53 2 1\n",
      "27\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 2\n",
      "26\t56\t0.0\t2\t54 2\n",
      "27\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S291_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S291_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S291_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S291_81_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S291_80_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S291_81_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S291_80_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S291_81_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        37,972 reads  @     79.6 µs/read;   0.75 M reads/minute\n",
      "Finished in 3.03 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             37,972\n",
      "  Read 1 with adapter:                  37,944 (99.9%)\n",
      "  Read 2 with adapter:                  37,899 (99.8%)\n",
      "Pairs written (passing filters):        37,972 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,402,398 bp\n",
      "  Read 1:    11,201,690 bp\n",
      "  Read 2:    11,200,708 bp\n",
      "Total written (filtered):     20,430,964 bp (91.2%)\n",
      "  Read 1:    10,215,143 bp\n",
      "  Read 2:    10,215,821 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 37944 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t229\t0.0\t2\t0 223 6\n",
      "26\t37485\t0.0\t2\t36255 1192 38\n",
      "27\t156\t0.0\t2\t0 151 5\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 37899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t571\t0.0\t2\t0 559 12\n",
      "26\t37129\t0.0\t2\t36385 710 34\n",
      "27\t138\t0.0\t2\t0 132 6\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S292_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S292_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S292_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S292_83_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S292_82_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S292_83_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S292_82_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S292_83_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,460 reads  @     72.1 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.79 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,460\n",
      "  Read 1 with adapter:                  52,397 (99.9%)\n",
      "  Read 2 with adapter:                  52,355 (99.8%)\n",
      "Pairs written (passing filters):        52,460 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,949,326 bp\n",
      "  Read 1:    15,475,695 bp\n",
      "  Read 2:    15,473,631 bp\n",
      "Total written (filtered):     28,226,577 bp (91.2%)\n",
      "  Read 1:    14,113,399 bp\n",
      "  Read 2:    14,113,178 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52397 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t343\t0.0\t2\t0 331 12\n",
      "26\t51743\t0.0\t2\t50190 1513 40\n",
      "27\t197\t0.0\t2\t0 191 6\n",
      "28\t87\t0.0\t2\t0 0 87\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52355 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t74\t0.0\t2\t0 0 74\n",
      "25\t843\t0.0\t2\t0 813 30\n",
      "26\t51235\t0.0\t2\t50134 1051 50\n",
      "27\t192\t0.0\t2\t0 187 5\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S293_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S293_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S293_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S293_85_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S293_84_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S293_85_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S293_84_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S293_85_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,289 reads  @     72.4 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.28 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,289\n",
      "  Read 1 with adapter:                  45,227 (99.9%)\n",
      "  Read 2 with adapter:                  45,227 (99.9%)\n",
      "Pairs written (passing filters):        45,289 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,717,370 bp\n",
      "  Read 1:    13,360,082 bp\n",
      "  Read 2:    13,357,288 bp\n",
      "Total written (filtered):     24,366,270 bp (91.2%)\n",
      "  Read 1:    12,184,239 bp\n",
      "  Read 2:    12,182,031 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45227 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t336\t0.0\t2\t0 326 10\n",
      "26\t44633\t0.0\t2\t43279 1317 37\n",
      "27\t163\t0.0\t2\t0 160 3\n",
      "28\t76\t0.0\t2\t0 0 76\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45227 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t676\t0.0\t2\t0 659 17\n",
      "26\t44330\t0.0\t2\t43446 861 23\n",
      "27\t151\t0.0\t2\t0 147 4\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S294_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S294_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S294_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S294_87_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S294_86_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S294_87_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S294_86_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S294_87_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:03        38,661 reads  @     84.6 µs/read;   0.71 M reads/minute\n",
      "Finished in 3.28 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,661\n",
      "  Read 1 with adapter:                  38,625 (99.9%)\n",
      "  Read 2 with adapter:                  38,459 (99.5%)\n",
      "Pairs written (passing filters):        38,661 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,790,159 bp\n",
      "  Read 1:    11,404,084 bp\n",
      "  Read 2:    11,386,075 bp\n",
      "Total written (filtered):     20,786,536 bp (91.2%)\n",
      "  Read 1:    10,399,908 bp\n",
      "  Read 2:    10,386,628 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38625 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t265\t0.0\t2\t0 257 8\n",
      "26\t38162\t0.0\t2\t36979 1149 34\n",
      "27\t133\t0.0\t2\t0 130 3\n",
      "28\t47\t0.0\t2\t0 0 47\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38459 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t43\t0.0\t2\t0 0 43\n",
      "25\t568\t0.0\t2\t0 557 11\n",
      "26\t37687\t0.0\t2\t36843 811 33\n",
      "27\t155\t0.0\t2\t0 152 3\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S295_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S295_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S295_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S295_89_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S295_88_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S295_89_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S295_88_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S295_89_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,494 reads  @     90.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 3.75 s (90 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,494\n",
      "  Read 1 with adapter:                  41,439 (99.9%)\n",
      "  Read 2 with adapter:                  39,034 (94.1%)\n",
      "Pairs written (passing filters):        41,492 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,960,452 bp\n",
      "  Read 1:    12,240,389 bp\n",
      "  Read 2:    11,720,063 bp\n",
      "Total written (filtered):     21,868,502 bp (91.3%)\n",
      "  Read 1:    11,162,582 bp\n",
      "  Read 2:    10,705,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41439 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t331\t0.0\t2\t0 317 14\n",
      "26\t40905\t0.0\t2\t39614 1263 28\n",
      "27\t132\t0.0\t2\t0 128 4\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39034 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t762\t0.0\t2\t0 739 23\n",
      "26\t38076\t0.0\t2\t37246 800 30\n",
      "27\t131\t0.0\t2\t0 127 4\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S296_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S296_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S296_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S296_91_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S296_90_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S296_91_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S296_90_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S296_91_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        43,814 reads  @     97.0 µs/read;   0.62 M reads/minute\n",
      "Finished in 4.25 s (97 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,814\n",
      "  Read 1 with adapter:                  43,753 (99.9%)\n",
      "  Read 2 with adapter:                  43,650 (99.6%)\n",
      "Pairs written (passing filters):        43,812 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,834,630 bp\n",
      "  Read 1:    12,924,656 bp\n",
      "  Read 2:    12,909,974 bp\n",
      "Total written (filtered):     23,562,480 bp (91.2%)\n",
      "  Read 1:    11,786,747 bp\n",
      "  Read 2:    11,775,733 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43753 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t395\t0.0\t2\t0 386 9\n",
      "26\t43144\t0.0\t2\t41724 1371 49\n",
      "27\t140\t0.0\t2\t0 136 4\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43650 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t724\t0.0\t2\t0 706 18\n",
      "26\t42655\t0.0\t2\t41679 941 35\n",
      "27\t189\t0.0\t2\t0 183 6\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S297_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S297_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S297_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S297_93_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S297_92_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S297_93_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S297_92_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S297_93_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        49,274 reads  @     91.3 µs/read;   0.66 M reads/minute\n",
      "Finished in 4.50 s (91 us/read; 0.66 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,274\n",
      "  Read 1 with adapter:                  49,204 (99.9%)\n",
      "  Read 2 with adapter:                  49,181 (99.8%)\n",
      "Pairs written (passing filters):        49,274 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,068,066 bp\n",
      "  Read 1:    14,535,679 bp\n",
      "  Read 2:    14,532,387 bp\n",
      "Total written (filtered):     26,511,187 bp (91.2%)\n",
      "  Read 1:    13,256,558 bp\n",
      "  Read 2:    13,254,629 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t377\t0.0\t2\t0 362 15\n",
      "26\t48602\t0.0\t2\t47042 1527 33\n",
      "27\t144\t0.0\t2\t0 136 8\n",
      "28\t53\t0.0\t2\t0 0 53\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49181 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t74\t0.0\t2\t0 0 74\n",
      "25\t1026\t0.0\t2\t0 994 32\n",
      "26\t47871\t0.0\t2\t46739 1104 28\n",
      "27\t194\t0.0\t2\t0 186 8\n",
      "28\t16\t0.0\t2\t0 0 16\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S298_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S298_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S298_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S298_95_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S298_94_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S298_95_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S298_94_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S298_95_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,281 reads  @     82.8 µs/read;   0.72 M reads/minute\n",
      "Finished in 4.00 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,281\n",
      "  Read 1 with adapter:                  48,223 (99.9%)\n",
      "  Read 2 with adapter:                  48,198 (99.8%)\n",
      "Pairs written (passing filters):        48,281 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,484,038 bp\n",
      "  Read 1:    14,242,599 bp\n",
      "  Read 2:    14,241,439 bp\n",
      "Total written (filtered):     25,977,894 bp (91.2%)\n",
      "  Read 1:    12,988,935 bp\n",
      "  Read 2:    12,988,959 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48223 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t354\t0.0\t2\t0 344 10\n",
      "26\t47643\t0.0\t2\t46202 1398 43\n",
      "27\t148\t0.0\t2\t0 143 5\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48198 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t752\t0.0\t2\t0 734 18\n",
      "26\t47227\t0.0\t2\t46268 934 25\n",
      "27\t146\t0.0\t2\t0 141 5\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S299_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S299_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S299_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S299_97_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S299_96_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S299_97_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S299_96_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S299_97_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,914 reads  @     81.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.42 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,914\n",
      "  Read 1 with adapter:                  41,841 (99.8%)\n",
      "  Read 2 with adapter:                  41,831 (99.8%)\n",
      "Pairs written (passing filters):        41,914 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,724,849 bp\n",
      "  Read 1:    12,361,234 bp\n",
      "  Read 2:    12,363,615 bp\n",
      "Total written (filtered):     22,549,939 bp (91.2%)\n",
      "  Read 1:    11,273,397 bp\n",
      "  Read 2:    11,276,542 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41841 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t321\t0.0\t2\t0 310 11\n",
      "26\t41220\t0.0\t2\t39816 1372 32\n",
      "27\t192\t0.0\t2\t0 185 7\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41831 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t635\t0.0\t2\t0 623 12\n",
      "26\t40958\t0.0\t2\t40130 795 33\n",
      "27\t162\t0.0\t2\t0 154 8\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S300_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S300_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S300_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S300_99_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S300_98_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S300_99_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S300_98_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S300_99_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,071 reads  @     76.8 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.77 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,071\n",
      "  Read 1 with adapter:                  61,940 (99.8%)\n",
      "  Read 2 with adapter:                  61,885 (99.7%)\n",
      "Pairs written (passing filters):        62,071 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,608,322 bp\n",
      "  Read 1:    18,300,356 bp\n",
      "  Read 2:    18,307,966 bp\n",
      "Total written (filtered):     33,389,879 bp (91.2%)\n",
      "  Read 1:    16,690,016 bp\n",
      "  Read 2:    16,699,863 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 61940 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t435\t0.0\t2\t0 423 12\n",
      "26\t61132\t0.0\t2\t59055 2005 72\n",
      "27\t263\t0.0\t2\t0 248 15\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 61885 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t90\t0.0\t2\t0 0 90\n",
      "25\t1032\t0.0\t2\t0 1008 24\n",
      "26\t60472\t0.0\t2\t58973 1434 65\n",
      "27\t277\t0.0\t2\t0 268 9\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S301_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S301_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S301_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S301_101_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S301_100_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S301_101_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S301_100_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S301_101_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,457 reads  @     74.0 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.89 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,457\n",
      "  Read 1 with adapter:                  52,355 (99.8%)\n",
      "  Read 2 with adapter:                  52,336 (99.8%)\n",
      "Pairs written (passing filters):        52,457 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,942,322 bp\n",
      "  Read 1:    15,469,464 bp\n",
      "  Read 2:    15,472,858 bp\n",
      "Total written (filtered):     28,221,075 bp (91.2%)\n",
      "  Read 1:    14,108,189 bp\n",
      "  Read 2:    14,112,886 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52355 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t358\t0.0\t2\t0 348 10\n",
      "26\t51601\t0.0\t2\t49802 1755 44\n",
      "27\t257\t0.0\t2\t0 248 9\n",
      "28\t106\t0.0\t2\t0 0 106\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t68\t0.0\t2\t0 0 68\n",
      "25\t858\t0.0\t2\t0 836 22\n",
      "26\t51185\t0.0\t2\t50101 1053 31\n",
      "27\t220\t0.0\t2\t0 213 7\n",
      "28\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S302_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S302_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S302_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S302_103_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S302_102_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S302_103_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S302_102_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S302_103_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,891 reads  @     75.8 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.48 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,891\n",
      "  Read 1 with adapter:                  45,825 (99.9%)\n",
      "  Read 2 with adapter:                  45,641 (99.5%)\n",
      "Pairs written (passing filters):        45,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,052,303 bp\n",
      "  Read 1:    13,535,309 bp\n",
      "  Read 2:    13,516,994 bp\n",
      "Total written (filtered):     24,674,515 bp (91.2%)\n",
      "  Read 1:    12,343,524 bp\n",
      "  Read 2:    12,330,991 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45825 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t261\t0.0\t2\t0 248 13\n",
      "26\t45247\t0.0\t2\t43850 1368 29\n",
      "27\t211\t0.0\t2\t0 205 6\n",
      "28\t82\t0.0\t2\t0 0 82\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45641 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t716\t0.0\t2\t0 702 14\n",
      "26\t44702\t0.0\t2\t43736 924 42\n",
      "27\t157\t0.0\t2\t0 150 7\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S303_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S303_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S303_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S303_105_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S303_104_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S303_105_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S303_104_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S303_105_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,902 reads  @     82.9 µs/read;   0.72 M reads/minute\n",
      "Finished in 3.89 s (83 us/read; 0.72 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,902\n",
      "  Read 1 with adapter:                  46,785 (99.8%)\n",
      "  Read 2 with adapter:                  46,442 (99.0%)\n",
      "Pairs written (passing filters):        46,902 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,589,426 bp\n",
      "  Read 1:    13,825,372 bp\n",
      "  Read 2:    13,764,054 bp\n",
      "Total written (filtered):     25,166,435 bp (91.2%)\n",
      "  Read 1:    12,608,986 bp\n",
      "  Read 2:    12,557,449 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46785 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t323\t0.0\t2\t0 309 14\n",
      "26\t46165\t0.0\t2\t44675 1446 44\n",
      "27\t187\t0.0\t2\t0 182 5\n",
      "28\t83\t0.0\t2\t0 0 83\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46442 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t69\t0.0\t2\t0 0 69\n",
      "25\t919\t0.0\t2\t0 896 23\n",
      "26\t45292\t0.0\t2\t44331 935 26\n",
      "27\t154\t0.0\t2\t0 146 8\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S304_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S304_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S304_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S304_107_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S304_106_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S304_107_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S304_106_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S304_107_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,124 reads  @     82.7 µs/read;   0.73 M reads/minute\n",
      "Finished in 3.57 s (83 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,124\n",
      "  Read 1 with adapter:                  42,967 (99.6%)\n",
      "  Read 2 with adapter:                  42,980 (99.7%)\n",
      "Pairs written (passing filters):        43,124 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,406,526 bp\n",
      "  Read 1:    12,698,317 bp\n",
      "  Read 2:    12,708,209 bp\n",
      "Total written (filtered):     23,172,647 bp (91.2%)\n",
      "  Read 1:    11,581,191 bp\n",
      "  Read 2:    11,591,456 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 42967 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t316\t0.0\t2\t0 304 12\n",
      "26\t42352\t0.0\t2\t41031 1288 33\n",
      "27\t206\t0.0\t2\t0 197 9\n",
      "28\t70\t0.0\t2\t0 0 70\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 42980 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t73\t0.0\t2\t0 0 73\n",
      "25\t770\t0.0\t2\t0 748 22\n",
      "26\t41961\t0.0\t2\t40965 960 36\n",
      "27\t163\t0.0\t2\t0 160 3\n",
      "28\t13\t0.0\t2\t0 0 13\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S305_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S305_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S305_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S305_109_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S305_108_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S305_109_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S305_108_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S305_109_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,232 reads  @     79.7 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.24 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,232\n",
      "  Read 1 with adapter:                  53,085 (99.7%)\n",
      "  Read 2 with adapter:                  53,092 (99.7%)\n",
      "Pairs written (passing filters):        53,232 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,386,302 bp\n",
      "  Read 1:    15,686,331 bp\n",
      "  Read 2:    15,699,971 bp\n",
      "Total written (filtered):     28,627,195 bp (91.2%)\n",
      "  Read 1:    14,306,442 bp\n",
      "  Read 2:    14,320,753 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53085 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t476\t0.0\t2\t0 459 17\n",
      "26\t52399\t0.0\t2\t50501 1832 66\n",
      "27\t153\t0.0\t2\t0 149 4\n",
      "28\t29\t0.0\t2\t0 0 29\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53092 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t123\t0.0\t2\t0 0 123\n",
      "25\t1158\t0.0\t2\t0 1127 31\n",
      "26\t51589\t0.0\t2\t50301 1248 40\n",
      "27\t214\t0.0\t2\t0 203 11\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S306_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S306_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S306_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S306_111_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S306_110_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S306_111_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S306_110_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S306_111_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,786 reads  @     76.6 µs/read;   0.78 M reads/minute\n",
      "Finished in 4.28 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,786\n",
      "  Read 1 with adapter:                  55,518 (99.5%)\n",
      "  Read 2 with adapter:                  55,694 (99.8%)\n",
      "Pairs written (passing filters):        55,786 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,862,309 bp\n",
      "  Read 1:    16,405,724 bp\n",
      "  Read 2:    16,456,585 bp\n",
      "Total written (filtered):     29,971,629 bp (91.2%)\n",
      "  Read 1:    14,962,359 bp\n",
      "  Read 2:    15,009,270 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55518 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t373\t0.0\t2\t0 361 12\n",
      "26\t54821\t0.0\t2\t53303 1474 44\n",
      "27\t246\t0.0\t2\t0 242 4\n",
      "28\t45\t0.0\t2\t0 0 45\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55694 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t811\t0.0\t2\t0 797 14\n",
      "26\t54618\t0.0\t2\t53535 1027 56\n",
      "27\t168\t0.0\t2\t0 161 7\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S307_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S307_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S307_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S307_113_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S307_112_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S307_113_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S307_112_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S307_113_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        51,513 reads  @     80.4 µs/read;   0.75 M reads/minute\n",
      "Finished in 4.14 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,513\n",
      "  Read 1 with adapter:                  51,436 (99.9%)\n",
      "  Read 2 with adapter:                  51,401 (99.8%)\n",
      "Pairs written (passing filters):        51,513 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,391,135 bp\n",
      "  Read 1:    15,195,766 bp\n",
      "  Read 2:    15,195,369 bp\n",
      "Total written (filtered):     27,718,112 bp (91.2%)\n",
      "  Read 1:    13,858,444 bp\n",
      "  Read 2:    13,859,668 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51436 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t344\t0.0\t2\t0 337 7\n",
      "26\t50802\t0.0\t2\t49154 1606 42\n",
      "27\t182\t0.0\t2\t0 178 4\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51401 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t795\t0.0\t2\t0 767 28\n",
      "26\t50361\t0.0\t2\t49310 1008 43\n",
      "27\t172\t0.0\t2\t0 167 5\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S308_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S308_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S308_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S308_115_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S308_114_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S308_115_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S308_114_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S308_115_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        55,438 reads  @     80.9 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.49 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             55,438\n",
      "  Read 1 with adapter:                  55,360 (99.9%)\n",
      "  Read 2 with adapter:                  55,328 (99.8%)\n",
      "Pairs written (passing filters):        55,438 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,706,228 bp\n",
      "  Read 1:    16,353,867 bp\n",
      "  Read 2:    16,352,361 bp\n",
      "Total written (filtered):     29,829,189 bp (91.2%)\n",
      "  Read 1:    14,914,533 bp\n",
      "  Read 2:    14,914,656 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 55360 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t376\t0.0\t2\t0 365 11\n",
      "26\t54643\t0.0\t2\t52930 1665 48\n",
      "27\t196\t0.0\t2\t0 189 7\n",
      "28\t111\t0.0\t2\t0 0 111\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 55328 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t77\t0.0\t2\t0 0 77\n",
      "25\t895\t0.0\t2\t0 862 33\n",
      "26\t54132\t0.0\t2\t52806 1284 42\n",
      "27\t222\t0.0\t2\t0 215 7\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S309_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S309_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S309_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S309_117_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S309_116_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S309_117_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S309_116_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S309_117_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        43,179 reads  @     72.6 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.14 s (73 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             43,179\n",
      "  Read 1 with adapter:                  43,099 (99.8%)\n",
      "  Read 2 with adapter:                  43,081 (99.8%)\n",
      "Pairs written (passing filters):        43,179 (100.0%)\n",
      "\n",
      "Total basepairs processed:    25,469,344 bp\n",
      "  Read 1:    12,737,186 bp\n",
      "  Read 2:    12,732,158 bp\n",
      "Total written (filtered):     23,229,208 bp (91.2%)\n",
      "  Read 1:    11,616,604 bp\n",
      "  Read 2:    11,612,604 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 43099 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t22\t0.0\t2\t0 0 22\n",
      "25\t262\t0.0\t2\t0 255 7\n",
      "26\t42573\t0.0\t2\t41217 1325 31\n",
      "27\t170\t0.0\t2\t0 166 4\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 43081 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t618\t0.0\t2\t0 606 12\n",
      "26\t42232\t0.0\t2\t41372 835 25\n",
      "27\t172\t0.0\t2\t0 163 9\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S310_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S310_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S310_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S310_119_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S310_118_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S310_119_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S310_118_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S310_119_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,392 reads  @     78.2 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.24 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,392\n",
      "  Read 1 with adapter:                  41,322 (99.8%)\n",
      "  Read 2 with adapter:                  41,253 (99.7%)\n",
      "Pairs written (passing filters):        41,392 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,409,584 bp\n",
      "  Read 1:    12,208,505 bp\n",
      "  Read 2:    12,201,079 bp\n",
      "Total written (filtered):     22,263,188 bp (91.2%)\n",
      "  Read 1:    11,134,098 bp\n",
      "  Read 2:    11,129,090 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41322 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t283\t0.0\t2\t0 275 8\n",
      "26\t40757\t0.0\t2\t39467 1258 32\n",
      "27\t174\t0.0\t2\t0 169 5\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 41253 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t623\t0.0\t2\t0 606 17\n",
      "26\t40418\t0.0\t2\t39486 890 42\n",
      "27\t150\t0.0\t2\t0 141 9\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S311_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S311_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S311_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S311_121_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S311_120_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S311_121_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S311_120_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S311_121_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,850 reads  @     77.2 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.78 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,850\n",
      "  Read 1 with adapter:                  48,780 (99.9%)\n",
      "  Read 2 with adapter:                  48,201 (98.7%)\n",
      "Pairs written (passing filters):        48,850 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,705,818 bp\n",
      "  Read 1:    14,410,215 bp\n",
      "  Read 2:    14,295,603 bp\n",
      "Total written (filtered):     26,185,161 bp (91.2%)\n",
      "  Read 1:    13,141,879 bp\n",
      "  Read 2:    13,043,282 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48780 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t300\t0.0\t2\t0 285 15\n",
      "26\t48153\t0.0\t2\t46609 1507 37\n",
      "27\t166\t0.0\t2\t0 159 7\n",
      "28\t128\t0.0\t2\t0 0 128\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t60\t0.0\t2\t0 0 60\n",
      "25\t977\t0.0\t2\t0 947 30\n",
      "26\t46974\t0.0\t2\t45916 1018 40\n",
      "27\t188\t0.0\t2\t0 175 13\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S312_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S312_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S312_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S312_123_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S312_122_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S312_123_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S312_122_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S312_123_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,837 reads  @     77.5 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.71 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,837\n",
      "  Read 1 with adapter:                  47,762 (99.8%)\n",
      "  Read 2 with adapter:                  47,718 (99.8%)\n",
      "Pairs written (passing filters):        47,835 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,215,558 bp\n",
      "  Read 1:    14,110,294 bp\n",
      "  Read 2:    14,105,264 bp\n",
      "Total written (filtered):     25,733,228 bp (91.2%)\n",
      "  Read 1:    12,867,921 bp\n",
      "  Read 2:    12,865,307 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47762 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t327\t0.0\t2\t0 314 13\n",
      "26\t47119\t0.0\t2\t45614 1465 40\n",
      "27\t178\t0.0\t2\t0 171 7\n",
      "28\t112\t0.0\t2\t0 0 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47718 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t61\t0.0\t2\t0 0 61\n",
      "25\t770\t0.0\t2\t0 747 23\n",
      "26\t46710\t0.0\t2\t45644 1022 44\n",
      "27\t173\t0.0\t2\t0 166 7\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S313_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S313_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S313_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S313_125_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S313_124_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S313_125_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S313_124_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S313_125_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,060 reads  @     75.6 µs/read;   0.79 M reads/minute\n",
      "Finished in 4.01 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,060\n",
      "  Read 1 with adapter:                  52,959 (99.8%)\n",
      "  Read 2 with adapter:                  52,947 (99.8%)\n",
      "Pairs written (passing filters):        53,060 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,299,771 bp\n",
      "  Read 1:    15,650,829 bp\n",
      "  Read 2:    15,648,942 bp\n",
      "Total written (filtered):     28,547,305 bp (91.2%)\n",
      "  Read 1:    14,273,971 bp\n",
      "  Read 2:    14,273,334 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52959 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t393\t0.0\t2\t0 378 15\n",
      "26\t52256\t0.0\t2\t50637 1578 41\n",
      "27\t191\t0.0\t2\t0 187 4\n",
      "28\t91\t0.0\t2\t0 0 91\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52947 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t86\t0.0\t2\t0 0 86\n",
      "25\t1078\t0.0\t2\t0 1054 24\n",
      "26\t51559\t0.0\t2\t50398 1128 33\n",
      "27\t212\t0.0\t2\t0 207 5\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S314_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S314_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S314_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S314_127_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S314_126_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S314_127_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S314_126_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S314_127_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,187 reads  @     73.0 µs/read;   0.82 M reads/minute\n",
      "Finished in 4.18 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,187\n",
      "  Read 1 with adapter:                  57,100 (99.8%)\n",
      "  Read 2 with adapter:                  57,083 (99.8%)\n",
      "Pairs written (passing filters):        57,187 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,739,169 bp\n",
      "  Read 1:    16,870,143 bp\n",
      "  Read 2:    16,869,026 bp\n",
      "Total written (filtered):     30,771,537 bp (91.2%)\n",
      "  Read 1:    15,385,820 bp\n",
      "  Read 2:    15,385,717 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57100 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t33\t0.0\t2\t0 0 33\n",
      "25\t479\t0.0\t2\t0 461 18\n",
      "26\t56364\t0.0\t2\t54581 1741 42\n",
      "27\t180\t0.0\t2\t0 177 3\n",
      "28\t44\t0.0\t2\t0 0 44\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57083 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t79\t0.0\t2\t0 0 79\n",
      "25\t891\t0.0\t2\t0 870 21\n",
      "26\t55925\t0.0\t2\t54679 1205 41\n",
      "27\t176\t0.0\t2\t0 171 5\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S315_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S315_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S315_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S315_129_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S315_128_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S315_129_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S315_128_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S315_129_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,259 reads  @     68.9 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.19 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,259\n",
      "  Read 1 with adapter:                  45,854 (99.1%)\n",
      "  Read 2 with adapter:                  46,189 (99.8%)\n",
      "Pairs written (passing filters):        46,259 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,265,604 bp\n",
      "  Read 1:    13,619,261 bp\n",
      "  Read 2:    13,646,343 bp\n",
      "Total written (filtered):     24,873,155 bp (91.2%)\n",
      "  Read 1:    12,427,093 bp\n",
      "  Read 2:    12,446,062 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45854 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t326\t0.0\t2\t0 311 15\n",
      "26\t45247\t0.0\t2\t43883 1321 43\n",
      "27\t164\t0.0\t2\t0 156 8\n",
      "28\t90\t0.0\t2\t0 0 90\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46189 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t720\t0.0\t2\t0 706 14\n",
      "26\t45207\t0.0\t2\t44320 861 26\n",
      "27\t201\t0.0\t2\t0 194 7\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S316_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S316_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S316_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S316_131_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S316_130_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S316_131_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S316_130_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S316_131_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,078 reads  @     73.3 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.31 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,078\n",
      "  Read 1 with adapter:                  45,006 (99.8%)\n",
      "  Read 2 with adapter:                  44,978 (99.8%)\n",
      "Pairs written (passing filters):        45,078 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,591,866 bp\n",
      "  Read 1:    13,295,524 bp\n",
      "  Read 2:    13,296,342 bp\n",
      "Total written (filtered):     24,253,040 bp (91.2%)\n",
      "  Read 1:    12,125,494 bp\n",
      "  Read 2:    12,127,546 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45006 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t357\t0.0\t2\t0 347 10\n",
      "26\t44414\t0.0\t2\t43087 1295 32\n",
      "27\t131\t0.0\t2\t0 123 8\n",
      "28\t77\t0.0\t2\t0 0 77\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44978 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t749\t0.0\t2\t0 731 18\n",
      "26\t43983\t0.0\t2\t43027 922 34\n",
      "27\t179\t0.0\t2\t0 170 9\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S317_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S317_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S317_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S317_133_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S317_132_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S317_133_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S317_132_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S317_133_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,274 reads  @     68.9 µs/read;   0.87 M reads/minute\n",
      "Finished in 3.33 s (69 us/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,274\n",
      "  Read 1 with adapter:                  48,180 (99.8%)\n",
      "  Read 2 with adapter:                  48,182 (99.8%)\n",
      "Pairs written (passing filters):        48,274 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,474,831 bp\n",
      "  Read 1:    14,236,197 bp\n",
      "  Read 2:    14,238,634 bp\n",
      "Total written (filtered):     25,970,188 bp (91.2%)\n",
      "  Read 1:    12,983,625 bp\n",
      "  Read 2:    12,986,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48180 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t321\t0.0\t2\t0 308 13\n",
      "26\t47635\t0.0\t2\t46128 1468 39\n",
      "27\t151\t0.0\t2\t0 146 5\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48182 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t59\t0.0\t2\t0 0 59\n",
      "25\t786\t0.0\t2\t0 772 14\n",
      "26\t47128\t0.0\t2\t46135 970 23\n",
      "27\t175\t0.0\t2\t0 167 8\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S318_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S318_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S318_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S318_135_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S318_134_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S318_135_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S318_134_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S318_135_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        32,681 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 2.41 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,681\n",
      "  Read 1 with adapter:                  32,592 (99.7%)\n",
      "  Read 2 with adapter:                  32,594 (99.7%)\n",
      "Pairs written (passing filters):        32,681 (100.0%)\n",
      "\n",
      "Total basepairs processed:    19,271,096 bp\n",
      "  Read 1:     9,634,378 bp\n",
      "  Read 2:     9,636,718 bp\n",
      "Total written (filtered):     17,576,780 bp (91.2%)\n",
      "  Read 1:     8,787,035 bp\n",
      "  Read 2:     8,789,745 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 32592 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t269\t0.0\t2\t0 260 9\n",
      "26\t32110\t0.0\t2\t31058 1023 29\n",
      "27\t142\t0.0\t2\t0 136 6\n",
      "28\t55\t0.0\t2\t0 0 55\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 32594 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t56\t0.0\t2\t0 0 56\n",
      "25\t518\t0.0\t2\t0 502 16\n",
      "26\t31882\t0.0\t2\t31124 728 30\n",
      "27\t117\t0.0\t2\t0 113 4\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S319_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S319_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S319_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S319_137_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S319_136_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S319_137_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S319_136_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S319_137_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,331 reads  @     68.2 µs/read;   0.88 M reads/minute\n",
      "Finished in 2.68 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,331\n",
      "  Read 1 with adapter:                  39,261 (99.8%)\n",
      "  Read 2 with adapter:                  38,893 (98.9%)\n",
      "Pairs written (passing filters):        39,331 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,131,739 bp\n",
      "  Read 1:    11,598,798 bp\n",
      "  Read 2:    11,532,941 bp\n",
      "Total written (filtered):     21,100,728 bp (91.2%)\n",
      "  Read 1:    10,578,203 bp\n",
      "  Read 2:    10,522,525 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39261 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t296\t0.0\t2\t0 293 3\n",
      "26\t38834\t0.0\t2\t37730 1070 34\n",
      "27\t89\t0.0\t2\t0 87 2\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38893 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t836\t0.0\t2\t0 816 20\n",
      "26\t37868\t0.0\t2\t37095 752 21\n",
      "27\t92\t0.0\t2\t0 86 6\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S320_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S320_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S320_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S320_139_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S320_138_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S320_139_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S320_138_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S320_139_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        38,311 reads  @     64.0 µs/read;   0.94 M reads/minute\n",
      "Finished in 2.46 s (64 us/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             38,311\n",
      "  Read 1 with adapter:                  38,246 (99.8%)\n",
      "  Read 2 with adapter:                  38,169 (99.6%)\n",
      "Pairs written (passing filters):        38,311 (100.0%)\n",
      "\n",
      "Total basepairs processed:    22,595,996 bp\n",
      "  Read 1:    11,301,261 bp\n",
      "  Read 2:    11,294,735 bp\n",
      "Total written (filtered):     20,609,833 bp (91.2%)\n",
      "  Read 1:    10,306,891 bp\n",
      "  Read 2:    10,302,942 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 38246 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t266\t0.0\t2\t0 256 10\n",
      "26\t37751\t0.0\t2\t36641 1076 34\n",
      "27\t142\t0.0\t2\t0 133 9\n",
      "28\t68\t0.0\t2\t0 0 68\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 38169 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t660\t0.0\t2\t0 646 14\n",
      "26\t37309\t0.0\t2\t36552 717 40\n",
      "27\t145\t0.0\t2\t0 140 5\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S321_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S321_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S321_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S321_141_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S321_140_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S321_141_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S321_140_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S321_141_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,827 reads  @     70.6 µs/read;   0.85 M reads/minute\n",
      "Finished in 4.01 s (71 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,827\n",
      "  Read 1 with adapter:                  56,707 (99.8%)\n",
      "  Read 2 with adapter:                  56,695 (99.8%)\n",
      "Pairs written (passing filters):        56,827 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,514,863 bp\n",
      "  Read 1:    16,757,621 bp\n",
      "  Read 2:    16,757,242 bp\n",
      "Total written (filtered):     30,567,643 bp (91.2%)\n",
      "  Read 1:    15,283,278 bp\n",
      "  Read 2:    15,284,365 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56707 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t423\t0.0\t2\t0 411 12\n",
      "26\t55914\t0.0\t2\t54085 1794 35\n",
      "27\t212\t0.0\t2\t0 203 9\n",
      "28\t122\t0.0\t2\t0 0 122\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56695 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t102\t0.0\t2\t0 0 102\n",
      "25\t1201\t0.0\t2\t0 1168 33\n",
      "26\t55190\t0.0\t2\t53933 1215 42\n",
      "27\t192\t0.0\t2\t0 185 7\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S322_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S322_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S322_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S322_143_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S322_142_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S322_143_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S322_142_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S322_143_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8=---------] 00:00:04        62,287 reads  @     68.4 µs/read;   0.88 M reads/minute\n",
      "Finished in 4.27 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             62,287\n",
      "  Read 1 with adapter:                  62,201 (99.9%)\n",
      "  Read 2 with adapter:                  62,139 (99.8%)\n",
      "Pairs written (passing filters):        62,287 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,747,239 bp\n",
      "  Read 1:    18,374,143 bp\n",
      "  Read 2:    18,373,096 bp\n",
      "Total written (filtered):     33,515,556 bp (91.2%)\n",
      "  Read 1:    16,757,252 bp\n",
      "  Read 2:    16,758,304 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 62201 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t534\t0.0\t2\t0 517 17\n",
      "26\t61402\t0.0\t2\t59404 1935 63\n",
      "27\t187\t0.0\t2\t0 182 5\n",
      "28\t42\t0.0\t2\t0 0 42\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 62139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t83\t0.0\t2\t0 0 83\n",
      "25\t924\t0.0\t2\t0 897 27\n",
      "26\t60867\t0.0\t2\t59399 1439 29\n",
      "27\t262\t0.0\t2\t0 257 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S323_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S323_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S323_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S323_145_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S323_144_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S323_145_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S323_144_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S323_145_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,039 reads  @     72.9 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.28 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,039\n",
      "  Read 1 with adapter:                  44,982 (99.9%)\n",
      "  Read 2 with adapter:                  44,933 (99.8%)\n",
      "Pairs written (passing filters):        45,039 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,571,297 bp\n",
      "  Read 1:    13,285,885 bp\n",
      "  Read 2:    13,285,412 bp\n",
      "Total written (filtered):     24,234,730 bp (91.2%)\n",
      "  Read 1:    12,116,903 bp\n",
      "  Read 2:    12,117,827 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 44982 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t638\t0.0\t2\t0 621 17\n",
      "26\t44169\t0.0\t2\t42637 1490 42\n",
      "27\t114\t0.0\t2\t0 109 5\n",
      "28\t24\t0.0\t2\t0 0 24\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 44933 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t64\t0.0\t2\t0 0 64\n",
      "25\t714\t0.0\t2\t0 695 19\n",
      "26\t43989\t0.0\t2\t43113 847 29\n",
      "27\t163\t0.0\t2\t0 158 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S324_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S324_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S324_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S324_147_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S324_146_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S324_147_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S324_146_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S324_147_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,353 reads  @     71.0 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.22 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,353\n",
      "  Read 1 with adapter:                  45,290 (99.9%)\n",
      "  Read 2 with adapter:                  45,236 (99.7%)\n",
      "Pairs written (passing filters):        45,353 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,757,024 bp\n",
      "  Read 1:    13,378,788 bp\n",
      "  Read 2:    13,378,236 bp\n",
      "Total written (filtered):     24,404,642 bp (91.2%)\n",
      "  Read 1:    12,201,817 bp\n",
      "  Read 2:    12,202,825 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45290 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t700\t0.0\t2\t0 681 19\n",
      "26\t44379\t0.0\t2\t42845 1496 38\n",
      "27\t139\t0.0\t2\t0 132 7\n",
      "28\t34\t0.0\t2\t0 0 34\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45236 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t62\t0.0\t2\t0 0 62\n",
      "25\t814\t0.0\t2\t0 791 23\n",
      "26\t44155\t0.0\t2\t43090 1028 37\n",
      "27\t197\t0.0\t2\t0 193 4\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S325_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S325_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S325_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S325_149_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S325_148_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S325_149_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S325_148_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S325_149_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,901 reads  @     70.3 µs/read;   0.85 M reads/minute\n",
      "Finished in 3.65 s (70 us/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,901\n",
      "  Read 1 with adapter:                  51,845 (99.9%)\n",
      "  Read 2 with adapter:                  51,787 (99.8%)\n",
      "Pairs written (passing filters):        51,901 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,614,417 bp\n",
      "  Read 1:    15,309,074 bp\n",
      "  Read 2:    15,305,343 bp\n",
      "Total written (filtered):     27,921,062 bp (91.2%)\n",
      "  Read 1:    13,961,437 bp\n",
      "  Read 2:    13,959,625 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51845 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 0 21\n",
      "25\t504\t0.0\t2\t0 489 15\n",
      "26\t51140\t0.0\t2\t49495 1598 47\n",
      "27\t147\t0.0\t2\t0 142 5\n",
      "28\t33\t0.0\t2\t0 0 33\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51787 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t81\t0.0\t2\t0 0 81\n",
      "25\t805\t0.0\t2\t0 778 27\n",
      "26\t50688\t0.0\t2\t49573 1085 30\n",
      "27\t203\t0.0\t2\t0 202 1\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S326_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S326_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S326_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S326_151_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S326_150_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S326_151_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S326_150_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S326_151_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         3,933 reads  @    113.9 µs/read;   0.53 M reads/minute\n",
      "Finished in 0.45 s (115 us/read; 0.52 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              3,933\n",
      "  Read 1 with adapter:                   3,904 (99.3%)\n",
      "  Read 2 with adapter:                   3,740 (95.1%)\n",
      "Pairs written (passing filters):         3,933 (100.0%)\n",
      "\n",
      "Total basepairs processed:     2,310,635 bp\n",
      "  Read 1:     1,160,048 bp\n",
      "  Read 2:     1,150,587 bp\n",
      "Total written (filtered):      2,111,947 bp (91.4%)\n",
      "  Read 1:     1,058,553 bp\n",
      "  Read 2:     1,053,394 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 3904 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t44\t0.0\t2\t0 43 1\n",
      "26\t3807\t0.0\t2\t3629 176 2\n",
      "27\t47\t0.0\t2\t0 45 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 3740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t60\t0.0\t2\t0 56 4\n",
      "26\t3655\t0.0\t2\t3519 127 9\n",
      "27\t9\t0.0\t2\t0 8 1\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S327_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S327_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S327_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S327_153_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S327_152_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S327_153_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S327_152_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S327_153_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,007 reads  @     75.5 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.48 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,007\n",
      "  Read 1 with adapter:                  45,914 (99.8%)\n",
      "  Read 2 with adapter:                  45,732 (99.4%)\n",
      "Pairs written (passing filters):        46,005 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,096,983 bp\n",
      "  Read 1:    13,568,196 bp\n",
      "  Read 2:    13,528,787 bp\n",
      "Total written (filtered):     24,714,793 bp (91.2%)\n",
      "  Read 1:    12,374,191 bp\n",
      "  Read 2:    12,340,602 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45914 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t531\t0.0\t2\t0 511 20\n",
      "26\t45129\t0.0\t2\t43708 1385 36\n",
      "27\t158\t0.0\t2\t0 148 10\n",
      "28\t67\t0.0\t2\t0 0 67\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45732 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t57\t0.0\t2\t0 0 57\n",
      "25\t936\t0.0\t2\t0 912 24\n",
      "26\t44550\t0.0\t2\t43618 899 33\n",
      "27\t175\t0.0\t2\t0 167 8\n",
      "28\t14\t0.0\t2\t0 0 14\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S328_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S328_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S328_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S328_155_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S328_154_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S328_155_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S328_154_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S328_155_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:01        27,317 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 1.86 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,317\n",
      "  Read 1 with adapter:                  27,278 (99.9%)\n",
      "  Read 2 with adapter:                  27,097 (99.2%)\n",
      "Pairs written (passing filters):        27,316 (100.0%)\n",
      "\n",
      "Total basepairs processed:    16,084,197 bp\n",
      "  Read 1:     8,058,035 bp\n",
      "  Read 2:     8,026,162 bp\n",
      "Total written (filtered):     14,671,016 bp (91.2%)\n",
      "  Read 1:     7,348,899 bp\n",
      "  Read 2:     7,322,117 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 27278 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t18\t0.0\t2\t0 0 18\n",
      "25\t446\t0.0\t2\t0 435 11\n",
      "26\t26711\t0.0\t2\t25692 987 32\n",
      "27\t85\t0.0\t2\t0 84 1\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 27097 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t534\t0.0\t2\t0 519 15\n",
      "26\t26376\t0.0\t2\t25711 642 23\n",
      "27\t113\t0.0\t2\t0 110 3\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S329_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S329_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S329_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S329_157_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S329_156_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S329_157_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S329_156_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S329_157_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,768 reads  @     85.8 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.10 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,768\n",
      "  Read 1 with adapter:                  47,698 (99.9%)\n",
      "  Read 2 with adapter:                  47,665 (99.8%)\n",
      "Pairs written (passing filters):        47,768 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,177,601 bp\n",
      "  Read 1:    14,089,015 bp\n",
      "  Read 2:    14,088,586 bp\n",
      "Total written (filtered):     25,699,519 bp (91.2%)\n",
      "  Read 1:    12,849,313 bp\n",
      "  Read 2:    12,850,206 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47698 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 0 16\n",
      "25\t583\t0.0\t2\t0 567 16\n",
      "26\t46953\t0.0\t2\t45481 1418 54\n",
      "27\t123\t0.0\t2\t0 119 4\n",
      "28\t23\t0.0\t2\t0 0 23\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47665 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t74\t0.0\t2\t0 0 74\n",
      "25\t950\t0.0\t2\t0 933 17\n",
      "26\t46464\t0.0\t2\t45487 946 31\n",
      "27\t166\t0.0\t2\t0 163 3\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S330_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S330_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S330_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S330_159_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S330_158_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S330_159_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S330_158_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S330_159_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00            13 reads  @   2471.8 µs/read;   0.02 M reads/minute\n",
      "Finished in 0.03 s (2659 us/read; 0.02 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 13\n",
      "  Read 1 with adapter:                       0 (0.0%)\n",
      "  Read 2 with adapter:                       0 (0.0%)\n",
      "Pairs written (passing filters):            13 (100.0%)\n",
      "\n",
      "Total basepairs processed:         7,670 bp\n",
      "  Read 1:         3,835 bp\n",
      "  Read 2:         3,835 bp\n",
      "Total written (filtered):          7,670 bp (100.0%)\n",
      "  Read 1:         3,835 bp\n",
      "  Read 2:         3,835 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 0 times\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S331_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S331_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S331_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S331_161_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S331_160_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S331_161_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S331_160_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S331_161_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,241 reads  @     71.5 µs/read;   0.84 M reads/minute\n",
      "Finished in 4.02 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,241\n",
      "  Read 1 with adapter:                  56,169 (99.9%)\n",
      "  Read 2 with adapter:                  56,185 (99.9%)\n",
      "Pairs written (passing filters):        56,240 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,180,291 bp\n",
      "  Read 1:    16,590,570 bp\n",
      "  Read 2:    16,589,721 bp\n",
      "Total written (filtered):     30,259,485 bp (91.2%)\n",
      "  Read 1:    15,129,992 bp\n",
      "  Read 2:    15,129,493 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56169 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t399\t0.0\t2\t0 388 11\n",
      "26\t55433\t0.0\t2\t53927 1469 37\n",
      "27\t232\t0.0\t2\t0 219 13\n",
      "28\t73\t0.0\t2\t0 0 73\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56185 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t698\t0.0\t2\t0 681 17\n",
      "26\t55243\t0.0\t2\t54181 1030 32\n",
      "27\t168\t0.0\t2\t0 161 7\n",
      "28\t25\t0.0\t2\t0 0 25\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S332_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S332_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S332_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S332_163_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S332_162_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S332_163_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S332_162_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S332_163_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        56,804 reads  @     70.0 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.98 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,804\n",
      "  Read 1 with adapter:                  56,740 (99.9%)\n",
      "  Read 2 with adapter:                  56,703 (99.8%)\n",
      "Pairs written (passing filters):        56,804 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,509,015 bp\n",
      "  Read 1:    16,756,779 bp\n",
      "  Read 2:    16,752,236 bp\n",
      "Total written (filtered):     30,560,398 bp (91.2%)\n",
      "  Read 1:    15,281,649 bp\n",
      "  Read 2:    15,278,749 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56740 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 0 49\n",
      "25\t408\t0.0\t2\t0 397 11\n",
      "26\t55982\t0.0\t2\t54278 1660 44\n",
      "27\t206\t0.0\t2\t0 203 3\n",
      "28\t95\t0.0\t2\t0 0 95\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56703 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t870\t0.0\t2\t0 848 22\n",
      "26\t55561\t0.0\t2\t54372 1142 47\n",
      "27\t197\t0.0\t2\t0 189 8\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S333_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S333_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S333_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S333_165_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S333_164_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S333_165_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S333_164_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S333_165_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        41,062 reads  @     76.7 µs/read;   0.78 M reads/minute\n",
      "Finished in 3.15 s (77 us/read; 0.78 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             41,062\n",
      "  Read 1 with adapter:                  41,002 (99.9%)\n",
      "  Read 2 with adapter:                  40,988 (99.8%)\n",
      "Pairs written (passing filters):        41,062 (100.0%)\n",
      "\n",
      "Total basepairs processed:    24,221,789 bp\n",
      "  Read 1:    12,112,652 bp\n",
      "  Read 2:    12,109,137 bp\n",
      "Total written (filtered):     22,090,526 bp (91.2%)\n",
      "  Read 1:    11,046,661 bp\n",
      "  Read 2:    11,043,865 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 41002 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t291\t0.0\t2\t0 282 9\n",
      "26\t40445\t0.0\t2\t39254 1156 35\n",
      "27\t158\t0.0\t2\t0 154 4\n",
      "28\t72\t0.0\t2\t0 0 72\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 40988 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t493\t0.0\t2\t0 479 14\n",
      "26\t40353\t0.0\t2\t39447 869 37\n",
      "27\t111\t0.0\t2\t0 104 7\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S334_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S334_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S334_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S334_167_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S334_166_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S334_167_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S334_166_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S334_167_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        56,550 reads  @     77.8 µs/read;   0.77 M reads/minute\n",
      "Finished in 4.41 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,550\n",
      "  Read 1 with adapter:                  56,460 (99.8%)\n",
      "  Read 2 with adapter:                  56,453 (99.8%)\n",
      "Pairs written (passing filters):        56,550 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,362,219 bp\n",
      "  Read 1:    16,681,818 bp\n",
      "  Read 2:    16,680,401 bp\n",
      "Total written (filtered):     30,427,369 bp (91.2%)\n",
      "  Read 1:    15,214,031 bp\n",
      "  Read 2:    15,213,338 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56460 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t429\t0.0\t2\t0 420 9\n",
      "26\t55724\t0.0\t2\t53949 1736 39\n",
      "27\t206\t0.0\t2\t0 198 8\n",
      "28\t63\t0.0\t2\t0 0 63\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56453 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t39\t0.0\t2\t0 0 39\n",
      "25\t868\t0.0\t2\t0 846 22\n",
      "26\t55323\t0.0\t2\t54207 1080 36\n",
      "27\t215\t0.0\t2\t0 204 11\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S335_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S335_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S335_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S335_169_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S335_168_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S335_169_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S335_168_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S335_169_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        47,654 reads  @     89.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.29 s (90 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,654\n",
      "  Read 1 with adapter:                  47,595 (99.9%)\n",
      "  Read 2 with adapter:                  47,510 (99.7%)\n",
      "Pairs written (passing filters):        47,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,107,506 bp\n",
      "  Read 1:    14,057,624 bp\n",
      "  Read 2:    14,049,882 bp\n",
      "Total written (filtered):     25,635,416 bp (91.2%)\n",
      "  Read 1:    12,820,412 bp\n",
      "  Read 2:    12,815,004 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47595 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t24\t0.0\t2\t0 0 24\n",
      "25\t468\t0.0\t2\t0 457 11\n",
      "26\t46897\t0.0\t2\t45302 1546 49\n",
      "27\t154\t0.0\t2\t0 149 5\n",
      "28\t52\t0.0\t2\t0 0 52\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47510 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t54\t0.0\t2\t0 0 54\n",
      "25\t745\t0.0\t2\t0 732 13\n",
      "26\t46521\t0.0\t2\t45439 1054 28\n",
      "27\t178\t0.0\t2\t0 173 5\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S336_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S336_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S336_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S336_171_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S336_170_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S336_171_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S336_170_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S336_171_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        56,251 reads  @     95.1 µs/read;   0.63 M reads/minute\n",
      "Finished in 5.35 s (95 us/read; 0.63 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             56,251\n",
      "  Read 1 with adapter:                  56,164 (99.8%)\n",
      "  Read 2 with adapter:                  56,139 (99.8%)\n",
      "Pairs written (passing filters):        56,251 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,181,221 bp\n",
      "  Read 1:    16,592,992 bp\n",
      "  Read 2:    16,588,229 bp\n",
      "Total written (filtered):     30,262,506 bp (91.2%)\n",
      "  Read 1:    15,133,025 bp\n",
      "  Read 2:    15,129,481 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 56164 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t542\t0.0\t2\t0 528 14\n",
      "26\t55349\t0.0\t2\t53604 1698 47\n",
      "27\t197\t0.0\t2\t0 187 10\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 56139 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t83\t0.0\t2\t0 0 83\n",
      "25\t950\t0.0\t2\t0 920 30\n",
      "26\t54866\t0.0\t2\t53564 1255 47\n",
      "27\t230\t0.0\t2\t0 223 7\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S337_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S337_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S337_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S337_173_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S337_172_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S337_173_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S337_172_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S337_173_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:05        57,327 reads  @     88.0 µs/read;   0.68 M reads/minute\n",
      "Finished in 5.05 s (88 us/read; 0.68 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,327\n",
      "  Read 1 with adapter:                  57,264 (99.9%)\n",
      "  Read 2 with adapter:                  57,185 (99.8%)\n",
      "Pairs written (passing filters):        57,326 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,819,334 bp\n",
      "  Read 1:    16,910,373 bp\n",
      "  Read 2:    16,908,961 bp\n",
      "Total written (filtered):     30,844,509 bp (91.2%)\n",
      "  Read 1:    15,421,790 bp\n",
      "  Read 2:    15,422,719 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t501\t0.0\t2\t0 491 10\n",
      "26\t56497\t0.0\t2\t54642 1808 47\n",
      "27\t184\t0.0\t2\t0 179 5\n",
      "28\t50\t0.0\t2\t0 0 50\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57185 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t86\t0.0\t2\t0 0 86\n",
      "25\t938\t0.0\t2\t0 916 22\n",
      "26\t55905\t0.0\t2\t54593 1262 50\n",
      "27\t239\t0.0\t2\t0 234 5\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S338_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S338_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S338_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S338_175_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S338_174_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S338_175_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S338_174_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S338_175_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        34,908 reads  @     81.6 µs/read;   0.74 M reads/minute\n",
      "Finished in 2.85 s (82 us/read; 0.73 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             34,908\n",
      "  Read 1 with adapter:                  34,858 (99.9%)\n",
      "  Read 2 with adapter:                  34,842 (99.8%)\n",
      "Pairs written (passing filters):        34,908 (100.0%)\n",
      "\n",
      "Total basepairs processed:    20,595,071 bp\n",
      "  Read 1:    10,297,595 bp\n",
      "  Read 2:    10,297,476 bp\n",
      "Total written (filtered):     18,783,544 bp (91.2%)\n",
      "  Read 1:     9,391,463 bp\n",
      "  Read 2:     9,392,081 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 34858 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 0 17\n",
      "25\t325\t0.0\t2\t0 316 9\n",
      "26\t34371\t0.0\t2\t33343 1011 17\n",
      "27\t107\t0.0\t2\t0 104 3\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 34842 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t559\t0.0\t2\t0 540 19\n",
      "26\t34103\t0.0\t2\t33388 696 19\n",
      "27\t118\t0.0\t2\t0 110 8\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S339_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S339_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S339_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S339_177_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S339_176_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S339_177_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S339_176_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S339_177_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:01        16,931 reads  @     79.7 µs/read;   0.75 M reads/minute\n",
      "Finished in 1.35 s (80 us/read; 0.75 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,931\n",
      "  Read 1 with adapter:                  16,883 (99.7%)\n",
      "  Read 2 with adapter:                  16,899 (99.8%)\n",
      "Pairs written (passing filters):        16,931 (100.0%)\n",
      "\n",
      "Total basepairs processed:     9,986,511 bp\n",
      "  Read 1:     4,993,149 bp\n",
      "  Read 2:     4,993,362 bp\n",
      "Total written (filtered):      9,108,400 bp (91.2%)\n",
      "  Read 1:     4,554,228 bp\n",
      "  Read 2:     4,554,172 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 16883 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 0 11\n",
      "25\t125\t0.0\t2\t0 122 3\n",
      "26\t16654\t0.0\t2\t16135 504 15\n",
      "27\t76\t0.0\t2\t0 72 4\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 16899 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t245\t0.0\t2\t0 241 4\n",
      "26\t16564\t0.0\t2\t16209 342 13\n",
      "27\t67\t0.0\t2\t0 63 4\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S340_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S340_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S340_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S340_179_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S340_178_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S340_179_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S340_178_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S340_179_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        48,161 reads  @     75.7 µs/read;   0.79 M reads/minute\n",
      "Finished in 3.65 s (76 us/read; 0.79 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             48,161\n",
      "  Read 1 with adapter:                  47,887 (99.4%)\n",
      "  Read 2 with adapter:                  48,065 (99.8%)\n",
      "Pairs written (passing filters):        48,161 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,402,222 bp\n",
      "  Read 1:    14,197,907 bp\n",
      "  Read 2:    14,204,315 bp\n",
      "Total written (filtered):     25,908,210 bp (91.2%)\n",
      "  Read 1:    12,952,932 bp\n",
      "  Read 2:    12,955,278 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47887 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t345\t0.0\t2\t0 332 13\n",
      "26\t47227\t0.0\t2\t45691 1490 46\n",
      "27\t204\t0.0\t2\t0 197 7\n",
      "28\t69\t0.0\t2\t0 0 69\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48065 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t52\t0.0\t2\t0 0 52\n",
      "25\t771\t0.0\t2\t0 749 22\n",
      "26\t47037\t0.0\t2\t45928 1073 36\n",
      "27\t188\t0.0\t2\t0 181 7\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S341_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S341_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S341_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S341_181_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S341_180_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S341_181_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S341_180_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S341_181_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,098 reads  @     73.4 µs/read;   0.82 M reads/minute\n",
      "Finished in 3.61 s (73 us/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,098\n",
      "  Read 1 with adapter:                  48,832 (99.5%)\n",
      "  Read 2 with adapter:                  48,992 (99.8%)\n",
      "Pairs written (passing filters):        49,098 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,954,337 bp\n",
      "  Read 1:    14,475,487 bp\n",
      "  Read 2:    14,478,850 bp\n",
      "Total written (filtered):     26,411,541 bp (91.2%)\n",
      "  Read 1:    13,205,868 bp\n",
      "  Read 2:    13,205,673 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 48832 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t25\t0.0\t2\t0 0 25\n",
      "25\t328\t0.0\t2\t0 315 13\n",
      "26\t48200\t0.0\t2\t46659 1504 37\n",
      "27\t193\t0.0\t2\t0 185 8\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 48992 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t53\t0.0\t2\t0 0 53\n",
      "25\t696\t0.0\t2\t0 678 18\n",
      "26\t48058\t0.0\t2\t47022 994 42\n",
      "27\t183\t0.0\t2\t0 176 7\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S342_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S342_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S342_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S342_183_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S342_182_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S342_183_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S342_182_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S342_183_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,653 reads  @     71.6 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.84 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,653\n",
      "  Read 1 with adapter:                  53,385 (99.5%)\n",
      "  Read 2 with adapter:                  53,561 (99.8%)\n",
      "Pairs written (passing filters):        53,653 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,643,042 bp\n",
      "  Read 1:    15,817,105 bp\n",
      "  Read 2:    15,825,937 bp\n",
      "Total written (filtered):     28,863,179 bp (91.2%)\n",
      "  Read 1:    14,429,107 bp\n",
      "  Read 2:    14,434,072 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53385 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t382\t0.0\t2\t0 371 11\n",
      "26\t52634\t0.0\t2\t50925 1663 46\n",
      "27\t232\t0.0\t2\t0 227 5\n",
      "28\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53561 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t834\t0.0\t2\t0 807 27\n",
      "26\t52450\t0.0\t2\t51253 1163 34\n",
      "27\t209\t0.0\t2\t0 197 12\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S343_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S343_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S343_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S343_185_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S343_184_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S343_185_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S343_184_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S343_185_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        50,891 reads  @     77.7 µs/read;   0.77 M reads/minute\n",
      "Finished in 3.96 s (78 us/read; 0.77 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,891\n",
      "  Read 1 with adapter:                  50,817 (99.9%)\n",
      "  Read 2 with adapter:                  50,786 (99.8%)\n",
      "Pairs written (passing filters):        50,890 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,019,647 bp\n",
      "  Read 1:    15,011,640 bp\n",
      "  Read 2:    15,008,007 bp\n",
      "Total written (filtered):     27,378,430 bp (91.2%)\n",
      "  Read 1:    13,690,155 bp\n",
      "  Read 2:    13,688,275 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50817 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t356\t0.0\t2\t0 344 12\n",
      "26\t50143\t0.0\t2\t48526 1584 33\n",
      "27\t198\t0.0\t2\t0 188 10\n",
      "28\t93\t0.0\t2\t0 0 93\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50786 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t76\t0.0\t2\t0 0 76\n",
      "25\t774\t0.0\t2\t0 749 25\n",
      "26\t49720\t0.0\t2\t48554 1138 28\n",
      "27\t210\t0.0\t2\t0 199 11\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S344_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S344_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S344_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S344_187_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S344_186_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S344_187_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S344_186_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S344_187_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        51,787 reads  @     88.9 µs/read;   0.67 M reads/minute\n",
      "Finished in 4.61 s (89 us/read; 0.67 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,787\n",
      "  Read 1 with adapter:                  51,710 (99.9%)\n",
      "  Read 2 with adapter:                  51,681 (99.8%)\n",
      "Pairs written (passing filters):        51,787 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,551,016 bp\n",
      "  Read 1:    15,277,154 bp\n",
      "  Read 2:    15,273,862 bp\n",
      "Total written (filtered):     27,863,788 bp (91.2%)\n",
      "  Read 1:    13,932,822 bp\n",
      "  Read 2:    13,930,966 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51710 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t36\t0.0\t2\t0 0 36\n",
      "25\t377\t0.0\t2\t0 364 13\n",
      "26\t51050\t0.0\t2\t49379 1631 40\n",
      "27\t173\t0.0\t2\t0 170 3\n",
      "28\t74\t0.0\t2\t0 0 74\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51681 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t73\t0.0\t2\t0 0 73\n",
      "25\t882\t0.0\t2\t0 853 29\n",
      "26\t50517\t0.0\t2\t49262 1221 34\n",
      "27\t200\t0.0\t2\t0 194 6\n",
      "28\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S345_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S345_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S345_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S345_189_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S345_188_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S345_189_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S345_188_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S345_189_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        46,561 reads  @     85.4 µs/read;   0.70 M reads/minute\n",
      "Finished in 3.98 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             46,561\n",
      "  Read 1 with adapter:                  46,485 (99.8%)\n",
      "  Read 2 with adapter:                  46,453 (99.8%)\n",
      "Pairs written (passing filters):        46,561 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,468,822 bp\n",
      "  Read 1:    13,735,092 bp\n",
      "  Read 2:    13,733,730 bp\n",
      "Total written (filtered):     25,053,461 bp (91.2%)\n",
      "  Read 1:    12,526,782 bp\n",
      "  Read 2:    12,526,679 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 46485 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t407\t0.0\t2\t0 398 9\n",
      "26\t45905\t0.0\t2\t44317 1549 39\n",
      "27\t123\t0.0\t2\t0 120 3\n",
      "28\t21\t0.0\t2\t0 0 21\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 46453 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t71\t0.0\t2\t0 0 71\n",
      "25\t804\t0.0\t2\t0 774 30\n",
      "26\t45366\t0.0\t2\t44183 1151 32\n",
      "27\t205\t0.0\t2\t0 197 8\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S346_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S346_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S346_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S346_191_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S346_190_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S346_191_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S346_190_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S346_191_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,670 reads  @     81.5 µs/read;   0.74 M reads/minute\n",
      "Finished in 3.89 s (82 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,670\n",
      "  Read 1 with adapter:                  47,558 (99.8%)\n",
      "  Read 2 with adapter:                  47,573 (99.8%)\n",
      "Pairs written (passing filters):        47,670 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,120,413 bp\n",
      "  Read 1:    14,060,018 bp\n",
      "  Read 2:    14,060,395 bp\n",
      "Total written (filtered):     25,647,875 bp (91.2%)\n",
      "  Read 1:    12,823,623 bp\n",
      "  Read 2:    12,824,252 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47558 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t317\t0.0\t2\t0 306 11\n",
      "26\t46999\t0.0\t2\t45579 1379 41\n",
      "27\t176\t0.0\t2\t0 174 2\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47573 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t853\t0.0\t2\t0 833 20\n",
      "26\t46444\t0.0\t2\t45402 1003 39\n",
      "27\t166\t0.0\t2\t0 159 7\n",
      "28\t38\t0.0\t2\t0 0 38\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S347_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S347_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S347_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S347_193_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S347_192_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S347_193_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S347_192_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S347_193_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:04        47,722 reads  @     86.1 µs/read;   0.70 M reads/minute\n",
      "Finished in 4.11 s (86 us/read; 0.70 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,722\n",
      "  Read 1 with adapter:                  47,687 (99.9%)\n",
      "  Read 2 with adapter:                  47,626 (99.8%)\n",
      "Pairs written (passing filters):        47,722 (100.0%)\n",
      "\n",
      "Total basepairs processed:    28,152,021 bp\n",
      "  Read 1:    14,077,450 bp\n",
      "  Read 2:    14,074,571 bp\n",
      "Total written (filtered):     25,674,786 bp (91.2%)\n",
      "  Read 1:    12,837,863 bp\n",
      "  Read 2:    12,836,923 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47687 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t29\t0.0\t2\t0 0 29\n",
      "25\t591\t0.0\t2\t0 574 17\n",
      "26\t46777\t0.0\t2\t45264 1467 46\n",
      "27\t206\t0.0\t2\t0 198 8\n",
      "28\t84\t0.0\t2\t0 0 84\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47626 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t69\t0.0\t2\t0 0 69\n",
      "25\t699\t0.0\t2\t0 684 15\n",
      "26\t46660\t0.0\t2\t45601 1027 32\n",
      "27\t187\t0.0\t2\t0 182 5\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S348_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S348_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S348_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S348_195_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S348_194_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S348_195_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S348_194_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S348_195_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        53,458 reads  @     81.2 µs/read;   0.74 M reads/minute\n",
      "Finished in 4.34 s (81 us/read; 0.74 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,458\n",
      "  Read 1 with adapter:                  53,391 (99.9%)\n",
      "  Read 2 with adapter:                  53,347 (99.8%)\n",
      "Pairs written (passing filters):        53,458 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,537,705 bp\n",
      "  Read 1:    15,770,093 bp\n",
      "  Read 2:    15,767,612 bp\n",
      "Total written (filtered):     28,763,553 bp (91.2%)\n",
      "  Read 1:    14,382,209 bp\n",
      "  Read 2:    14,381,344 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53391 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t26\t0.0\t2\t0 0 26\n",
      "25\t653\t0.0\t2\t0 634 19\n",
      "26\t52393\t0.0\t2\t50650 1700 43\n",
      "27\t215\t0.0\t2\t0 209 6\n",
      "28\t104\t0.0\t2\t0 0 104\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53347 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t67\t0.0\t2\t0 0 67\n",
      "25\t859\t0.0\t2\t0 819 40\n",
      "26\t52188\t0.0\t2\t50864 1286 38\n",
      "27\t227\t0.0\t2\t0 219 8\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S349_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S349_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S349_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S349_197_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S349_196_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S349_197_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S349_196_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S349_197_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,409 reads  @     75.2 µs/read;   0.80 M reads/minute\n",
      "Finished in 4.32 s (75 us/read; 0.80 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,409\n",
      "  Read 1 with adapter:                  57,342 (99.9%)\n",
      "  Read 2 with adapter:                  57,273 (99.8%)\n",
      "Pairs written (passing filters):        57,409 (100.0%)\n",
      "\n",
      "Total basepairs processed:    33,864,239 bp\n",
      "  Read 1:    16,934,565 bp\n",
      "  Read 2:    16,929,674 bp\n",
      "Total written (filtered):     30,885,337 bp (91.2%)\n",
      "  Read 1:    15,443,930 bp\n",
      "  Read 2:    15,441,407 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57342 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t667\t0.0\t2\t0 641 26\n",
      "26\t56266\t0.0\t2\t54347 1875 44\n",
      "27\t272\t0.0\t2\t0 264 8\n",
      "28\t103\t0.0\t2\t0 0 103\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57273 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t63\t0.0\t2\t0 0 63\n",
      "25\t941\t0.0\t2\t0 918 23\n",
      "26\t56036\t0.0\t2\t54664 1339 33\n",
      "27\t230\t0.0\t2\t0 225 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S350_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S350_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S350_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S350_199_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S350_198_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S350_199_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S350_198_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S350_199_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        53,962 reads  @     69.9 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.78 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,962\n",
      "  Read 1 with adapter:                  53,900 (99.9%)\n",
      "  Read 2 with adapter:                  53,862 (99.8%)\n",
      "Pairs written (passing filters):        53,962 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,834,974 bp\n",
      "  Read 1:    15,917,538 bp\n",
      "  Read 2:    15,917,436 bp\n",
      "Total written (filtered):     29,034,212 bp (91.2%)\n",
      "  Read 1:    14,516,321 bp\n",
      "  Read 2:    14,517,891 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53900 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t598\t0.0\t2\t0 578 20\n",
      "26\t52921\t0.0\t2\t51173 1696 52\n",
      "27\t235\t0.0\t2\t0 227 8\n",
      "28\t118\t0.0\t2\t0 0 118\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53862 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t81\t0.0\t2\t0 0 81\n",
      "25\t949\t0.0\t2\t0 920 29\n",
      "26\t52595\t0.0\t2\t51340 1221 34\n",
      "27\t230\t0.0\t2\t0 221 9\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S351_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S351_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S351_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S351_201_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S351_200_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S351_201_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S351_200_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S351_201_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,596 reads  @     69.4 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.73 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,596\n",
      "  Read 1 with adapter:                  53,533 (99.9%)\n",
      "  Read 2 with adapter:                  53,496 (99.8%)\n",
      "Pairs written (passing filters):        53,596 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,618,645 bp\n",
      "  Read 1:    15,810,616 bp\n",
      "  Read 2:    15,808,029 bp\n",
      "Total written (filtered):     28,836,647 bp (91.2%)\n",
      "  Read 1:    14,418,716 bp\n",
      "  Read 2:    14,417,931 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53533 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t393\t0.0\t2\t0 377 16\n",
      "26\t52766\t0.0\t2\t50994 1734 38\n",
      "27\t237\t0.0\t2\t0 229 8\n",
      "28\t118\t0.0\t2\t0 0 118\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53496 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t72\t0.0\t2\t0 0 72\n",
      "25\t861\t0.0\t2\t0 838 23\n",
      "26\t52358\t0.0\t2\t51193 1132 33\n",
      "27\t203\t0.0\t2\t0 192 11\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S352_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S352_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S352_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S352_203_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S352_202_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S352_203_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S352_202_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S352_203_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        45,386 reads  @     73.8 µs/read;   0.81 M reads/minute\n",
      "Finished in 3.35 s (74 us/read; 0.81 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             45,386\n",
      "  Read 1 with adapter:                  45,336 (99.9%)\n",
      "  Read 2 with adapter:                  45,264 (99.7%)\n",
      "Pairs written (passing filters):        45,386 (100.0%)\n",
      "\n",
      "Total basepairs processed:    26,772,017 bp\n",
      "  Read 1:    13,388,863 bp\n",
      "  Read 2:    13,383,154 bp\n",
      "Total written (filtered):     24,417,169 bp (91.2%)\n",
      "  Read 1:    12,210,160 bp\n",
      "  Read 2:    12,207,009 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 45336 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t19\t0.0\t2\t0 0 19\n",
      "25\t335\t0.0\t2\t0 326 9\n",
      "26\t44734\t0.0\t2\t43252 1444 38\n",
      "27\t156\t0.0\t2\t0 153 3\n",
      "28\t92\t0.0\t2\t0 0 92\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 45264 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t66\t0.0\t2\t0 0 66\n",
      "25\t776\t0.0\t2\t0 751 25\n",
      "26\t44235\t0.0\t2\t43115 1071 49\n",
      "27\t185\t0.0\t2\t0 177 8\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S353_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S353_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S353_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S353_205_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S353_204_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S353_205_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S353_204_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S353_205_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,630 reads  @     72.2 µs/read;   0.83 M reads/minute\n",
      "Finished in 3.66 s (72 us/read; 0.83 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,630\n",
      "  Read 1 with adapter:                  50,564 (99.9%)\n",
      "  Read 2 with adapter:                  50,500 (99.7%)\n",
      "Pairs written (passing filters):        50,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,867,648 bp\n",
      "  Read 1:    14,935,684 bp\n",
      "  Read 2:    14,931,964 bp\n",
      "Total written (filtered):     27,240,784 bp (91.2%)\n",
      "  Read 1:    13,621,055 bp\n",
      "  Read 2:    13,619,729 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50564 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t387\t0.0\t2\t0 369 18\n",
      "26\t49849\t0.0\t2\t48144 1653 52\n",
      "27\t192\t0.0\t2\t0 185 7\n",
      "28\t108\t0.0\t2\t0 0 108\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50500 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t77\t0.0\t2\t0 0 77\n",
      "25\t853\t0.0\t2\t0 822 31\n",
      "26\t49343\t0.0\t2\t48121 1184 38\n",
      "27\t212\t0.0\t2\t0 199 13\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S354_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S354_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S354_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S354_207_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S354_206_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S354_207_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S354_206_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S354_207_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        49,602 reads  @     71.7 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.56 s (72 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             49,602\n",
      "  Read 1 with adapter:                  49,559 (99.9%)\n",
      "  Read 2 with adapter:                  49,511 (99.8%)\n",
      "Pairs written (passing filters):        49,602 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,262,435 bp\n",
      "  Read 1:    14,631,992 bp\n",
      "  Read 2:    14,630,443 bp\n",
      "Total written (filtered):     26,687,661 bp (91.2%)\n",
      "  Read 1:    13,343,725 bp\n",
      "  Read 2:    13,343,936 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 49559 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t426\t0.0\t2\t0 410 16\n",
      "26\t48933\t0.0\t2\t47242 1645 46\n",
      "27\t129\t0.0\t2\t0 124 5\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 49511 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t58\t0.0\t2\t0 0 58\n",
      "25\t871\t0.0\t2\t0 854 17\n",
      "26\t48386\t0.0\t2\t47206 1144 36\n",
      "27\t184\t0.0\t2\t0 177 7\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S355_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S355_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S355_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S355_209_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S355_208_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S355_209_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S355_208_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S355_209_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8=----------] 00:00:02        39,858 reads  @     64.4 µs/read;   0.93 M reads/minute\n",
      "Finished in 2.57 s (64 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             39,858\n",
      "  Read 1 with adapter:                  39,544 (99.2%)\n",
      "  Read 2 with adapter:                  39,802 (99.9%)\n",
      "Pairs written (passing filters):        39,858 (100.0%)\n",
      "\n",
      "Total basepairs processed:    23,442,570 bp\n",
      "  Read 1:    11,685,077 bp\n",
      "  Read 2:    11,757,493 bp\n",
      "Total written (filtered):     21,380,155 bp (91.2%)\n",
      "  Read 1:    10,656,971 bp\n",
      "  Read 2:    10,723,184 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 39544 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t273\t0.0\t2\t0 264 9\n",
      "26\t39004\t0.0\t2\t37706 1260 38\n",
      "27\t171\t0.0\t2\t0 167 4\n",
      "28\t64\t0.0\t2\t0 0 64\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 39802 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t619\t0.0\t2\t0 606 13\n",
      "26\t38979\t0.0\t2\t38093 858 28\n",
      "27\t156\t0.0\t2\t0 149 7\n",
      "28\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S356_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S356_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S356_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S356_211_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S356_210_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S356_211_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S356_210_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S356_211_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8<---------] 00:00:03        50,413 reads  @     71.2 µs/read;   0.84 M reads/minute\n",
      "Finished in 3.59 s (71 us/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             50,413\n",
      "  Read 1 with adapter:                  50,338 (99.9%)\n",
      "  Read 2 with adapter:                  50,289 (99.8%)\n",
      "Pairs written (passing filters):        50,413 (100.0%)\n",
      "\n",
      "Total basepairs processed:    29,736,540 bp\n",
      "  Read 1:    14,869,551 bp\n",
      "  Read 2:    14,866,989 bp\n",
      "Total written (filtered):     27,121,068 bp (91.2%)\n",
      "  Read 1:    13,560,839 bp\n",
      "  Read 2:    13,560,229 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 50338 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t34\t0.0\t2\t0 0 34\n",
      "25\t371\t0.0\t2\t0 355 16\n",
      "26\t49649\t0.0\t2\t47987 1613 49\n",
      "27\t205\t0.0\t2\t0 200 5\n",
      "28\t79\t0.0\t2\t0 0 79\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 50289 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t815\t0.0\t2\t0 789 26\n",
      "26\t49226\t0.0\t2\t48077 1122 27\n",
      "27\t175\t0.0\t2\t0 167 8\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S357_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S357_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S357_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S357_213_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S357_212_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S357_213_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S357_212_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S357_213_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,708 reads  @     68.3 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.67 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,708\n",
      "  Read 1 with adapter:                  53,605 (99.8%)\n",
      "  Read 2 with adapter:                  53,591 (99.8%)\n",
      "Pairs written (passing filters):        53,708 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,678,466 bp\n",
      "  Read 1:    15,839,890 bp\n",
      "  Read 2:    15,838,576 bp\n",
      "Total written (filtered):     28,892,156 bp (91.2%)\n",
      "  Read 1:    14,446,250 bp\n",
      "  Read 2:    14,445,906 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53605 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t390\t0.0\t2\t0 374 16\n",
      "26\t52908\t0.0\t2\t51119 1747 42\n",
      "27\t190\t0.0\t2\t0 184 6\n",
      "28\t86\t0.0\t2\t0 0 86\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53591 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 0 51\n",
      "25\t806\t0.0\t2\t0 788 18\n",
      "26\t52534\t0.0\t2\t51340 1145 49\n",
      "27\t188\t0.0\t2\t0 180 8\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S358_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S358_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S358_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S358_215_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S358_214_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S358_215_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S358_214_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S358_215_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,422 reads  @     67.5 µs/read;   0.89 M reads/minute\n",
      "Finished in 4.02 s (68 us/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,422\n",
      "  Read 1 with adapter:                  59,286 (99.8%)\n",
      "  Read 2 with adapter:                  59,311 (99.8%)\n",
      "Pairs written (passing filters):        59,422 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,046,659 bp\n",
      "  Read 1:    17,518,914 bp\n",
      "  Read 2:    17,527,745 bp\n",
      "Total written (filtered):     31,964,151 bp (91.2%)\n",
      "  Read 1:    15,977,588 bp\n",
      "  Read 2:    15,986,563 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59286 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t45\t0.0\t2\t0 0 45\n",
      "25\t445\t0.0\t2\t0 436 9\n",
      "26\t58477\t0.0\t2\t56482 1946 49\n",
      "27\t213\t0.0\t2\t0 207 6\n",
      "28\t106\t0.0\t2\t0 0 106\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59311 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t85\t0.0\t2\t0 0 85\n",
      "25\t984\t0.0\t2\t0 965 19\n",
      "26\t58019\t0.0\t2\t56682 1286 51\n",
      "27\t196\t0.0\t2\t0 189 7\n",
      "28\t27\t0.0\t2\t0 0 27\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S359_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S359_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S359_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S359_217_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S359_216_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S359_217_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S359_216_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S359_217_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        47,412 reads  @     64.8 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.08 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             47,412\n",
      "  Read 1 with adapter:                  47,354 (99.9%)\n",
      "  Read 2 with adapter:                  47,313 (99.8%)\n",
      "Pairs written (passing filters):        47,412 (100.0%)\n",
      "\n",
      "Total basepairs processed:    27,971,562 bp\n",
      "  Read 1:    13,986,282 bp\n",
      "  Read 2:    13,985,280 bp\n",
      "Total written (filtered):     25,511,019 bp (91.2%)\n",
      "  Read 1:    12,755,289 bp\n",
      "  Read 2:    12,755,730 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 47354 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t38\t0.0\t2\t0 0 38\n",
      "25\t367\t0.0\t2\t0 356 11\n",
      "26\t46773\t0.0\t2\t45194 1542 37\n",
      "27\t120\t0.0\t2\t0 117 3\n",
      "28\t56\t0.0\t2\t0 0 56\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 47313 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t742\t0.0\t2\t0 724 18\n",
      "26\t46346\t0.0\t2\t45346 980 20\n",
      "27\t148\t0.0\t2\t0 144 4\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S360_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S360_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S360_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S360_219_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S360_218_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S360_219_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S360_218_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S360_219_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        58,655 reads  @     61.9 µs/read;   0.97 M reads/minute\n",
      "Finished in 3.63 s (62 us/read; 0.97 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,655\n",
      "  Read 1 with adapter:                  58,552 (99.8%)\n",
      "  Read 2 with adapter:                  58,525 (99.8%)\n",
      "Pairs written (passing filters):        58,655 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,601,536 bp\n",
      "  Read 1:    17,302,765 bp\n",
      "  Read 2:    17,298,771 bp\n",
      "Total written (filtered):     31,558,471 bp (91.2%)\n",
      "  Read 1:    15,780,445 bp\n",
      "  Read 2:    15,778,026 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58552 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 0 28\n",
      "25\t403\t0.0\t2\t0 384 19\n",
      "26\t57806\t0.0\t2\t55821 1942 43\n",
      "27\t203\t0.0\t2\t0 198 5\n",
      "28\t112\t0.0\t2\t0 0 112\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58525 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t75\t0.0\t2\t0 0 75\n",
      "25\t998\t0.0\t2\t0 973 25\n",
      "26\t57217\t0.0\t2\t55811 1366 40\n",
      "27\t227\t0.0\t2\t0 213 14\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S361_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S361_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S361_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S361_221_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S361_220_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S361_221_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S361_220_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S361_221_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        54,304 reads  @     66.3 µs/read;   0.90 M reads/minute\n",
      "Finished in 3.61 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             54,304\n",
      "  Read 1 with adapter:                  54,204 (99.8%)\n",
      "  Read 2 with adapter:                  54,173 (99.8%)\n",
      "Pairs written (passing filters):        54,304 (100.0%)\n",
      "\n",
      "Total basepairs processed:    32,035,016 bp\n",
      "  Read 1:    16,019,662 bp\n",
      "  Read 2:    16,015,354 bp\n",
      "Total written (filtered):     29,218,130 bp (91.2%)\n",
      "  Read 1:    14,610,379 bp\n",
      "  Read 2:    14,607,751 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 54204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t42\t0.0\t2\t0 0 42\n",
      "25\t372\t0.0\t2\t0 367 5\n",
      "26\t53462\t0.0\t2\t51539 1880 43\n",
      "27\t221\t0.0\t2\t0 215 6\n",
      "28\t107\t0.0\t2\t0 0 107\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 54173 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t89\t0.0\t2\t0 0 89\n",
      "25\t913\t0.0\t2\t0 887 26\n",
      "26\t52981\t0.0\t2\t51751 1193 37\n",
      "27\t184\t0.0\t2\t0 179 5\n",
      "28\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S362_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S362_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S362_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S362_223_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S362_222_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S362_223_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S362_222_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S362_223_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        52,630 reads  @     64.7 µs/read;   0.93 M reads/minute\n",
      "Finished in 3.41 s (65 us/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             52,630\n",
      "  Read 1 with adapter:                  52,547 (99.8%)\n",
      "  Read 2 with adapter:                  52,515 (99.8%)\n",
      "Pairs written (passing filters):        52,630 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,049,572 bp\n",
      "  Read 1:    15,524,964 bp\n",
      "  Read 2:    15,524,608 bp\n",
      "Total written (filtered):     28,319,158 bp (91.2%)\n",
      "  Read 1:    14,159,075 bp\n",
      "  Read 2:    14,160,083 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 52547 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t27\t0.0\t2\t0 0 27\n",
      "25\t489\t0.0\t2\t0 480 9\n",
      "26\t51847\t0.0\t2\t50134 1665 48\n",
      "27\t158\t0.0\t2\t0 152 6\n",
      "28\t26\t0.0\t2\t0 0 26\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 52515 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t70\t0.0\t2\t0 0 70\n",
      "25\t914\t0.0\t2\t0 891 23\n",
      "26\t51345\t0.0\t2\t50238 1079 28\n",
      "27\t183\t0.0\t2\t0 178 5\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S363_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S363_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S363_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S363_225_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S363_224_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S363_225_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S363_224_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S363_225_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        53,276 reads  @     68.0 µs/read;   0.88 M reads/minute\n",
      "Finished in 3.63 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             53,276\n",
      "  Read 1 with adapter:                  53,203 (99.9%)\n",
      "  Read 2 with adapter:                  53,140 (99.7%)\n",
      "Pairs written (passing filters):        53,275 (100.0%)\n",
      "\n",
      "Total basepairs processed:    31,428,236 bp\n",
      "  Read 1:    15,716,031 bp\n",
      "  Read 2:    15,712,205 bp\n",
      "Total written (filtered):     28,664,063 bp (91.2%)\n",
      "  Read 1:    14,332,804 bp\n",
      "  Read 2:    14,331,259 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 53203 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t32\t0.0\t2\t0 0 32\n",
      "25\t507\t0.0\t2\t0 497 10\n",
      "26\t52456\t0.0\t2\t50779 1620 57\n",
      "27\t165\t0.0\t2\t0 158 7\n",
      "28\t43\t0.0\t2\t0 0 43\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 53140 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t66\t0.0\t2\t0 0 66\n",
      "25\t752\t0.0\t2\t0 733 19\n",
      "26\t52134\t0.0\t2\t51112 1006 16\n",
      "27\t186\t0.0\t2\t0 180 6\n",
      "28\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S364_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S364_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S364_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S364_227_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S364_226_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S364_227_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S364_226_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S364_227_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        58,285 reads  @     69.7 µs/read;   0.86 M reads/minute\n",
      "Finished in 4.07 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             58,285\n",
      "  Read 1 with adapter:                  58,216 (99.9%)\n",
      "  Read 2 with adapter:                  58,165 (99.8%)\n",
      "Pairs written (passing filters):        58,285 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,381,721 bp\n",
      "  Read 1:    17,192,992 bp\n",
      "  Read 2:    17,188,729 bp\n",
      "Total written (filtered):     31,356,849 bp (91.2%)\n",
      "  Read 1:    15,679,593 bp\n",
      "  Read 2:    15,677,256 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 58216 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t23\t0.0\t2\t0 0 23\n",
      "25\t484\t0.0\t2\t0 468 16\n",
      "26\t57453\t0.0\t2\t55460 1933 60\n",
      "27\t199\t0.0\t2\t0 192 7\n",
      "28\t57\t0.0\t2\t0 0 57\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 58165 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t83\t0.0\t2\t0 0 83\n",
      "25\t930\t0.0\t2\t0 900 30\n",
      "26\t56891\t0.0\t2\t55461 1389 41\n",
      "27\t243\t0.0\t2\t0 236 7\n",
      "28\t18\t0.0\t2\t0 0 18\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S365_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S365_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S365_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S365_229_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S365_228_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S365_229_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S365_228_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S365_229_L001_R2_001.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        59,367 reads  @     67.8 µs/read;   0.88 M reads/minute\n",
      "Finished in 4.03 s (68 us/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             59,367\n",
      "  Read 1 with adapter:                  59,298 (99.9%)\n",
      "  Read 2 with adapter:                  59,204 (99.7%)\n",
      "Pairs written (passing filters):        59,367 (100.0%)\n",
      "\n",
      "Total basepairs processed:    35,021,096 bp\n",
      "  Read 1:    17,512,704 bp\n",
      "  Read 2:    17,508,392 bp\n",
      "Total written (filtered):     31,941,214 bp (91.2%)\n",
      "  Read 1:    15,971,264 bp\n",
      "  Read 2:    15,969,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 59298 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t31\t0.0\t2\t0 0 31\n",
      "25\t550\t0.0\t2\t0 530 20\n",
      "26\t58453\t0.0\t2\t56499 1912 42\n",
      "27\t224\t0.0\t2\t0 219 5\n",
      "28\t40\t0.0\t2\t0 0 40\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 59204 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t938\t0.0\t2\t0 909 29\n",
      "26\t58002\t0.0\t2\t56686 1281 35\n",
      "27\t192\t0.0\t2\t0 186 6\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S366_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S366_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S366_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S366_231_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S366_230_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S366_231_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S366_230_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S366_231_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[8<----------] 00:00:00         1,236 reads  @     95.5 µs/read;   0.63 M reads/minute\n",
      "Finished in 0.12 s (98 us/read; 0.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,236\n",
      "  Read 1 with adapter:                   1,235 (99.9%)\n",
      "  Read 2 with adapter:                   1,235 (99.9%)\n",
      "Pairs written (passing filters):         1,236 (100.0%)\n",
      "\n",
      "Total basepairs processed:       729,240 bp\n",
      "  Read 1:       364,620 bp\n",
      "  Read 2:       364,620 bp\n",
      "Total written (filtered):        665,051 bp (91.2%)\n",
      "  Read 1:       332,520 bp\n",
      "  Read 2:       332,531 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 1235 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t11\t0.0\t2\t0 11\n",
      "26\t1217\t0.0\t2\t1165 46 6\n",
      "27\t5\t0.0\t2\t0 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 1235 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t2\t0.0\t2\t0 0 2\n",
      "25\t25\t0.0\t2\t0 23 2\n",
      "26\t1203\t0.0\t2\t1165 37 1\n",
      "27\t2\t0.0\t2\t0 2\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S367_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S367_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S367_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S367_233_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S367_232_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S367_233_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S367_232_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S367_233_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:03        51,511 reads  @     69.5 µs/read;   0.86 M reads/minute\n",
      "Finished in 3.58 s (70 us/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             51,511\n",
      "  Read 1 with adapter:                  51,407 (99.8%)\n",
      "  Read 2 with adapter:                  51,413 (99.8%)\n",
      "Pairs written (passing filters):        51,511 (100.0%)\n",
      "\n",
      "Total basepairs processed:    30,383,594 bp\n",
      "  Read 1:    15,189,773 bp\n",
      "  Read 2:    15,193,821 bp\n",
      "Total written (filtered):     27,711,370 bp (91.2%)\n",
      "  Read 1:    13,853,594 bp\n",
      "  Read 2:    13,857,776 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 51407 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t37\t0.0\t2\t0 0 37\n",
      "25\t712\t0.0\t2\t0 693 19\n",
      "26\t50380\t0.0\t2\t48642 1686 52\n",
      "27\t173\t0.0\t2\t0 168 5\n",
      "28\t105\t0.0\t2\t0 0 105\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 51413 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t65\t0.0\t2\t0 0 65\n",
      "25\t813\t0.0\t2\t0 788 25\n",
      "26\t50300\t0.0\t2\t49129 1144 27\n",
      "27\t220\t0.0\t2\t0 213 7\n",
      "28\t15\t0.0\t2\t0 0 15\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S368_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S368_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S368_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S368_235_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S368_234_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S368_235_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S368_234_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S368_235_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[ 8=---------] 00:00:04        61,047 reads  @     66.3 µs/read;   0.90 M reads/minute\n",
      "Finished in 4.05 s (66 us/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             61,047\n",
      "  Read 1 with adapter:                  60,965 (99.9%)\n",
      "  Read 2 with adapter:                  60,926 (99.8%)\n",
      "Pairs written (passing filters):        61,047 (100.0%)\n",
      "\n",
      "Total basepairs processed:    36,012,491 bp\n",
      "  Read 1:    18,006,531 bp\n",
      "  Read 2:    18,005,960 bp\n",
      "Total written (filtered):     32,844,940 bp (91.2%)\n",
      "  Read 1:    16,422,104 bp\n",
      "  Read 2:    16,422,836 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 60965 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t50\t0.0\t2\t0 0 50\n",
      "25\t858\t0.0\t2\t0 827 31\n",
      "26\t59810\t0.0\t2\t57523 2219 68\n",
      "27\t199\t0.0\t2\t0 188 11\n",
      "28\t48\t0.0\t2\t0 0 48\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 60926 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t97\t0.0\t2\t0 0 97\n",
      "25\t1118\t0.0\t2\t0 1073 45\n",
      "26\t59400\t0.0\t2\t57765 1592 43\n",
      "27\t262\t0.0\t2\t0 255 7\n",
      "28\t49\t0.0\t2\t0 0 49\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S369_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S369_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S369_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S369_237_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 2.10 with Python 3.6.10\n",
      "Command line parameters: --cores 1 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 1 -o /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S369_236_L001_R1_001.fastq.gz -p /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-ouxicsfi/S369_237_L001_R2_001.fastq.gz --front ^GGWACWGGWTGAACWGTWTAYCCYCC -G ^TANACYTCNGGRTGNCCRAARAAYCA /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S369_236_L001_R1_001.fastq.gz /var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-0ie5uv5m/4154d902-5de8-4e9a-a2e0-1be0ebb982f5/data/S369_237_L001_R2_001.fastq.gz\n",
      "Processing reads on 1 core in paired-end mode ...\n",
      "[ 8<---------] 00:00:04        57,748 reads  @     84.5 µs/read;   0.71 M reads/minute\n",
      "Finished in 4.88 s (85 us/read; 0.71 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             57,748\n",
      "  Read 1 with adapter:                  57,628 (99.8%)\n",
      "  Read 2 with adapter:                  57,607 (99.8%)\n",
      "Pairs written (passing filters):        57,748 (100.0%)\n",
      "\n",
      "Total basepairs processed:    34,059,910 bp\n",
      "  Read 1:    17,027,974 bp\n",
      "  Read 2:    17,031,936 bp\n",
      "Total written (filtered):     31,065,509 bp (91.2%)\n",
      "  Read 1:    15,530,409 bp\n",
      "  Read 2:    15,535,100 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GGWACWGGWTGAACWGTWTAYCCYCC; Type: anchored 5'; Length: 26; Trimmed: 57628 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t44\t0.0\t2\t0 0 44\n",
      "25\t886\t0.0\t2\t0 859 27\n",
      "26\t56517\t0.0\t2\t54457 2005 55\n",
      "27\t151\t0.0\t2\t0 144 7\n",
      "28\t30\t0.0\t2\t0 0 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: TANACYTCNGGRTGNCCRAARAAYCA; Type: anchored 5'; Length: 26; Trimmed: 57607 times\n",
      "\n",
      "No. of allowed errors:\n",
      "0-9 bp: 0; 10-19 bp: 1; 20-23 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t94\t0.0\t2\t0 0 94\n",
      "25\t998\t0.0\t2\t0 974 24\n",
      "26\t56286\t0.0\t2\t55037 1211 38\n",
      "27\t218\t0.0\t2\t0 214 4\n",
      "28\t11\t0.0\t2\t0 0 11\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: COI_2-demux-pairedend-trimmed1.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#Import the fastq files into a QIIME2 artifact \n",
    "\n",
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path leray_manifest_run2_v1.csv \\\n",
    "  --output-path COI_2-paired-end-demux.qza \\\n",
    "  --input-format PairedEndFastqManifestPhred33\n",
    "!qiime cutadapt trim-paired \\\n",
    "    --i-demultiplexed-sequences COI_2-paired-end-demux.qza \\\n",
    "    --p-front-f '^GGWACWGGWTGAACWGTWTAYCCYCC' \\\n",
    "    --p-front-r '^TANACYTCNGGRTGNCCRAARAAYCA' \\\n",
    "    --o-trimmed-sequences COI_2-demux-pairedend-trimmed1.qza \\\n",
    "    --verbose\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI_2-demux-pairedend-trimmed1.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "  --i-data COI_2-demux-pairedend-trimmed1.qza \\\n",
    "  --o-visualization COI_2-demux-pairedend-trimmed1.qzv\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-z7_r9jz9')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: f08cc630-8e09-4f5e-ae81-c188b5510da6>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "q2.Visualization.load('COI_2-demux-pairedend-trimmed1.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI_2-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI_2-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: COI_2-paired-end-demux_stats-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#Generate ESVs (Exact Sequence Variants). QIIME2 has two packages available for this. We are using dada2 for our single end reads.\n",
    "# NOTE: if you are wanting to do paired end reads, you'll use qiime dada2 denoise-paired (https://docs.qiime2.org/2019.7/plugins/available/dada2/denoise-paired/)\n",
    "# NOTE: if you are wanting to join reads you'll have to use Deblur\n",
    "\n",
    "\n",
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs COI_2-demux-pairedend-trimmed1.qza \\\n",
    "  --p-trunc-len-f 265 \\\n",
    "  --p-trunc-len-r 235 \\\n",
    "  --p-pooling-method pseudo \\\n",
    "  --p-chimera-method consensus \\\n",
    "  --p-n-threads 0 \\\n",
    "  --p-n-reads-learn 10000 \\\n",
    "  --o-representative-sequences COI_2-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-table COI_2-paired-end-demux_table-dada2.qza \\\n",
    "  --o-denoising-stats COI_2-paired-end-demux_stats-dada2.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI-paired-end-demux_table-dada2.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI-paired-end-demux_rep-seqs-dada2.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "#now merge the two runs\n",
    "!qiime feature-table merge \\\n",
    "  --i-tables COI_1-paired-end-demux_table-dada2.qza \\\n",
    "  --i-tables COI_2-paired-end-demux_table-dada2.qza \\\n",
    "  --o-merged-table COI-paired-end-demux_table-dada2.qza\n",
    "!qiime feature-table merge-seqs \\\n",
    "  --i-data COI_1-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --i-data COI_2-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-merged-data COI-paired-end-demux_rep-seqs-dada2.qza\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-table.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "# The feature-table summarize command will give you information on how many sequences are associated with each sample and with each feature, histograms of those distributions, and some related summary statistics. \n",
    "!qiime feature-table summarize \\\n",
    "  --i-table COI-paired-end-demux_table-dada2.qza \\\n",
    "  --o-visualization COI-table.qzv \\\n",
    "  --m-sample-metadata-file metadata_COI_clean.txt\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-i9kh7ib9')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 1aca7d5c-1b20-4a5d-98a2-6a48f56cfdbb>"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('COI-table.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-paired-end-demux_rep-seqs-dada2.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  COI-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --o-visualization COI-paired-end-demux_rep-seqs-dada2.qzv\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI-paired-end-demux_table-dada2_vsearch97.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!qiime vsearch cluster-features-de-novo \\\n",
    "  --i-table COI-paired-end-demux_table-dada2.qza \\\n",
    "  --i-sequences COI-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --p-perc-identity 0.97 \\\n",
    "  --o-clustered-table  COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --o-clustered-sequences COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data  COI-paired-end-demux_rep-seqs-dada2_vsearch97.qza  \\\n",
    "  --o-visualization COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-v5fs00ac')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: 97305863-bdf6-4e76-a9d2-79d086e4e313>"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('COI-paired-end-demux_rep-seqs-dada2_vsearch97.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: COI-paired-end-demux_table-dada2_vsearch97.qzv\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime feature-table summarize \\\n",
    "  --i-table COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --o-visualization COI-paired-end-demux_table-dada2_vsearch97.qzv \\\n",
    "  --m-sample-metadata-file metadata_COI_clean.txt\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/6q/fwkjrzpn4tnf1xgdkbr06_p80000gn/T/qiime2-archive-84ec783y')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: b13796a3-7def-4f50-a076-51ea7477fef4>"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "#View the .qzv\n",
    "q2.Visualization.load('COI-paired-end-demux_table-dada2_vsearch97.qzv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported COI-paired-end-demux_table-dada2_vsearch97.qza as BIOMV210DirFmt to directory exported-feature-table\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "!qiime tools export \\\n",
    "  --input-path COI-paired-end-demux_table-dada2_vsearch97.qza \\\n",
    "  --output-path exported-feature-table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o COI_97vsearch_OTUtable.tsv --to-tsv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o exported-feature-table/feature-table_hdf5.biom --to-hdf5 --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "!biom convert -i exported-feature-table/feature-table.biom -o exported-feature-table/feature-table_json.biom --to-json --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mExported COI-paired-end-demux_table-dada2.qza as BIOMV210DirFmt to directory exported-feature-table-asvs\u001b[0m\r\n"
     ]
    }
   ],
   "source": [
    "#convert COI asvs for direct comparison against 18S: \n",
    "!qiime tools export \\\n",
    "  --input-path COI-paired-end-demux_table-dada2.qza \\\n",
    "  --output-path exported-feature-table-asvs\n",
    "!biom convert -i exported-feature-table-asvs/feature-table.biom -o exported-feature-table-asvs/feature-table_json.biom --to-json --table-type=\"OTU table\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: COI-paired-end-demux_table-dada2_vsearch95.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: COI-paired-end-demux_rep-seqs-dada2_vsearch95.qza\u001b[0m\n",
      "\u001b[32mExported COI-paired-end-demux_table-dada2_vsearch95.qza as BIOMV210DirFmt to directory exported-feature-table-95\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "## cluster at 95%\n",
    "!qiime vsearch cluster-features-de-novo \\\n",
    "  --i-table COI-paired-end-demux_table-dada2.qza \\\n",
    "  --i-sequences COI-paired-end-demux_rep-seqs-dada2.qza \\\n",
    "  --p-perc-identity 0.95 \\\n",
    "  --o-clustered-table  COI-paired-end-demux_table-dada2_vsearch95.qza \\\n",
    "  --o-clustered-sequences COI-paired-end-demux_rep-seqs-dada2_vsearch95.qza\n",
    "\n",
    "!qiime tools export \\\n",
    "  --input-path COI-paired-end-demux_table-dada2_vsearch95.qza \\\n",
    "  --output-path exported-feature-table-95\n",
    "!biom convert -i exported-feature-table-95/feature-table.biom -o exported-feature-table-95/feature-table_json.biom --to-json --table-type=\"OTU table\"\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "\n",
    "qiime alignment mafft \\\n",
    "  --i-sequences $OUTDIR/rep-seqs-dn-97.qza \\\n",
    "  --o-alignment $OUTDIR/aligned-rep-seqs-dn-97.qza\n",
    "\n",
    "qiime alignment mask \\\n",
    "  --i-alignment $OUTDIR/aligned-rep-seqs-dn-97.qza \\\n",
    "  --o-masked-alignment $OUTDIR/masked-aligned-rep-seqs-dn-97.qza\n",
    "\n",
    "qiime phylogeny fasttree \\\n",
    "  --i-alignment $OUTDIR/masked-aligned-rep-seqs-dn-97.qza \\\n",
    "  --o-tree $OUTDIR/unrooted-tree.qza\n",
    "\n",
    "qiime phylogeny midpoint-root \\\n",
    "  --i-tree $OUTDIR/unrooted-tree.qza \\\n",
    "  --o-rooted-tree $OUTDIR/rooted-tree.qza\n",
    "\n",
    "qiime tools export \\\n",
    "  $OUTDIR/unrooted-tree.qza \\\n",
    "  --output-dir $OUTDIR/exported-tree-unrooted-97\n",
    "\n",
    "qiime tools export \\\n",
    "  $OUTDIR/rooted-tree.qza \\\n",
    "  --output-dir $OUTDIR/exported-tree-rooted-97\n",
    "\n",
    "\n",
    "##########################################\n",
    "#classify based on consensus blast against nt\n",
    "#export TMPDIR='/usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/CCE_LTER_P1604_02-11-2019/leray_qiime2_3-7-2019/newtempdir'\n",
    "#df -h\n",
    "#\n",
    "#qiime tools import \\\n",
    "#  --type 'FeatureData[Sequence]' \\\n",
    "#  --input-path /usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/databases/references_ncbi_nt/nt.fasta \\\n",
    "#  --output-path /usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/databases/references_ncbi_nt/nt_sequences_all.qza\n",
    "#\n",
    "#export TMPDIR='/usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/CCE_LTER_P1604_02-11-2019/leray_qiime2_3-7-2019/newtempdir'\n",
    "#df -h\n",
    "#\n",
    "#qiime tools import \\\n",
    "#  --type 'FeatureData[Taxonomy]' \\\n",
    "#  --source-format HeaderlessTSVTaxonomyFormat \\\n",
    "#  --input-path /usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/databases/references_ncbi_nt/GIlist.taxid.lineages.dedup.tax \\\n",
    "#  --output-path /usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/databases/references_ncbi_nt/nt_taxonomy_pasted.qza\n",
    "#\n",
    "#export TMPDIR='/usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/CCE_LTER_P1604_02-11-2019/leray_qiime2_3-7-2019/newtempdir'\n",
    "#df -h\n",
    "#qiime feature-classifier classify-consensus-blast \\\n",
    "#  --i-query $OUTDIR/rep-seqs-dn-97.qza \\\n",
    "#  --i-reference-reads /usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/databases/references_ncbi_nt/nt_sequences_all.qza \\\n",
    "#  --i-reference-taxonomy /usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/databases/references_ncbi_nt/nt_taxonomy_pasted.qza \\\n",
    "#  --o-classification $OUTDIR/taxonomy-97_blast.qza\n",
    "#\n",
    "#export TMPDIR='/usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/CCE_LTER_P1604_02-11-2019/leray_qiime2_3-7-2019/newtempdir'\n",
    "#df -h\n",
    "#qiime metadata tabulate \\\n",
    "#  --m-input-file $OUTDIRtax/taxonomy-97_blast.qza \\\n",
    "#  --o-visualization $OUTDIRtax/taxonomy-97_blast.qzv\n",
    "#\n",
    "###classifier trained on MIDORI sequences - see /usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/databases/references_midori/train_classifier_midori_unique.sh for details\n",
    "\n",
    "qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier /usr/local/projdata/0568/projects/PLANKTON/illumina_aallen/sasommer/databases/references_midori/COI_unique_4-26-2019/taxo_classifier_Midori_CO1_unique_Leray.qza \\\n",
    "  --i-reads $OUTDIR/rep-seqs-dn-97.qza \\\n",
    "  --o-classification $OUTDIR/taxonomy-97_MIDORI.qza\n",
    "\n",
    "qiime metadata tabulate \\\n",
    "  --m-input-file $OUTDIR/taxonomy-97_MIDORI.qza \\\n",
    "  --o-visualization $OUTDIR/taxonomy-97_MIDORI.qzv\n",
    "\n",
    "  \n",
    "## Taxonomy barplot\n",
    "printf \"\\nProducing taxonomy barplots\\n\"\n",
    "qiime taxa barplot \\\n",
    " --i-table $OUTDIR/table-dn-97.qza \\\n",
    " --i-taxonomy $OUTDIR/taxonomy-97_MIDORI.qza \\\n",
    " --m-metadata-file $WORKDIR/leray_metadata.tsv \\\n",
    " --o-visualization $OUTDIR/taxa_bar_plots_97_MIDORI.qzv\n",
    "\n",
    "## Taxonomy barplot\n",
    "printf \"\\nProducing taxonomy barplots\\n\"\n",
    "qiime taxa barplot \\\n",
    " --i-table $OUTDIR/table-dn-97.qza \\\n",
    " --i-taxonomy $OUTDIR/taxonomy-97_blast.qza \\\n",
    " --m-metadata-file $WORKDIR/leray_metadata.tsv \\\n",
    " --o-visualization $OUTDIR/taxa_bar_plots_97_blast.qzv\n",
    "\n",
    "\n",
    "#########################################\n",
    "#Alpha-rarefaction curves\n",
    "\n",
    "qiime diversity alpha-rarefaction \\\n",
    " --i-table $OUTDIR/table_dada2.qza \\\n",
    " --i-phylogeny $OUTDIR/rooted-tree.qza \\\n",
    " --p-max-depth  8000\\\n",
    " --m-metadata-file $WORKDIR/leray_metadata.tsv \\\n",
    " --o-visualization $OUTDIR/alpha_rarefaction.qzv\n",
    "\n",
    "#Beta-diversity\n",
    "\n",
    "qiime diversity core-metrics-phylogenetic \\\n",
    "--i-phylogeny $OUTDIR/rooted-tree.qza \\\n",
    "--i-table $OUTDIR/table_dada2.qza \\\n",
    "--p-sampling-depth  8000 \\\n",
    "--m-metadata-file $WORKDIR/leray_metadata.tsv \\\n",
    "--output-dir $OUTDIR/core-metrics-results\n",
    "\n",
    "qiime diversity alpha-group-significance \\\n",
    "--i-alpha-diversity $OUTDIR/core-metrics-results/faith_pd_vector.qza \\\n",
    "--m-metadata-file $WORKDIR/leray_metadata.tsv \\\n",
    "--o-visualization $OUTDIR/core-metrics-results/faith-pd-group-significance.qzv\n",
    "\n",
    "qiime diversity alpha-group-significance \\\n",
    "--i-alpha-diversity $OUTDIR/core-metrics-results/evenness_vector.qza \\\n",
    "--m-metadata-file $WORKDIR/leray_metadata.tsv \\\n",
    "--o-visualization $OUTDIR/core-metrics-results/evenness-group-significance.qzv\n",
    "\n",
    "qiime emperor plot \\\n",
    "  --i-pcoa $OUTDIR/core-metrics-results/bray_curtis_pcoa_results.qza \\\n",
    "  --m-metadata-file $WORKDIR/leray_metadata.tsv \\\n",
    "  --p-custom-axes numreads \\\n",
    "  --o-visualization $OUTDIR/core-metrics-results/bray-curtis-emperor-numreads.qzv\n",
    "\n",
    "qiime emperor plot \\\n",
    "  --i-pcoa $OUTDIR/core-metrics-results/bray_curtis_pcoa_results.qza \\\n",
    "  --m-metadata-file $WORKDIR/leray_metadata.tsv \\\n",
    "  --p-custom-axes percenprocessed \\\n",
    "  --o-visualization $OUTDIR/core-metrics-results/bray-curtis-emperor-perprocess.qzv\n",
    "\n",
    "\n",
    "qiime tools export \\\n",
    "$OUTDIR/table-dn-97.qza \\\n",
    "--output-dir $OUTDIR/exported-table\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
